{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Bwa Aln hg38 + QDNAseq new version hg38"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# This is a Notebook to run CNV analysis server\n",
    "\n",
    "I am using it to have the steps for the CNV analaysis in one go and notbook as pipeline, and it can be run it form here but is running really running from the terminal\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/DATA/home/l.gonzalez/Notebooks\r\n"
     ]
    }
   ],
   "source": [
    "!pwd\n",
    "#!ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os, sys\n",
    "import glob\n",
    "import logging"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Before starting the analysis we will check the QC of the fastq files usign fastqc\n",
    "To do that, please go to the cell below and change the path to the folder where the fastq files are, then run it"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 5% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 10% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 15% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 20% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 25% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 30% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 35% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 40% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 45% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 50% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 55% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 60% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 65% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 70% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 75% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 80% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 85% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 90% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 95% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Approx 100% complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Analysis complete for 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "Started analysis of 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 5% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 10% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 15% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 20% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 25% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 30% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 35% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 40% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 45% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 50% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 55% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 60% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 65% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 70% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 75% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 80% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 85% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 90% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 95% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Approx 100% complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Analysis complete for 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "Started analysis of 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 5% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 10% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 15% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 20% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 25% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 30% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 35% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 40% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 45% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 50% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 55% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 60% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 65% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 70% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 75% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 80% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 85% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 90% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 95% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Approx 100% complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Analysis complete for 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "Started analysis of 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 5% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 10% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 15% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 20% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 25% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 30% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 35% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 40% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 45% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 50% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 55% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 60% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 65% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 70% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 75% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 80% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 85% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 90% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Approx 95% complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n",
      "Analysis complete for 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n"
     ]
    }
   ],
   "source": [
    "## This is to run with the Organids 50 and 51,\n",
    "\n",
    "!fastqc /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/markdup/*markdup.bam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### As an optional step we can run multiqc after fastqc to create a unified html files with all the fastqc to see then in one go"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run bwa aln for the mapping part single end, check first the QC with fastqc\n",
    "There is no need for trimming (conda)\n",
    "The CLI can be run directly"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Script:\n",
    "#BWA ALN\n",
    "\n",
    "#Ref\n",
    "#/DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa\n",
    "\n",
    "#data\n",
    "prj=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq\n",
    "\n",
    "\n",
    "for file in ${prj}/*.f*q.gz\n",
    "do\n",
    "\n",
    "   inputfqgz=$file\n",
    "   #outputbam=${file/%.fastq.gz/.bam}  #  % means that suffixes are matched, # would mean prefixes\"\n",
    "   filenamestem=${file/%.f*q.gz/}\n",
    "   \n",
    "   echo \"!!!!!!!!!!!!!! Expected final output {$filenamestem.markdup.bam}\"\n",
    "\n",
    "   bwa aln -t 32 -n 2 \\\n",
    "    /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa  \\\n",
    "    $inputfqgz | bwa samse -r '@RG\\tID:inputfqgz\\tSM:inputfqgz' \\\n",
    "    /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa \\\n",
    "    - $inputfqgz | samtools view -Su - | samtools sort - -o $filenamestem.sorted.bam > $filenamestem.sorted.bam\n",
    "\n",
    "   picard MarkDuplicates -I $filenamestem.sorted.bam -O $filenamestem.markdup.bam -M $filenamestem.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX true -VALIDATION_STRINGENCY LENIENT\n",
    "\n",
    "done\n",
    "\n",
    "                        \n",
    "\n",
    "mkdir fastQC\n",
    "#mv *f*q.gz fastQC\n",
    "\n",
    "mkdir sorted_bam\n",
    "mv *sorted.bam sorted_bam\n",
    "#rm *sorted.bam\n",
    "\n",
    "mkdir markdup_bam\n",
    "mv *markdup.bam markdup_bam  \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 40.17 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 39.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.42 sec\n",
      "[bwa_aln_core] refine gapped alignments... 1.30 sec\n",
      "[bwa_aln_core] print alignments... 0.44 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.85 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.88 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.37 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 39.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.24 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 37.41 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.03 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.21 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 37.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.11 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.75 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.77 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.72 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.16 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.07 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 46.71 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.39 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.20 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.41 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.75 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.39 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.07 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.02 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.03 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.72 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 13369344 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 13369344 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 13631488 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.15 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 13631488 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 13893632 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 4.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 13893632 sequences have been processed.\n",
      "0.01 sec\n",
      "[bwa_aln_core] 13917850 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.fastq.gz\n",
      "[main] Real time: 415.375 sec; CPU: 2405.635 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 2.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.45 sec\n",
      "[bwa_aln_core] print alignments... 0.03 sec\n",
      "[bwa_aln_core] 13917850 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.fastq.gz\n",
      "[main] Real time: 418.482 sec; CPU: 272.692 sec\n",
      "[bam_sort_core] merging from 4 files and 1 in-memory blocks...\n",
      "INFO\t2021-12-06 15:16:24\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "15:16:25.876 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Mon Dec 06 15:16:25 CET 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Mon Dec 06 15:16:25 CET 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-91-generic amd64; OpenJDK 64-Bit Server VM 10.0.2+13; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-12-06 15:16:26\tMarkDuplicates\tStart of doWork freeMemory: 530785248; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:16:26\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-12-06 15:16:26\tMarkDuplicates\tWill retain up to 3890368 data points before spilling to disk.\n",
      "INFO\t2021-12-06 15:16:28\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:02s.  Time for last 1,000,000:    2s.  Last read position: 10:12,418,350\n",
      "INFO\t2021-12-06 15:16:28\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:16:31\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    2s.  Last read position: 11:107,419,368\n",
      "INFO\t2021-12-06 15:16:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:16:33\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:07s.  Time for last 1,000,000:    2s.  Last read position: 13:50,210,962\n",
      "INFO\t2021-12-06 15:16:33\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:16:38\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    4s.  Last read position: 15:60,850,626\n",
      "INFO\t2021-12-06 15:16:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:16:40\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    2s.  Last read position: 18:22,032,221\n",
      "INFO\t2021-12-06 15:16:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:16:43\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    2s.  Last read position: 2:96,074,520\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-12-06 15:16:43\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:16:45\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:19s.  Time for last 1,000,000:    2s.  Last read position: 20:38,061,569\n",
      "INFO\t2021-12-06 15:16:45\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:16:49\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:23s.  Time for last 1,000,000:    3s.  Last read position: 3:123,764,868\n",
      "INFO\t2021-12-06 15:16:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:16:52\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:26s.  Time for last 1,000,000:    2s.  Last read position: 4:155,670,038\n",
      "INFO\t2021-12-06 15:16:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:16:54\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:    2s.  Last read position: 6:58,208,645\n",
      "INFO\t2021-12-06 15:16:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:16:56\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:00:30s.  Time for last 1,000,000:    2s.  Last read position: 7:93,633,807\n",
      "INFO\t2021-12-06 15:16:56\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:17:00\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:00:34s.  Time for last 1,000,000:    3s.  Last read position: 8:115,764,752\n",
      "INFO\t2021-12-06 15:17:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 12569362, Read name D00608:266:CCYKGANXX:2:2304:12670:70771, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569363, Read name D00608:266:CCYKGANXX:2:2112:2475:50494, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569364, Read name D00608:266:CCYKGANXX:2:1110:20556:40721, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569365, Read name D00608:266:CCYKGANXX:2:1208:4682:36087, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569366, Read name D00608:266:CCYKGANXX:2:2308:9861:82815, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569367, Read name D00608:266:CCYKGANXX:2:1207:10441:54196, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569368, Read name D00608:266:CCYKGANXX:2:2210:5407:39177, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12610000, Read name D00608:266:CCYKGANXX:2:1214:12850:46585, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12610001, Read name D00608:266:CCYKGANXX:2:2311:1825:24184, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12610002, Read name D00608:266:CCYKGANXX:2:1201:17397:12024, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12610003, Read name D00608:266:CCYKGANXX:2:1213:16760:83075, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12610004, Read name D00608:266:CCYKGANXX:2:1212:19529:60489, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-12-06 15:17:03\tMarkDuplicates\tRead    13,000,000 records.  Elapsed time: 00:00:37s.  Time for last 1,000,000:    2s.  Last read position: X:113,955,512\n",
      "INFO\t2021-12-06 15:17:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 13340736, Read name D00608:266:CCYKGANXX:2:2205:20485:6847, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 13367766, Read name D00608:266:CCYKGANXX:2:2316:10999:22185, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 13370037, Read name D00608:266:CCYKGANXX:2:1110:20628:42326, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-12-06 15:17:04\tMarkDuplicates\tRead 13370519 records. 0 pairs never matched.\n",
      "INFO\t2021-12-06 15:17:05\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 512121384; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:17:05\tMarkDuplicates\tWill retain up to 33554432 duplicate indices before spilling to disk.\n",
      "INFO\t2021-12-06 15:17:05\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-12-06 15:17:05\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-12-06 15:17:05\tSortingCollection\tCreating merging iterator from 4 files\n",
      "INFO\t2021-12-06 15:17:07\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-12-06 15:17:07\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 285754904; totalMemory: 564133888; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:17:07\tMarkDuplicates\tMarking 166500 records as duplicates.\n",
      "INFO\t2021-12-06 15:17:07\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-12-06 15:17:07\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2021-12-06 15:18:24\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:16s.  Time for last 10,000,000:   76s.  Last read position: 6:58,208,645\n",
      "Ignoring SAM validation error: ERROR: Record 12569362, Read name D00608:266:CCYKGANXX:2:2304:12670:70771, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569363, Read name D00608:266:CCYKGANXX:2:2112:2475:50494, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569364, Read name D00608:266:CCYKGANXX:2:1110:20556:40721, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569365, Read name D00608:266:CCYKGANXX:2:1208:4682:36087, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569366, Read name D00608:266:CCYKGANXX:2:2308:9861:82815, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569367, Read name D00608:266:CCYKGANXX:2:1207:10441:54196, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12569368, Read name D00608:266:CCYKGANXX:2:2210:5407:39177, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12610000, Read name D00608:266:CCYKGANXX:2:1214:12850:46585, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12610001, Read name D00608:266:CCYKGANXX:2:2311:1825:24184, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12610002, Read name D00608:266:CCYKGANXX:2:1201:17397:12024, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12610003, Read name D00608:266:CCYKGANXX:2:1213:16760:83075, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12610004, Read name D00608:266:CCYKGANXX:2:1212:19529:60489, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 13340736, Read name D00608:266:CCYKGANXX:2:2205:20485:6847, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 13367766, Read name D00608:266:CCYKGANXX:2:2316:10999:22185, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 13370037, Read name D00608:266:CCYKGANXX:2:1110:20628:42326, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-12-06 15:18:53\tMarkDuplicates\tBefore output close freeMemory: 527588976; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:18:53\tMarkDuplicates\tAfter output close freeMemory: 527588904; totalMemory: 536870912; maxMemory: 1073741824\n",
      "[Mon Dec 06 15:18:53 CET 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.47 minutes.\n",
      "Runtime.totalMemory()=536870912\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 49.84 sec\n",
      "[bwa_aln_core] write to the disk... 0.07 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.90 sec\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.20 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.99 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.28 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.43 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.20 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.70 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.20 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.14 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.29 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.60 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.52 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 51.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.36 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.71 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.35 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 7.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.70 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.01 sec\n",
      "[bwa_aln_core] 8424795 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.fastq.gz[bwa_aln_core] convert to sequence coordinate... \n",
      "[main] Real time: 255.653 sec; CPU: 1705.629 sec\n",
      "2.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.46 sec\n",
      "[bwa_aln_core] print alignments... 0.04 sec\n",
      "[bwa_aln_core] 8424795 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.fastq.gz\n",
      "[main] Real time: 258.962 sec; CPU: 170.669 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-12-06 15:23:41\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "15:23:41.729 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Mon Dec 06 15:23:41 CET 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Mon Dec 06 15:23:41 CET 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-91-generic amd64; OpenJDK 64-Bit Server VM 10.0.2+13; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-12-06 15:23:41\tMarkDuplicates\tStart of doWork freeMemory: 530785792; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:23:41\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-12-06 15:23:41\tMarkDuplicates\tWill retain up to 3890368 data points before spilling to disk.\n",
      "INFO\t2021-12-06 15:23:44\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:02s.  Time for last 1,000,000:    2s.  Last read position: 10:117,551,812\n",
      "INFO\t2021-12-06 15:23:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:23:47\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    2s.  Last read position: 14:50,362,916\n",
      "INFO\t2021-12-06 15:23:47\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:23:49\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:07s.  Time for last 1,000,000:    2s.  Last read position: 17:63,692,664\n",
      "INFO\t2021-12-06 15:23:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:23:54\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    4s.  Last read position: 2:172,575,542\n",
      "INFO\t2021-12-06 15:23:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:23:56\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    2s.  Last read position: 3:128,623,564\n",
      "INFO\t2021-12-06 15:23:56\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-12-06 15:23:59\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    2s.  Last read position: 6:43,281,388\n",
      "INFO\t2021-12-06 15:23:59\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:24:01\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:19s.  Time for last 1,000,000:    2s.  Last read position: 8:61,271,000\n",
      "INFO\t2021-12-06 15:24:01\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 7503412, Read name D00608:266:CCYKGANXX:2:1301:3175:14092, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7503413, Read name D00608:266:CCYKGANXX:2:1208:9266:21348, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7503414, Read name D00608:266:CCYKGANXX:2:1208:12566:32992, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7503415, Read name D00608:266:CCYKGANXX:2:1208:3771:54742, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7503416, Read name D00608:266:CCYKGANXX:2:1109:16226:3777, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7503417, Read name D00608:266:CCYKGANXX:2:2306:12900:11982, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7515091, Read name D00608:266:CCYKGANXX:2:2115:2325:81014, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7515092, Read name D00608:266:CCYKGANXX:2:1203:15706:9275, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7515093, Read name D00608:266:CCYKGANXX:2:2309:7575:65415, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7515094, Read name D00608:266:CCYKGANXX:2:2314:18644:90778, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7515095, Read name D00608:266:CCYKGANXX:2:1314:4819:26455, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7822638, Read name D00608:266:CCYKGANXX:2:2215:12733:10873, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-12-06 15:24:05\tMarkDuplicates\tRead 7824305 records. 0 pairs never matched.\n",
      "INFO\t2021-12-06 15:24:05\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 512123160; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:24:05\tMarkDuplicates\tWill retain up to 33554432 duplicate indices before spilling to disk.\n",
      "INFO\t2021-12-06 15:24:05\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-12-06 15:24:05\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-12-06 15:24:05\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-12-06 15:24:06\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-12-06 15:24:06\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 258492432; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:24:06\tMarkDuplicates\tMarking 98923 records as duplicates.\n",
      "INFO\t2021-12-06 15:24:06\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-12-06 15:24:06\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 7503412, Read name D00608:266:CCYKGANXX:2:1301:3175:14092, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7503413, Read name D00608:266:CCYKGANXX:2:1208:9266:21348, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7503414, Read name D00608:266:CCYKGANXX:2:1208:12566:32992, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7503415, Read name D00608:266:CCYKGANXX:2:1208:3771:54742, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7503416, Read name D00608:266:CCYKGANXX:2:1109:16226:3777, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7503417, Read name D00608:266:CCYKGANXX:2:2306:12900:11982, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7515091, Read name D00608:266:CCYKGANXX:2:2115:2325:81014, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7515092, Read name D00608:266:CCYKGANXX:2:1203:15706:9275, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7515093, Read name D00608:266:CCYKGANXX:2:2309:7575:65415, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7515094, Read name D00608:266:CCYKGANXX:2:2314:18644:90778, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7515095, Read name D00608:266:CCYKGANXX:2:1314:4819:26455, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7822638, Read name D00608:266:CCYKGANXX:2:2215:12733:10873, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-12-06 15:25:09\tMarkDuplicates\tBefore output close freeMemory: 527590344; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:25:09\tMarkDuplicates\tAfter output close freeMemory: 527590344; totalMemory: 536870912; maxMemory: 1073741824\n",
      "[Mon Dec 06 15:25:09 CET 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.47 minutes.\n",
      "Runtime.totalMemory()=536870912\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 63.91 sec\n",
      "[bwa_aln_core] write to the disk... 0.07 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.16 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.65 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.61 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.68 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.59 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.66 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.27 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.72 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 67.16 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.99 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.71 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.16 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.11 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.68 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 75.21 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 72.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 74.28 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.14 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 5139279 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.fastq.gz\n",
      "[main] Real time: 160.356 sec; CPU: 1326.920 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.24 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.54 sec\n",
      "[bwa_aln_core] print alignments... 0.17 sec\n",
      "[bwa_aln_core] 5139279 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.fastq.gz\n",
      "[main] Real time: 164.331 sec; CPU: 100.916 sec\n",
      "[bam_sort_core] merging from 1 files and 1 in-memory blocks...\n",
      "INFO\t2021-12-06 15:28:11\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "15:28:11.856 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Mon Dec 06 15:28:11 CET 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Mon Dec 06 15:28:11 CET 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-91-generic amd64; OpenJDK 64-Bit Server VM 10.0.2+13; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-12-06 15:28:11\tMarkDuplicates\tStart of doWork freeMemory: 530785920; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:28:11\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-12-06 15:28:11\tMarkDuplicates\tWill retain up to 3890368 data points before spilling to disk.\n",
      "INFO\t2021-12-06 15:28:14\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:02s.  Time for last 1,000,000:    2s.  Last read position: 13:105,438,443\n",
      "INFO\t2021-12-06 15:28:14\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-12-06 15:28:17\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    2s.  Last read position: 2:98,858,419\n",
      "INFO\t2021-12-06 15:28:17\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:28:19\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:07s.  Time for last 1,000,000:    2s.  Last read position: 5:116,882,157\n",
      "INFO\t2021-12-06 15:28:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 3975771, Read name D00608:266:CCYKGANXX:2:1214:13721:60932, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 3975772, Read name D00608:266:CCYKGANXX:2:2214:5086:82716, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 3975773, Read name D00608:266:CCYKGANXX:2:1303:16719:38311, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 3975774, Read name D00608:266:CCYKGANXX:2:2312:3305:42935, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 3975775, Read name D00608:266:CCYKGANXX:2:1305:8114:7983, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-12-06 15:28:24\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    4s.  Last read position: X:1,757,165\n",
      "INFO\t2021-12-06 15:28:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:28:25\tMarkDuplicates\tRead 4246988 records. 0 pairs never matched.\n",
      "INFO\t2021-12-06 15:28:25\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 512124472; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:28:25\tMarkDuplicates\tWill retain up to 33554432 duplicate indices before spilling to disk.\n",
      "INFO\t2021-12-06 15:28:25\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-12-06 15:28:25\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-12-06 15:28:25\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-12-06 15:28:26\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-12-06 15:28:26\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 258493928; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:28:26\tMarkDuplicates\tMarking 57438 records as duplicates.\n",
      "INFO\t2021-12-06 15:28:26\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-12-06 15:28:26\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 3975771, Read name D00608:266:CCYKGANXX:2:1214:13721:60932, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 3975772, Read name D00608:266:CCYKGANXX:2:2214:5086:82716, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 3975773, Read name D00608:266:CCYKGANXX:2:1303:16719:38311, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 3975774, Read name D00608:266:CCYKGANXX:2:2312:3305:42935, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 3975775, Read name D00608:266:CCYKGANXX:2:1305:8114:7983, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-12-06 15:29:05\tMarkDuplicates\tBefore output close freeMemory: 527558648; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:29:05\tMarkDuplicates\tAfter output close freeMemory: 527558648; totalMemory: 536870912; maxMemory: 1073741824\n",
      "[Mon Dec 06 15:29:05 CET 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 0.89 minutes.\n",
      "Runtime.totalMemory()=536870912\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 53.87 sec\n",
      "[bwa_aln_core] write to the disk... 0.07 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.38 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.43 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.99 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.50 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.06 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.91 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.24 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 57.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.03 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.12 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.78 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.20 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7846548 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.fastq.gz\n",
      "[main] Real time: 234.873 sec; CPU: 1703.221 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7846548 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.fastq.gz\n",
      "[main] Real time: 239.762 sec; CPU: 152.110 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-12-06 15:33:31\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "15:33:31.414 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Mon Dec 06 15:33:31 CET 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Mon Dec 06 15:33:31 CET 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-91-generic amd64; OpenJDK 64-Bit Server VM 10.0.2+13; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-12-06 15:33:31\tMarkDuplicates\tStart of doWork freeMemory: 530785808; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:33:31\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-12-06 15:33:31\tMarkDuplicates\tWill retain up to 3890368 data points before spilling to disk.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-12-06 15:33:34\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:02s.  Time for last 1,000,000:    2s.  Last read position: 11:22,389,014\n",
      "INFO\t2021-12-06 15:33:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:33:36\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    2s.  Last read position: 15:30,075,589\n",
      "INFO\t2021-12-06 15:33:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:33:39\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:07s.  Time for last 1,000,000:    2s.  Last read position: 19:19,348,934\n",
      "INFO\t2021-12-06 15:33:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:33:43\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    4s.  Last read position: 21:42,750,992\n",
      "INFO\t2021-12-06 15:33:43\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:33:46\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    2s.  Last read position: 5:40,428,287\n",
      "INFO\t2021-12-06 15:33:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-12-06 15:33:48\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    2s.  Last read position: 7:152,272,244\n",
      "INFO\t2021-12-06 15:33:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 6628090, Read name D00608:266:CCYKGANXX:2:1208:14816:76291, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6639891, Read name D00608:266:CCYKGANXX:2:2216:1634:61909, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-12-06 15:33:51\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:19s.  Time for last 1,000,000:    2s.  Last read position: GL000220.1:122,270\n",
      "INFO\t2021-12-06 15:33:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 7033186, Read name D00608:266:CCYKGANXX:2:2205:1423:15351, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-12-06 15:33:51\tMarkDuplicates\tRead 7033668 records. 0 pairs never matched.\n",
      "INFO\t2021-12-06 15:33:52\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 512122936; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:33:52\tMarkDuplicates\tWill retain up to 33554432 duplicate indices before spilling to disk.\n",
      "INFO\t2021-12-06 15:33:52\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-12-06 15:33:52\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-12-06 15:33:52\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-12-06 15:33:53\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-12-06 15:33:53\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 258493616; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:33:53\tMarkDuplicates\tMarking 69109 records as duplicates.\n",
      "INFO\t2021-12-06 15:33:53\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-12-06 15:33:53\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 6628090, Read name D00608:266:CCYKGANXX:2:1208:14816:76291, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6639891, Read name D00608:266:CCYKGANXX:2:2216:1634:61909, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7033186, Read name D00608:266:CCYKGANXX:2:2205:1423:15351, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-12-06 15:34:54\tMarkDuplicates\tBefore output close freeMemory: 527591528; totalMemory: 536870912; maxMemory: 1073741824\n",
      "INFO\t2021-12-06 15:34:54\tMarkDuplicates\tAfter output close freeMemory: 527591528; totalMemory: 536870912; maxMemory: 1073741824\n",
      "[Mon Dec 06 15:34:55 CET 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.39 minutes.\n",
      "Runtime.totalMemory()=536870912\n",
      "mv: cannot stat '*.sorted.bam': No such file or directory\n",
      "mv: cannot stat '*markdup.bam': No such file or directory\n"
     ]
    }
   ],
   "source": [
    "## Alignment with bwa aln, code is at /home/l.gonzalez/Notebooks/bash_R_scripts/1_bwaAln_hg38_lgl.sh\n",
    "## Go to the folder where you want to save the results\n",
    "## Check in the code, it is quite general but reference is hard copied\n",
    "## this code I meant to be run from the same folder where teh fastq files are store\n",
    "\n",
    "\n",
    "### Remember the project folder IS HARDCODE, need to be changed\n",
    "\n",
    "!bash /DATA/home/l.gonzalez/Notebooks/bash_R_scripts/alternatve_bwaAln_hg38_lgl.sh\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Check the results, \n",
    "put them in the appropiate folders. Folder should be created in the process, but better check\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Indexing the bam files to create the bai, otherwise we can not continue"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "indexing: /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/markdup/5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/markdup/5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/markdup/5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/markdup/5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup.bam\n"
     ]
    }
   ],
   "source": [
    "# for Indexing the bam I have a tiny script that work quite nicely\n",
    "### Remember the project folder IS HARDCODE, need to be changed\n",
    "\n",
    "##WORKS\n",
    "\n",
    "!bash /DATA/home/l.gonzalez/Notebooks/bash_R_scripts/indexing.sh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!/bin/bash\n",
    "#prj=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/markdup\n",
    "prj=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/markdup\n",
    "for i in ${prj}/*markdup.bam\n",
    "\n",
    "do\n",
    "\n",
    "echo \"indexing: \"$i\n",
    "samtools index $i $i\".bai\"\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### After the analysis we will check the QC of the steps that we ran\n",
    "To do that, please go to the cell below and change the path to the folders where the fastqc files are, also the bam/bai files and multiqc will create a html with all the files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[1;30m[WARNING]\u001b[0m         multiqc : \u001b[33mMultiQC Version v1.11 now available!\u001b[0m\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : This is MultiQC v1.10.1\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : Template    : default\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : Searching   : /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq\n",
      "\u001b[2KSearching   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[35m100%\u001b[0m \u001b[32m36/36\u001b[0m  ATCGCA_S41_R1_001.markdup_fastqc.html\u001b[0m.html\u001b[0m\n",
      "\u001b[?25h\u001b[1;30m[INFO   ]\u001b[0m          picard : Found 4 MarkDuplicates reports\n",
      "\u001b[1;30m[INFO   ]\u001b[0m          fastqc : Found 4 reports\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : Compressing plot data\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : Report      : multiqc_report.html\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : Data        : multiqc_data\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : MultiQC complete\n"
     ]
    }
   ],
   "source": [
    "!multiqc /DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/fastq/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run QDNAseq for the calling, \n",
    "This new script, modified form Erik run from bam to called \n",
    "The CLI can be run directly but using R\n",
    "\n",
    "The code have some hardcode parts, path to project directory(output) and path to bam files (input)\n",
    "If you wanna run it with hg38, not need to change that part\n",
    "For the bin size it should be change also inside the code, for the calling and devawing \n",
    "\n",
    "To make those cahnges, there are three options:\n",
    "- Open the file here using the notebooks \n",
    "- Open in Rstudio  \n",
    "- Open in the terminal\n",
    "\n",
    "Since it is a Rmarkdown file there are also two ways to run it:\n",
    "- Inside the Rstudio, step/chunk by step/chunk\n",
    "- From terminal (easier)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "R version 3.6.3 (2020-02-29) -- \"Holding the Windsock\"\n",
      "Copyright (C) 2020 The R Foundation for Statistical Computing\n",
      "Platform: x86_64-pc-linux-gnu (64-bit)\n",
      "\n",
      "R is free software and comes with ABSOLUTELY NO WARRANTY.\n",
      "You are welcome to redistribute it under certain conditions.\n",
      "Type 'license()' or 'licence()' for distribution details.\n",
      "\n",
      "  Natural language support but running in an English locale\n",
      "\n",
      "R is a collaborative project with many contributors.\n",
      "Type 'contributors()' for more information and\n",
      "'citation()' on how to cite R or R packages in publications.\n",
      "\n",
      "Type 'demo()' for some demos, 'help()' for on-line help, or\n",
      "'help.start()' for an HTML browser interface to help.\n",
      "Type 'q()' to quit R.\n",
      "\n",
      "> rmarkdown::render('/DATA/share/pipelines/lgleon/IntEnd/DNA/Shallow_Seq/Results_QDNAseqFlow/BatchD_9-10/btD_hg38_100kb/QDNASeq_hg38_IntEnd_btD_server100kb.Rmd')\n",
      "\n",
      "\n",
      "processing file: QDNASeq_hg38_IntEnd_btD_server100kb.Rmd\n",
      "  |..                                                                    |   3%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |....                                                                  |   6%\n",
      "label: unnamed-chunk-1 (with options) \n",
      "List of 1\n",
      " $ include: logi FALSE\n",
      "\n",
      "  |.......                                                               |   9%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.........                                                             |  12%\n",
      "label: setup (with options) \n",
      "List of 1\n",
      " $ include: logi FALSE\n",
      "\n",
      "  |...........                                                           |  16%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.............                                                         |  19%\n",
      "label: functions (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |...............                                                       |  22%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |..................                                                    |  25%\n",
      "label: wd (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |....................                                                  |  28%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |......................                                                |  31%\n",
      "label: paths (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |........................                                              |  34%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |..........................                                            |  38%\n",
      "label: bam2smooth (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "    6322_1_T20-60638_I-1 (1 of 177): binned read counts loaded from cache\n",
      "    6322_10_T20-60664_I-1 (2 of 177): binned read counts loaded from cache\n",
      "    6322_100_T20-60765_I-1 (3 of 177): binned read counts loaded from cache\n",
      "    6322_101_T20-60767_I-1 (4 of 177): binned read counts loaded from cache\n",
      "    6322_102_T20-60768_I-1 (5 of 177): binned read counts loaded from cache\n",
      "    6322_103_T20-60769_I-1 (6 of 177): binned read counts loaded from cache\n",
      "    6322_104_T20-60770_I-1 (7 of 177): binned read counts loaded from cache\n",
      "    6322_11_T20-60669_I-1 (8 of 177): binned read counts loaded from cache\n",
      "    6322_12_T20-60672_I-1 (9 of 177): binned read counts loaded from cache\n",
      "    6322_13_T20-60676_I-1 (10 of 177): binned read counts loaded from cache\n",
      "    6322_14_T20-60680_I-1 (11 of 177): binned read counts loaded from cache\n",
      "    6322_15_T20-60652_I-1 (12 of 177): binned read counts loaded from cache\n",
      "    6322_16_T20-60654_I-1 (13 of 177): binned read counts loaded from cache\n",
      "    6322_17_T20-60656_I-1 (14 of 177): binned read counts loaded from cache\n",
      "    6322_18_T20-60657_I-1 (15 of 177): binned read counts loaded from cache\n",
      "    6322_19_T20-60658_I-1 (16 of 177): binned read counts loaded from cache\n",
      "    6322_2_T20-60641_I-1 (17 of 177): binned read counts loaded from cache\n",
      "    6322_20_T20-60661_I-1 (18 of 177): binned read counts loaded from cache\n",
      "    6322_21_T20-60662_I-1 (19 of 177): binned read counts loaded from cache\n",
      "    6322_22_T20-60663_I-1 (20 of 177): binned read counts loaded from cache\n",
      "    6322_23_T20-60666_I-1 (21 of 177): binned read counts loaded from cache\n",
      "    6322_24_T20-60667_I-1 (22 of 177): binned read counts loaded from cache\n",
      "    6322_25_T20-60668_I-1 (23 of 177): binned read counts loaded from cache\n",
      "    6322_26_T20-60702_I-1 (24 of 177): binned read counts loaded from cache\n",
      "    6322_27_T20-60704_I-1 (25 of 177): binned read counts loaded from cache\n",
      "    6322_28_T20-60706_I-1 (26 of 177): binned read counts loaded from cache\n",
      "    6322_29_T20-60708_I-1 (27 of 177): binned read counts loaded from cache\n",
      "    6322_3_T20-60644_I-1 (28 of 177): binned read counts loaded from cache\n",
      "    6322_30_T20-60718_I-1 (29 of 177): binned read counts loaded from cache\n",
      "    6322_31_T20-60671_I-1 (30 of 177): binned read counts loaded from cache\n",
      "    6322_33_T20-60674_I-1 (31 of 177): binned read counts loaded from cache\n",
      "    6322_34_T20-60675_I-1 (32 of 177): binned read counts loaded from cache\n",
      "    6322_35_T20-60677_I-1 (33 of 177): binned read counts loaded from cache\n",
      "    6322_36_T20-60678_I-1 (34 of 177): binned read counts loaded from cache\n",
      "    6322_38_T20-60686_I-1 (35 of 177): binned read counts loaded from cache\n",
      "    6322_39_T20-60688_I-1 (36 of 177): binned read counts loaded from cache\n",
      "    6322_4_T20-60645_I-1 (37 of 177): binned read counts loaded from cache\n",
      "    6322_40_T20-60690_I-1 (38 of 177): binned read counts loaded from cache\n",
      "    6322_43_T20-60694_I-1 (39 of 177): binned read counts loaded from cache\n",
      "    6322_44_T20-60697_I-1 (40 of 177): binned read counts loaded from cache\n",
      "    6322_45_T20-60703_I-1 (41 of 177): binned read counts loaded from cache\n",
      "    6322_46_T20-60705_I-1 (42 of 177): binned read counts loaded from cache\n",
      "    6322_48_T20-60710_I-1 (43 of 177): binned read counts loaded from cache\n",
      "    6322_49_T20-60711_I-1 (44 of 177): binned read counts loaded from cache\n",
      "    6322_5_T20-60647_I-1 (45 of 177): binned read counts loaded from cache\n",
      "    6322_50_T20-60714_I-1 (46 of 177): binned read counts loaded from cache\n",
      "    6322_51_T20-60715_I-1 (47 of 177): binned read counts loaded from cache\n",
      "    6322_52_T20-60717_I-1 (48 of 177): binned read counts loaded from cache\n",
      "    6322_53_T20-60723_I-1 (49 of 177): binned read counts loaded from cache\n",
      "    6322_54_T20-60724_I-1 (50 of 177): binned read counts loaded from cache\n",
      "    6322_55_T20-60709_I-1 (51 of 177): binned read counts loaded from cache\n",
      "    6322_56_T20-60712_I-1 (52 of 177): binned read counts loaded from cache\n",
      "    6322_57_T20-60713_I-1 (53 of 177): binned read counts loaded from cache\n",
      "    6322_58_T20-60719_I-1 (54 of 177): binned read counts loaded from cache\n",
      "    6322_59_T20-60720_I-1 (55 of 177): binned read counts loaded from cache\n",
      "    6322_6_T20-60653_I-1 (56 of 177): binned read counts loaded from cache\n",
      "    6322_60_T20-60721_I-1 (57 of 177): binned read counts loaded from cache\n",
      "    6322_61_T20-60722_I-1 (58 of 177): binned read counts loaded from cache\n",
      "    6322_62_T20-60728_I-1 (59 of 177): binned read counts loaded from cache\n",
      "    6322_63_T20-60734_I-1 (60 of 177): binned read counts loaded from cache\n",
      "    6322_64_T20-60743_I-1 (61 of 177): binned read counts loaded from cache\n",
      "    6322_65_T20-60745_I-1 (62 of 177): binned read counts loaded from cache\n",
      "    6322_66_T20-60746_I-1 (63 of 177): binned read counts loaded from cache\n",
      "    6322_67_T20-60755_I-1 (64 of 177): binned read counts loaded from cache\n",
      "    6322_68_T20-60756_I-1 (65 of 177): binned read counts loaded from cache\n",
      "    6322_69_T20-60760_I-1 (66 of 177): binned read counts loaded from cache\n",
      "    6322_7_T20-60655_I-1 (67 of 177): binned read counts loaded from cache\n",
      "    6322_70_T20-60725_I-1 (68 of 177): binned read counts loaded from cache\n",
      "    6322_71_T20-60726_I-1 (69 of 177): binned read counts loaded from cache\n",
      "    6322_72_T20-60727_I-1 (70 of 177): binned read counts loaded from cache\n",
      "    6322_73_T20-60729_I-1 (71 of 177): binned read counts loaded from cache\n",
      "    6322_74_T20-60733_I-1 (72 of 177): binned read counts loaded from cache\n",
      "    6322_75_T20-60738_I-1 (73 of 177): binned read counts loaded from cache\n",
      "    6322_76_T20-60747_I-1 (74 of 177): binned read counts loaded from cache\n",
      "    6322_77_T20-60748_I-1 (75 of 177): binned read counts loaded from cache\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    6322_78_T20-60750_I-1 (76 of 177): binned read counts loaded from cache\n",
      "    6322_79_T20-60752_I-1 (77 of 177): binned read counts loaded from cache\n",
      "    6322_8_T20-60659_I-1 (78 of 177): binned read counts loaded from cache\n",
      "    6322_80_T20-60753_I-1 (79 of 177): binned read counts loaded from cache\n",
      "    6322_81_T20-60754_I-1 (80 of 177): binned read counts loaded from cache\n",
      "    6322_82_T20-60757_I-1 (81 of 177): binned read counts loaded from cache\n",
      "    6322_83_T20-60758_I-1 (82 of 177): binned read counts loaded from cache\n",
      "    6322_84_T20-60759_I-1 (83 of 177): binned read counts loaded from cache\n",
      "    6322_85_T20-60764_I-1 (84 of 177): binned read counts loaded from cache\n",
      "    6322_86_T20-60665_I-1 (85 of 177): binned read counts loaded from cache\n",
      "    6322_87_T20-60665_I-2 (86 of 177): binned read counts loaded from cache\n",
      "    6322_88_T20-60700_I-1 (87 of 177): binned read counts loaded from cache\n",
      "    6322_89_T20-60744_I-1 (88 of 177): binned read counts loaded from cache\n",
      "    6322_9_T20-60660_I-1 (89 of 177): binned read counts loaded from cache\n",
      "    6322_90_T20-60744_I-2 (90 of 177): binned read counts loaded from cache\n",
      "    6322_91_T20-60761_I-1 (91 of 177): binned read counts loaded from cache\n",
      "    6322_92_T20-60762_I-1 (92 of 177): binned read counts loaded from cache\n",
      "    6322_93_T20-60763_I-1 (93 of 177): binned read counts loaded from cache\n",
      "    6322_94_T20-60766_I-1 (94 of 177): binned read counts loaded from cache\n",
      "    6322_95_T20-60444_I-1 (95 of 177): binned read counts loaded from cache\n",
      "    6322_96_T20-60453_I-1 (96 of 177): binned read counts loaded from cache\n",
      "    6322_97_T20-60598_I-1 (97 of 177): binned read counts loaded from cache\n",
      "    6322_98_T20-60598_I-2 (98 of 177): binned read counts loaded from cache\n",
      "    6322_99_T20-60626_I-1 (99 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_1_T20-60470_I-1_TAACTTGGTC-GTCGTGAATC_S2 (100 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_10_T20-60558_I-1_TCCAACTGAA-TGTCCGTAGG_S11 (101 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_11_T20-60564_I-1_ATTCGGCTTA-TTAACGTCCG_S17 (102 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_12_T20-60570_I-1_TATGGCAAGC-AACAGCAATG_S18 (103 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_13_T20-60572_I-1_CCTAAGCTGA-CGTCGCATCT_S19 (104 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_14_T20-60580_I-1_GCCACCTAGT-TGCGATGTAT_S20 (105 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_15_T20-60475_I-1_AGTATCCGAT-TATTGCGCTA_S16 (106 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_16_T20-60476_I-1_CCTCGTTGGA-GACTGCGTTA_S23 (107 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_17_T20-60596_I-1_CGACCTTAAT-GCTCTTAGGT_S24 (108 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_18_T20-60597_I-1_GAACCGCATT-CATTATGCCT_S25 (109 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_19_T20-60600_I-1_CGCGGTATCT-ATTACCGATC_S26 (110 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_2_T20-60501_I-1_TCGAACACGA-GCGAGGTTCT_S5 (111 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_20_T20-60603_I-1_ACTCCGAAGC-TCCGTGTTGC_S27 (112 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_21_T20-60608_I-1_CGATGTCTTG-CCGAACGTTC_S22 (113 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_22_T20-60609_I-1_CTCCGCCATA-GCGTTCAGCA_S30 (114 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_23_T20-60611_I-1_AACGAAGATC-TGTGGTCTAA_S31 (115 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_24_T20-60612_I-1_AATAGGTAGG-TGGCTTATGG_S32 (116 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_25_T20-60613_I-1_ATGTTGCCAC-TAGAAGCCAT_S33 (117 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_26_T20-60615_I-1_ACAACAGGCT-TCCACTGTTG_S34 (118 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_27_T20-60616_I-1_TATCCGCGTC-GTTGCATCGT_S35 (119 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_28_T20-60618_I-1_AATCTGCAGT-TGCGAAGTGA_S29 (120 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_29_T20-60619_I-1_GAGATGAGAG-GAATGGCGTT_S38 (121 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_3_T20-60502_I-1_TTACTTCGTG-AAGATCGGAT_S4 (122 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_30_T20-60620_I-1_TGACACGATT-GCCTAGGAAG_S39 (123 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_31_T20-60622_I-1_CCTCCTGTGA-CACGTCTACG_S43 (124 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_32_T20-60623_I-1_TATGCGTAGC-TCAGAGGCAC_S44 (125 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_33_T20-60628_I-1_CAGCAGGTCA-CACTTAGGTA_S42 (126 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_34_T20-60681_I-1_TTACGCGCAT-CTTAACGACA_S41 (127 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_35_T20-60682_I-1_AGGATGTCCA-ACTGGACGTT_S40 (128 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_36_T20-60683_I-1_AGTTCAGGCG-CACAGCTATT_S37 (129 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_37_T20-60685_I-1_TCACCATAAG-AAGTTCAGGA_S48 (130 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_38_T20-60687_I-1_GTGCTCTAGG-GTTACCAACG_S49 (131 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_39_T20-60689_I-1_TAGGCGAATG-AGTCCAGTGA_S50 (132 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_4_T20-60504_I-1_ACGAGACGTC-CGGTCTCTCT_S8 (133 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_40_T20-60691_I-1_CCAGTCGTCA-CGCTAGGCTA_S1 (134 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_41_T20-60695_I-1_TCTAGCTGCG-ATAGGTAAGG_S3 (135 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_42_T20-60696_I-1_TTCTCTTAGC-TCCTCGAGCT_S6 (136 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_43_T20-60698_I-1_GCCAGCTATC-TTGTTCGTGT_S47 (137 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_44_T20-60699_I-1_CGCCAAGGTA-GATAGTGTTC_S10 (138 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_45_T20-60701_I-1_ACTTCCGGCA-GACCAATTGT_S46 (139 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_46_T20-60437_I-1_GCAATCTGCT-ACGAACCAAG_S54 (140 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_47_T20-60446_I-1_AGTAAGGAGA-ATGCTTGGTC_S15 (141 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_48_T20-60455_I-1_AGAAGACCTA-AACCGTAACA_S55 (142 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_49_T20-60456_I-1_AGATTAGCGT-CGCCTTATAT_S53 (143 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_5_T20-60505_I-1_CATAAGCGAC-CCTCTTCAAG_S9 (144 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_50_T20-60457_I-1_TCGCGACACT-CGAGTAATGT_S21 (145 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_51_T20-60462_I-1_CTTACGAACT-ACGGACTGAG_S52 (146 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_52_T20-60466_I-1_CCTGACATCT-TAGCTACCAG_S59 (147 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_53_T20-60468_I-1_AAGCGCTTAG-TGTCTGTGAC_S28 (148 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_54_T20-60469_I-1_CTCTATTGAG-CACGCAAGTC_S60 (149 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_55_T20-60474_I-1_CCGAACAGAG-TGATACTCGC_S58 (150 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_56_T20-60491_I-1_CCGTGAGTTC-CTTGGTATGA_S57 (151 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_57_T20-60542_I-1_AGGTAACAAC-GCACGATGCT_S36 (152 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_58_T20-60629_I-1_TGGAGATCGA-CAGCGGTATG_S64 (153 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_59_T20-60634_I-1_CAAGACTCAA-CACCGTCATA_S65 (154 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_6_T20-60527_I-1_AACTCTCTAC-ACGGCTTCTC_S7 (155 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_60_T20-60635_I-1_CGAGCGTAGT-TGCTCACGCA_S63 (156 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_61_T20-60636_I-1_AACAACACAG-GAACATTGGC_S62 (157 of 177): binned read counts loaded from cache\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    RUN1097_6309_62_T20-60639_I-1_CACTTACGGC-GCTGGAGAAG_S45 (158 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_63_T20-60640_I-1_AGAACTAACG-TTATCGAGAG_S69 (159 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_64_T20-60642_I-1_TCAATTCCGA-TGTAGATCCG_S68 (160 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_65_T20-60643_I-1_CGCACTTCGT-AATATGCCAG_S51 (161 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_66_T20-60646_I-1_CCAGACCGAT-GCTACTTGCA_S67 (162 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_67_T20-60648_I-1_TCGACGGCAA-GTAGTTGTGT_S56 (163 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_68_T20-60651_I-1_CTGGATTACT-AAGTGTGCTA_S70 (164 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_69_T20-60561_I-1_AGACTAGATG-CGTAACACTA_S71 (165 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_7_T20-60540_I-1_GTATGCCACA-CCGAGTCTTC_S12 (166 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_70_T20-60599_I-1_TCAGCATCTA-CCAGTGTACG_S61 (167 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_71_T20-60605_I-1_GAGCAGCTTG-TGCCAGTATT_S72 (168 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_72_T20-60606_I-1_CGGAACACAA-AACTCTACCG_S66 (169 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_73_T20-60610_I-1_CTTGCTATTC-TAGAGTCACG_S73 (170 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_74_T20-60617_I-1_CCACAGCCTT-GCTATTGAAG_S74 (171 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_75_T20-60621_I-1_TCAGTTCGAT-CTGAGTCCTT_S75 (172 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_76_T20-60625_I-1_AGCAATTCAC-TCATCGTCTG_S76 (173 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_77_T20-60627_I-1_CCTCTATAGA-GTACTTCCAA_S77 (174 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_78_T20-60637_I-1_GTTGCATCAA-GTTGATGCGA_S78 (175 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_8_T20-60541_I-1_GCCAACAGAC-CGTTCACGTT_S13 (176 of 177): binned read counts loaded from cache\n",
      "    RUN1097_6309_9_T20-60546_I-1_ATAAGCGGAG-CTGTTCTACG_S14 (177 of 177): binned read counts loaded from cache\n",
      "30,894\ttotal bins\n",
      "28,760\tof which in selected chromosomes\n",
      "27,679\tof which with reference sequence\n",
      "25,147\tfinal bins\n",
      "Calculating correction for GC content and mappability\n",
      "    Calculating fit for sample 6322_1_T20-60638_I-1 (1 of 177) ...\n",
      "    Calculating fit for sample 6322_10_T20-60664_I-1 (2 of 177) ...\n",
      "    Calculating fit for sample 6322_100_T20-60765_I-1 (3 of 177) ...\n",
      "    Calculating fit for sample 6322_101_T20-60767_I-1 (4 of 177) ...\n",
      "    Calculating fit for sample 6322_102_T20-60768_I-1 (5 of 177) ...\n",
      "    Calculating fit for sample 6322_103_T20-60769_I-1 (6 of 177) ...\n",
      "    Calculating fit for sample 6322_104_T20-60770_I-1 (7 of 177) ...\n",
      "    Calculating fit for sample 6322_11_T20-60669_I-1 (8 of 177) ...\n",
      "    Calculating fit for sample 6322_12_T20-60672_I-1 (9 of 177) ...\n",
      "    Calculating fit for sample 6322_13_T20-60676_I-1 (10 of 177) ...\n",
      "    Calculating fit for sample 6322_14_T20-60680_I-1 (11 of 177) ...\n",
      "    Calculating fit for sample 6322_15_T20-60652_I-1 (12 of 177) ...\n",
      "    Calculating fit for sample 6322_16_T20-60654_I-1 (13 of 177) ...\n",
      "    Calculating fit for sample 6322_17_T20-60656_I-1 (14 of 177) ...\n",
      "    Calculating fit for sample 6322_18_T20-60657_I-1 (15 of 177) ...\n",
      "    Calculating fit for sample 6322_19_T20-60658_I-1 (16 of 177) ...\n",
      "    Calculating fit for sample 6322_2_T20-60641_I-1 (17 of 177) ...\n",
      "    Calculating fit for sample 6322_20_T20-60661_I-1 (18 of 177) ...\n",
      "    Calculating fit for sample 6322_21_T20-60662_I-1 (19 of 177) ...\n",
      "    Calculating fit for sample 6322_22_T20-60663_I-1 (20 of 177) ...\n",
      "    Calculating fit for sample 6322_23_T20-60666_I-1 (21 of 177) ...\n",
      "    Calculating fit for sample 6322_24_T20-60667_I-1 (22 of 177) ...\n",
      "    Calculating fit for sample 6322_25_T20-60668_I-1 (23 of 177) ...\n",
      "    Calculating fit for sample 6322_26_T20-60702_I-1 (24 of 177) ...\n",
      "    Calculating fit for sample 6322_27_T20-60704_I-1 (25 of 177) ...\n",
      "    Calculating fit for sample 6322_28_T20-60706_I-1 (26 of 177) ...\n",
      "    Calculating fit for sample 6322_29_T20-60708_I-1 (27 of 177) ...\n",
      "    Calculating fit for sample 6322_3_T20-60644_I-1 (28 of 177) ...\n",
      "    Calculating fit for sample 6322_30_T20-60718_I-1 (29 of 177) ...\n",
      "    Calculating fit for sample 6322_31_T20-60671_I-1 (30 of 177) ...\n",
      "    Calculating fit for sample 6322_33_T20-60674_I-1 (31 of 177) ...\n",
      "    Calculating fit for sample 6322_34_T20-60675_I-1 (32 of 177) ...\n",
      "    Calculating fit for sample 6322_35_T20-60677_I-1 (33 of 177) ...\n",
      "    Calculating fit for sample 6322_36_T20-60678_I-1 (34 of 177) ...\n",
      "    Calculating fit for sample 6322_38_T20-60686_I-1 (35 of 177) ...\n",
      "    Calculating fit for sample 6322_39_T20-60688_I-1 (36 of 177) ...\n",
      "    Calculating fit for sample 6322_4_T20-60645_I-1 (37 of 177) ...\n",
      "    Calculating fit for sample 6322_40_T20-60690_I-1 (38 of 177) ...\n",
      "    Calculating fit for sample 6322_43_T20-60694_I-1 (39 of 177) ...\n",
      "    Calculating fit for sample 6322_44_T20-60697_I-1 (40 of 177) ...\n",
      "    Calculating fit for sample 6322_45_T20-60703_I-1 (41 of 177) ...\n",
      "    Calculating fit for sample 6322_46_T20-60705_I-1 (42 of 177) ...\n",
      "    Calculating fit for sample 6322_48_T20-60710_I-1 (43 of 177) ...\n",
      "    Calculating fit for sample 6322_49_T20-60711_I-1 (44 of 177) ...\n",
      "    Calculating fit for sample 6322_5_T20-60647_I-1 (45 of 177) ...\n",
      "    Calculating fit for sample 6322_50_T20-60714_I-1 (46 of 177) ...\n",
      "    Calculating fit for sample 6322_51_T20-60715_I-1 (47 of 177) ...\n",
      "    Calculating fit for sample 6322_52_T20-60717_I-1 (48 of 177) ...\n",
      "    Calculating fit for sample 6322_53_T20-60723_I-1 (49 of 177) ...\n",
      "    Calculating fit for sample 6322_54_T20-60724_I-1 (50 of 177) ...\n",
      "    Calculating fit for sample 6322_55_T20-60709_I-1 (51 of 177) ...\n",
      "    Calculating fit for sample 6322_56_T20-60712_I-1 (52 of 177) ...\n",
      "    Calculating fit for sample 6322_57_T20-60713_I-1 (53 of 177) ...\n",
      "    Calculating fit for sample 6322_58_T20-60719_I-1 (54 of 177) ...\n",
      "    Calculating fit for sample 6322_59_T20-60720_I-1 (55 of 177) ...\n",
      "    Calculating fit for sample 6322_6_T20-60653_I-1 (56 of 177) ...\n",
      "    Calculating fit for sample 6322_60_T20-60721_I-1 (57 of 177) ...\n",
      "    Calculating fit for sample 6322_61_T20-60722_I-1 (58 of 177) ...\n",
      "    Calculating fit for sample 6322_62_T20-60728_I-1 (59 of 177) ...\n",
      "    Calculating fit for sample 6322_63_T20-60734_I-1 (60 of 177) ...\n",
      "    Calculating fit for sample 6322_64_T20-60743_I-1 (61 of 177) ...\n",
      "    Calculating fit for sample 6322_65_T20-60745_I-1 (62 of 177) ...\n",
      "    Calculating fit for sample 6322_66_T20-60746_I-1 (63 of 177) ...\n",
      "    Calculating fit for sample 6322_67_T20-60755_I-1 (64 of 177) ...\n",
      "    Calculating fit for sample 6322_68_T20-60756_I-1 (65 of 177) ...\n",
      "    Calculating fit for sample 6322_69_T20-60760_I-1 (66 of 177) ...\n",
      "    Calculating fit for sample 6322_7_T20-60655_I-1 (67 of 177) ...\n",
      "    Calculating fit for sample 6322_70_T20-60725_I-1 (68 of 177) ...\n",
      "    Calculating fit for sample 6322_71_T20-60726_I-1 (69 of 177) ...\n",
      "    Calculating fit for sample 6322_72_T20-60727_I-1 (70 of 177) ...\n",
      "    Calculating fit for sample 6322_73_T20-60729_I-1 (71 of 177) ...\n",
      "    Calculating fit for sample 6322_74_T20-60733_I-1 (72 of 177) ...\n",
      "    Calculating fit for sample 6322_75_T20-60738_I-1 (73 of 177) ...\n",
      "    Calculating fit for sample 6322_76_T20-60747_I-1 (74 of 177) ...\n",
      "    Calculating fit for sample 6322_77_T20-60748_I-1 (75 of 177) ...\n",
      "    Calculating fit for sample 6322_78_T20-60750_I-1 (76 of 177) ...\n",
      "    Calculating fit for sample 6322_79_T20-60752_I-1 (77 of 177) ...\n",
      "    Calculating fit for sample 6322_8_T20-60659_I-1 (78 of 177) ...\n",
      "    Calculating fit for sample 6322_80_T20-60753_I-1 (79 of 177) ...\n",
      "    Calculating fit for sample 6322_81_T20-60754_I-1 (80 of 177) ...\n",
      "    Calculating fit for sample 6322_82_T20-60757_I-1 (81 of 177) ...\n",
      "    Calculating fit for sample 6322_83_T20-60758_I-1 (82 of 177) ...\n",
      "    Calculating fit for sample 6322_84_T20-60759_I-1 (83 of 177) ...\n",
      "    Calculating fit for sample 6322_85_T20-60764_I-1 (84 of 177) ...\n",
      "    Calculating fit for sample 6322_86_T20-60665_I-1 (85 of 177) ...\n",
      "    Calculating fit for sample 6322_87_T20-60665_I-2 (86 of 177) ...\n",
      "    Calculating fit for sample 6322_88_T20-60700_I-1 (87 of 177) ...\n",
      "    Calculating fit for sample 6322_89_T20-60744_I-1 (88 of 177) ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    Calculating fit for sample 6322_9_T20-60660_I-1 (89 of 177) ...\n",
      "    Calculating fit for sample 6322_90_T20-60744_I-2 (90 of 177) ...\n",
      "    Calculating fit for sample 6322_91_T20-60761_I-1 (91 of 177) ...\n",
      "    Calculating fit for sample 6322_92_T20-60762_I-1 (92 of 177) ...\n",
      "    Calculating fit for sample 6322_93_T20-60763_I-1 (93 of 177) ...\n",
      "    Calculating fit for sample 6322_94_T20-60766_I-1 (94 of 177) ...\n",
      "    Calculating fit for sample 6322_95_T20-60444_I-1 (95 of 177) ...\n",
      "    Calculating fit for sample 6322_96_T20-60453_I-1 (96 of 177) ...\n",
      "    Calculating fit for sample 6322_97_T20-60598_I-1 (97 of 177) ...\n",
      "    Calculating fit for sample 6322_98_T20-60598_I-2 (98 of 177) ...\n",
      "    Calculating fit for sample 6322_99_T20-60626_I-1 (99 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_1_T20-60470_I-1_TAACTTGGTC-GTCGTGAATC_S2 (100 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_10_T20-60558_I-1_TCCAACTGAA-TGTCCGTAGG_S11 (101 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_11_T20-60564_I-1_ATTCGGCTTA-TTAACGTCCG_S17 (102 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_12_T20-60570_I-1_TATGGCAAGC-AACAGCAATG_S18 (103 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_13_T20-60572_I-1_CCTAAGCTGA-CGTCGCATCT_S19 (104 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_14_T20-60580_I-1_GCCACCTAGT-TGCGATGTAT_S20 (105 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_15_T20-60475_I-1_AGTATCCGAT-TATTGCGCTA_S16 (106 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_16_T20-60476_I-1_CCTCGTTGGA-GACTGCGTTA_S23 (107 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_17_T20-60596_I-1_CGACCTTAAT-GCTCTTAGGT_S24 (108 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_18_T20-60597_I-1_GAACCGCATT-CATTATGCCT_S25 (109 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_19_T20-60600_I-1_CGCGGTATCT-ATTACCGATC_S26 (110 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_2_T20-60501_I-1_TCGAACACGA-GCGAGGTTCT_S5 (111 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_20_T20-60603_I-1_ACTCCGAAGC-TCCGTGTTGC_S27 (112 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_21_T20-60608_I-1_CGATGTCTTG-CCGAACGTTC_S22 (113 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_22_T20-60609_I-1_CTCCGCCATA-GCGTTCAGCA_S30 (114 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_23_T20-60611_I-1_AACGAAGATC-TGTGGTCTAA_S31 (115 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_24_T20-60612_I-1_AATAGGTAGG-TGGCTTATGG_S32 (116 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_25_T20-60613_I-1_ATGTTGCCAC-TAGAAGCCAT_S33 (117 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_26_T20-60615_I-1_ACAACAGGCT-TCCACTGTTG_S34 (118 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_27_T20-60616_I-1_TATCCGCGTC-GTTGCATCGT_S35 (119 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_28_T20-60618_I-1_AATCTGCAGT-TGCGAAGTGA_S29 (120 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_29_T20-60619_I-1_GAGATGAGAG-GAATGGCGTT_S38 (121 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_3_T20-60502_I-1_TTACTTCGTG-AAGATCGGAT_S4 (122 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_30_T20-60620_I-1_TGACACGATT-GCCTAGGAAG_S39 (123 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_31_T20-60622_I-1_CCTCCTGTGA-CACGTCTACG_S43 (124 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_32_T20-60623_I-1_TATGCGTAGC-TCAGAGGCAC_S44 (125 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_33_T20-60628_I-1_CAGCAGGTCA-CACTTAGGTA_S42 (126 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_34_T20-60681_I-1_TTACGCGCAT-CTTAACGACA_S41 (127 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_35_T20-60682_I-1_AGGATGTCCA-ACTGGACGTT_S40 (128 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_36_T20-60683_I-1_AGTTCAGGCG-CACAGCTATT_S37 (129 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_37_T20-60685_I-1_TCACCATAAG-AAGTTCAGGA_S48 (130 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_38_T20-60687_I-1_GTGCTCTAGG-GTTACCAACG_S49 (131 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_39_T20-60689_I-1_TAGGCGAATG-AGTCCAGTGA_S50 (132 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_4_T20-60504_I-1_ACGAGACGTC-CGGTCTCTCT_S8 (133 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_40_T20-60691_I-1_CCAGTCGTCA-CGCTAGGCTA_S1 (134 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_41_T20-60695_I-1_TCTAGCTGCG-ATAGGTAAGG_S3 (135 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_42_T20-60696_I-1_TTCTCTTAGC-TCCTCGAGCT_S6 (136 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_43_T20-60698_I-1_GCCAGCTATC-TTGTTCGTGT_S47 (137 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_44_T20-60699_I-1_CGCCAAGGTA-GATAGTGTTC_S10 (138 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_45_T20-60701_I-1_ACTTCCGGCA-GACCAATTGT_S46 (139 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_46_T20-60437_I-1_GCAATCTGCT-ACGAACCAAG_S54 (140 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_47_T20-60446_I-1_AGTAAGGAGA-ATGCTTGGTC_S15 (141 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_48_T20-60455_I-1_AGAAGACCTA-AACCGTAACA_S55 (142 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_49_T20-60456_I-1_AGATTAGCGT-CGCCTTATAT_S53 (143 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_5_T20-60505_I-1_CATAAGCGAC-CCTCTTCAAG_S9 (144 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_50_T20-60457_I-1_TCGCGACACT-CGAGTAATGT_S21 (145 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_51_T20-60462_I-1_CTTACGAACT-ACGGACTGAG_S52 (146 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_52_T20-60466_I-1_CCTGACATCT-TAGCTACCAG_S59 (147 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_53_T20-60468_I-1_AAGCGCTTAG-TGTCTGTGAC_S28 (148 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_54_T20-60469_I-1_CTCTATTGAG-CACGCAAGTC_S60 (149 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_55_T20-60474_I-1_CCGAACAGAG-TGATACTCGC_S58 (150 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_56_T20-60491_I-1_CCGTGAGTTC-CTTGGTATGA_S57 (151 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_57_T20-60542_I-1_AGGTAACAAC-GCACGATGCT_S36 (152 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_58_T20-60629_I-1_TGGAGATCGA-CAGCGGTATG_S64 (153 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_59_T20-60634_I-1_CAAGACTCAA-CACCGTCATA_S65 (154 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_6_T20-60527_I-1_AACTCTCTAC-ACGGCTTCTC_S7 (155 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_60_T20-60635_I-1_CGAGCGTAGT-TGCTCACGCA_S63 (156 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_61_T20-60636_I-1_AACAACACAG-GAACATTGGC_S62 (157 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_62_T20-60639_I-1_CACTTACGGC-GCTGGAGAAG_S45 (158 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_63_T20-60640_I-1_AGAACTAACG-TTATCGAGAG_S69 (159 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_64_T20-60642_I-1_TCAATTCCGA-TGTAGATCCG_S68 (160 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_65_T20-60643_I-1_CGCACTTCGT-AATATGCCAG_S51 (161 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_66_T20-60646_I-1_CCAGACCGAT-GCTACTTGCA_S67 (162 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_67_T20-60648_I-1_TCGACGGCAA-GTAGTTGTGT_S56 (163 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_68_T20-60651_I-1_CTGGATTACT-AAGTGTGCTA_S70 (164 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_69_T20-60561_I-1_AGACTAGATG-CGTAACACTA_S71 (165 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_7_T20-60540_I-1_GTATGCCACA-CCGAGTCTTC_S12 (166 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_70_T20-60599_I-1_TCAGCATCTA-CCAGTGTACG_S61 (167 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_71_T20-60605_I-1_GAGCAGCTTG-TGCCAGTATT_S72 (168 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_72_T20-60606_I-1_CGGAACACAA-AACTCTACCG_S66 (169 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_73_T20-60610_I-1_CTTGCTATTC-TAGAGTCACG_S73 (170 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_74_T20-60617_I-1_CCACAGCCTT-GCTATTGAAG_S74 (171 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_75_T20-60621_I-1_TCAGTTCGAT-CTGAGTCCTT_S75 (172 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_76_T20-60625_I-1_AGCAATTCAC-TCATCGTCTG_S76 (173 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_77_T20-60627_I-1_CCTCTATAGA-GTACTTCCAA_S77 (174 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_78_T20-60637_I-1_GTTGCATCAA-GTTGATGCGA_S78 (175 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_8_T20-60541_I-1_GCCAACAGAC-CGTTCACGTT_S13 (176 of 177) ...\n",
      "    Calculating fit for sample RUN1097_6309_9_T20-60546_I-1_ATAAGCGGAG-CTGTTCTACG_S14 (177 of 177) ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Done.\n",
      "Applying median normalization ...\n",
      "Smoothing outliers ...\n",
      "  |............................                                          |  41%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |...............................                                       |  44%\n",
      "label: dewaving (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "Loading required package: gtools\n",
      "Applying deWaving ... \n",
      "  |.................................                                     |  47%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |...................................                                   |  50%\n",
      "label: calling (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "Performing segmentation:\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "EM algorithm started ... \n",
      "\n",
      "386557362059557206.5473.57549529111456781403.2850.47549529132621890403.21011.9\n",
      "Calling iteration1:\n",
      "\n",
      "optim results\n",
      "time: 87\n",
      "minimum: 1317935.44235909\n",
      "1771317160.02209524-0.827706807009476-0.392765461918194-0.002190307128569950.3306530587287210.5652551374166440.9807189469671930.1299570171467320.1283543444925080.05850649397583360.09951808227379580.137574347975350.14176458926872\n",
      "386707662080951206.6473.77549529111456781403.2850.47549529132621890403.21011.9\n",
      "Calling iteration2:\n",
      "\n",
      "optim results\n",
      "time: 87\n",
      "minimum: 1316957.09962766\n",
      "1771316619.99527941-0.78626544290952-0.367420229087593-0.0006370160174882720.3056316482264590.5224807536374870.952365889728560.09978726653706810.08396544528174280.03848975697591660.06165457409479870.1295704211865240.13065828092136\n",
      "386707662080951206.6473.77549529111456781403.2850.47549529132621890403.21011.9\n",
      "Calling iteration3:\n",
      "\n",
      "optim results\n",
      "time: 88\n",
      "minimum: 1316583.69421309\n",
      "1771316343.06837213-0.697218808892574-0.332321007733934-0.0002733076346329850.2711848018125810.4635934807415130.9081504364727860.07630714651111630.0689688578162160.03155765601339190.04701236911614590.1312851879955830.134822107676665\n",
      "386707662080951206.6473.77549529111456781403.2850.47549529132621890403.21011.9\n",
      "Calling iteration4:\n",
      "\n",
      "optim results\n",
      "time: 69\n",
      "minimum: 1316343.50341102\n",
      "1771316107.6328503-0.646177912673207-0.300996870870526-0.0006201172200569270.2425552227015640.414650891984590.9175587966674250.08325940361875330.05728636462695350.02705534318896740.03967625346467610.1512325523985760.151235010468688\n",
      "386707662080951206.6473.77549529111456781403.2850.47549529132621890403.21011.9\n",
      "Calling iteration5:\n",
      "\n",
      "optim results\n",
      "time: 87\n",
      "minimum: 1316164.9432815\n",
      "1771315945.95691515-0.620774601429399-0.280439541031407-0.001053113329205680.2242718934194560.3833953341333310.902046999535480.09896700800210730.04860113535993460.0237972702876190.03415135727370360.1730884267357460.17309713340846\n",
      "386707662080951206.6473.77549529111456781403.2850.47549529132621890403.21011.9\n",
      "Calling iteration6:\n",
      "\n",
      "optim results\n",
      "time: 86\n",
      "minimum: 1316054.73380958\n",
      "1771315821.94965039-0.599245037755773-0.264978873497119-0.001561824955181660.2095463694526850.3582218846410680.9347338572988550.1054343069687950.04339709082487960.02145719984232650.02828314626107950.1853570238294650.186751291604532\n",
      "386707662080951206.6473.77549529111456781403.2850.47549529132621890403.21011.9\n",
      "Calling iteration7:\n",
      "\n",
      "optim results\n",
      "time: 87\n",
      "minimum: 1315958.14209685\n",
      "1771315717.25609314-0.567591645279808-0.255035571178123-0.001925729068594890.197535490181050.3376891509071490.8577258557595720.1124464944577460.03998233800050.01958712601565820.0216900541206840.1966454675010950.197782133500786\n",
      "EM algorithm done ...\n",
      "\n",
      "Computing posterior probabilities for all segments ...\n",
      "\n",
      "Total time:17minutes\n",
      "\n",
      "Adjusting segmented data for cellularity ... \n",
      "\n",
      "Cellularity sample1: 1\n",
      "\n",
      "Cellularity sample2: 1\n",
      "\n",
      "Cellularity sample3: 1\n",
      "\n",
      "Cellularity sample4: 1\n",
      "\n",
      "Cellularity sample5: 1\n",
      "\n",
      "Cellularity sample6: 1\n",
      "\n",
      "Cellularity sample7: 1\n",
      "\n",
      "Cellularity sample8: 1\n",
      "\n",
      "Cellularity sample9: 1\n",
      "\n",
      "Cellularity sample10: 1\n",
      "\n",
      "Cellularity sample11: 1\n",
      "\n",
      "Cellularity sample12: 1\n",
      "\n",
      "Cellularity sample13: 1\n",
      "\n",
      "Cellularity sample14: 1\n",
      "\n",
      "Cellularity sample15: 1\n",
      "\n",
      "Cellularity sample16: 1\n",
      "\n",
      "Cellularity sample17: 1\n",
      "\n",
      "Cellularity sample18: 1\n",
      "\n",
      "Cellularity sample19: 1\n",
      "\n",
      "Cellularity sample20: 1\n",
      "\n",
      "Cellularity sample21: 1\n",
      "\n",
      "Cellularity sample22: 1\n",
      "\n",
      "Cellularity sample23: 1\n",
      "\n",
      "Cellularity sample24: 1\n",
      "\n",
      "Cellularity sample25: 1\n",
      "\n",
      "Cellularity sample26: 1\n",
      "\n",
      "Cellularity sample27: 1\n",
      "\n",
      "Cellularity sample28: 1\n",
      "\n",
      "Cellularity sample29: 1\n",
      "\n",
      "Cellularity sample30: 1\n",
      "\n",
      "Cellularity sample31: 1\n",
      "\n",
      "Cellularity sample32: 1\n",
      "\n",
      "Cellularity sample33: 1\n",
      "\n",
      "Cellularity sample34: 1\n",
      "\n",
      "Cellularity sample35: 1\n",
      "\n",
      "Cellularity sample36: 1\n",
      "\n",
      "Cellularity sample37: 1\n",
      "\n",
      "Cellularity sample38: 1\n",
      "\n",
      "Cellularity sample39: 1\n",
      "\n",
      "Cellularity sample40: 1\n",
      "\n",
      "Cellularity sample41: 1\n",
      "\n",
      "Cellularity sample42: 1\n",
      "\n",
      "Cellularity sample43: 1\n",
      "\n",
      "Cellularity sample44: 1\n",
      "\n",
      "Cellularity sample45: 1\n",
      "\n",
      "Cellularity sample46: 1\n",
      "\n",
      "Cellularity sample47: 1\n",
      "\n",
      "Cellularity sample48: 1\n",
      "\n",
      "Cellularity sample49: 1\n",
      "\n",
      "Cellularity sample50: 1\n",
      "\n",
      "Cellularity sample51: 1\n",
      "\n",
      "Cellularity sample52: 1\n",
      "\n",
      "Cellularity sample53: 1\n",
      "\n",
      "Cellularity sample54: 1\n",
      "\n",
      "Cellularity sample55: 1\n",
      "\n",
      "Cellularity sample56: 1\n",
      "\n",
      "Cellularity sample57: 1\n",
      "\n",
      "Cellularity sample58: 1\n",
      "\n",
      "Cellularity sample59: 1\n",
      "\n",
      "Cellularity sample60: 1\n",
      "\n",
      "Cellularity sample61: 1\n",
      "\n",
      "Cellularity sample62: 1\n",
      "\n",
      "Cellularity sample63: 1\n",
      "\n",
      "Cellularity sample64: 1\n",
      "\n",
      "Cellularity sample65: 1\n",
      "\n",
      "Cellularity sample66: 1\n",
      "\n",
      "Cellularity sample67: 1\n",
      "\n",
      "Cellularity sample68: 1\n",
      "\n",
      "Cellularity sample69: 1\n",
      "\n",
      "Cellularity sample70: 1\n",
      "\n",
      "Cellularity sample71: 1\n",
      "\n",
      "Cellularity sample72: 1\n",
      "\n",
      "Cellularity sample73: 1\n",
      "\n",
      "Cellularity sample74: 1\n",
      "\n",
      "Cellularity sample75: 1\n",
      "\n",
      "Cellularity sample76: 1\n",
      "\n",
      "Cellularity sample77: 1\n",
      "\n",
      "Cellularity sample78: 1\n",
      "\n",
      "Cellularity sample79: 1\n",
      "\n",
      "Cellularity sample80: 1\n",
      "\n",
      "Cellularity sample81: 1\n",
      "\n",
      "Cellularity sample82: 1\n",
      "\n",
      "Cellularity sample83: 1\n",
      "\n",
      "Cellularity sample84: 1\n",
      "\n",
      "Cellularity sample85: 1\n",
      "\n",
      "Cellularity sample86: 1\n",
      "\n",
      "Cellularity sample87: 1\n",
      "\n",
      "Cellularity sample88: 1\n",
      "\n",
      "Cellularity sample89: 1\n",
      "\n",
      "Cellularity sample90: 1\n",
      "\n",
      "Cellularity sample91: 1\n",
      "\n",
      "Cellularity sample92: 1\n",
      "\n",
      "Cellularity sample93: 1\n",
      "\n",
      "Cellularity sample94: 1\n",
      "\n",
      "Cellularity sample95: 1\n",
      "\n",
      "Cellularity sample96: 1\n",
      "\n",
      "Cellularity sample97: 1\n",
      "\n",
      "Cellularity sample98: 1\n",
      "\n",
      "Cellularity sample99: 1\n",
      "\n",
      "Cellularity sample100: 1\n",
      "\n",
      "Cellularity sample101: 1\n",
      "\n",
      "Cellularity sample102: 1\n",
      "\n",
      "Cellularity sample103: 1\n",
      "\n",
      "Cellularity sample104: 1\n",
      "\n",
      "Cellularity sample105: 1\n",
      "\n",
      "Cellularity sample106: 1\n",
      "\n",
      "Cellularity sample107: 1\n",
      "\n",
      "Cellularity sample108: 1\n",
      "\n",
      "Cellularity sample109: 1\n",
      "\n",
      "Cellularity sample110: 1\n",
      "\n",
      "Cellularity sample111: 1\n",
      "\n",
      "Cellularity sample112: 1\n",
      "\n",
      "Cellularity sample113: 1\n",
      "\n",
      "Cellularity sample114: 1\n",
      "\n",
      "Cellularity sample115: 1\n",
      "\n",
      "Cellularity sample116: 1\n",
      "\n",
      "Cellularity sample117: 1\n",
      "\n",
      "Cellularity sample118: 1\n",
      "\n",
      "Cellularity sample119: 1\n",
      "\n",
      "Cellularity sample120: 1\n",
      "\n",
      "Cellularity sample121: 1\n",
      "\n",
      "Cellularity sample122: 1\n",
      "\n",
      "Cellularity sample123: 1\n",
      "\n",
      "Cellularity sample124: 1\n",
      "\n",
      "Cellularity sample125: 1\n",
      "\n",
      "Cellularity sample126: 1\n",
      "\n",
      "Cellularity sample127: 1\n",
      "\n",
      "Cellularity sample128: 1\n",
      "\n",
      "Cellularity sample129: 1\n",
      "\n",
      "Cellularity sample130: 1\n",
      "\n",
      "Cellularity sample131: 1\n",
      "\n",
      "Cellularity sample132: 1\n",
      "\n",
      "Cellularity sample133: 1\n",
      "\n",
      "Cellularity sample134: 1\n",
      "\n",
      "Cellularity sample135: 1\n",
      "\n",
      "Cellularity sample136: 1\n",
      "\n",
      "Cellularity sample137: 1\n",
      "\n",
      "Cellularity sample138: 1\n",
      "\n",
      "Cellularity sample139: 1\n",
      "\n",
      "Cellularity sample140: 1\n",
      "\n",
      "Cellularity sample141: 1\n",
      "\n",
      "Cellularity sample142: 1\n",
      "\n",
      "Cellularity sample143: 1\n",
      "\n",
      "Cellularity sample144: 1\n",
      "\n",
      "Cellularity sample145: 1\n",
      "\n",
      "Cellularity sample146: 1\n",
      "\n",
      "Cellularity sample147: 1\n",
      "\n",
      "Cellularity sample148: 1\n",
      "\n",
      "Cellularity sample149: 1\n",
      "\n",
      "Cellularity sample150: 1\n",
      "\n",
      "Cellularity sample151: 1\n",
      "\n",
      "Cellularity sample152: 1\n",
      "\n",
      "Cellularity sample153: 1\n",
      "\n",
      "Cellularity sample154: 1\n",
      "\n",
      "Cellularity sample155: 1\n",
      "\n",
      "Cellularity sample156: 1\n",
      "\n",
      "Cellularity sample157: 1\n",
      "\n",
      "Cellularity sample158: 1\n",
      "\n",
      "Cellularity sample159: 1\n",
      "\n",
      "Cellularity sample160: 1\n",
      "\n",
      "Cellularity sample161: 1\n",
      "\n",
      "Cellularity sample162: 1\n",
      "\n",
      "Cellularity sample163: 1\n",
      "\n",
      "Cellularity sample164: 1\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Cellularity sample165: 1\n",
      "\n",
      "Cellularity sample166: 1\n",
      "\n",
      "Cellularity sample167: 1\n",
      "\n",
      "Cellularity sample168: 1\n",
      "\n",
      "Cellularity sample169: 1\n",
      "\n",
      "Cellularity sample170: 1\n",
      "\n",
      "Cellularity sample171: 1\n",
      "\n",
      "Cellularity sample172: 1\n",
      "\n",
      "Cellularity sample173: 1\n",
      "\n",
      "Cellularity sample174: 1\n",
      "\n",
      "Cellularity sample175: 1\n",
      "\n",
      "Cellularity sample176: 1\n",
      "\n",
      "Cellularity sample177: 1\n",
      "\n",
      "Adjusting normalized data for cellularity ... \n",
      "\n",
      "Cellularity sample1: 1\n",
      "\n",
      "Cellularity sample2: 1\n",
      "\n",
      "Cellularity sample3: 1\n",
      "\n",
      "Cellularity sample4: 1\n",
      "\n",
      "Cellularity sample5: 1\n",
      "\n",
      "Cellularity sample6: 1\n",
      "\n",
      "Cellularity sample7: 1\n",
      "\n",
      "Cellularity sample8: 1\n",
      "\n",
      "Cellularity sample9: 1\n",
      "\n",
      "Cellularity sample10: 1\n",
      "\n",
      "Cellularity sample11: 1\n",
      "\n",
      "Cellularity sample12: 1\n",
      "\n",
      "Cellularity sample13: 1\n",
      "\n",
      "Cellularity sample14: 1\n",
      "\n",
      "Cellularity sample15: 1\n",
      "\n",
      "Cellularity sample16: 1\n",
      "\n",
      "Cellularity sample17: 1\n",
      "\n",
      "Cellularity sample18: 1\n",
      "\n",
      "Cellularity sample19: 1\n",
      "\n",
      "Cellularity sample20: 1\n",
      "\n",
      "Cellularity sample21: 1\n",
      "\n",
      "Cellularity sample22: 1\n",
      "\n",
      "Cellularity sample23: 1\n",
      "\n",
      "Cellularity sample24: 1\n",
      "\n",
      "Cellularity sample25: 1\n",
      "\n",
      "Cellularity sample26: 1\n",
      "\n",
      "Cellularity sample27: 1\n",
      "\n",
      "Cellularity sample28: 1\n",
      "\n",
      "Cellularity sample29: 1\n",
      "\n",
      "Cellularity sample30: 1\n",
      "\n",
      "Cellularity sample31: 1\n",
      "\n",
      "Cellularity sample32: 1\n",
      "\n",
      "Cellularity sample33: 1\n",
      "\n",
      "Cellularity sample34: 1\n",
      "\n",
      "Cellularity sample35: 1\n",
      "\n",
      "Cellularity sample36: 1\n",
      "\n",
      "Cellularity sample37: 1\n",
      "\n",
      "Cellularity sample38: 1\n",
      "\n",
      "Cellularity sample39: 1\n",
      "\n",
      "Cellularity sample40: 1\n",
      "\n",
      "Cellularity sample41: 1\n",
      "\n",
      "Cellularity sample42: 1\n",
      "\n",
      "Cellularity sample43: 1\n",
      "\n",
      "Cellularity sample44: 1\n",
      "\n",
      "Cellularity sample45: 1\n",
      "\n",
      "Cellularity sample46: 1\n",
      "\n",
      "Cellularity sample47: 1\n",
      "\n",
      "Cellularity sample48: 1\n",
      "\n",
      "Cellularity sample49: 1\n",
      "\n",
      "Cellularity sample50: 1\n",
      "\n",
      "Cellularity sample51: 1\n",
      "\n",
      "Cellularity sample52: 1\n",
      "\n",
      "Cellularity sample53: 1\n",
      "\n",
      "Cellularity sample54: 1\n",
      "\n",
      "Cellularity sample55: 1\n",
      "\n",
      "Cellularity sample56: 1\n",
      "\n",
      "Cellularity sample57: 1\n",
      "\n",
      "Cellularity sample58: 1\n",
      "\n",
      "Cellularity sample59: 1\n",
      "\n",
      "Cellularity sample60: 1\n",
      "\n",
      "Cellularity sample61: 1\n",
      "\n",
      "Cellularity sample62: 1\n",
      "\n",
      "Cellularity sample63: 1\n",
      "\n",
      "Cellularity sample64: 1\n",
      "\n",
      "Cellularity sample65: 1\n",
      "\n",
      "Cellularity sample66: 1\n",
      "\n",
      "Cellularity sample67: 1\n",
      "\n",
      "Cellularity sample68: 1\n",
      "\n",
      "Cellularity sample69: 1\n",
      "\n",
      "Cellularity sample70: 1\n",
      "\n",
      "Cellularity sample71: 1\n",
      "\n",
      "Cellularity sample72: 1\n",
      "\n",
      "Cellularity sample73: 1\n",
      "\n",
      "Cellularity sample74: 1\n",
      "\n",
      "Cellularity sample75: 1\n",
      "\n",
      "Cellularity sample76: 1\n",
      "\n",
      "Cellularity sample77: 1\n",
      "\n",
      "Cellularity sample78: 1\n",
      "\n",
      "Cellularity sample79: 1\n",
      "\n",
      "Cellularity sample80: 1\n",
      "\n",
      "Cellularity sample81: 1\n",
      "\n",
      "Cellularity sample82: 1\n",
      "\n",
      "Cellularity sample83: 1\n",
      "\n",
      "Cellularity sample84: 1\n",
      "\n",
      "Cellularity sample85: 1\n",
      "\n",
      "Cellularity sample86: 1\n",
      "\n",
      "Cellularity sample87: 1\n",
      "\n",
      "Cellularity sample88: 1\n",
      "\n",
      "Cellularity sample89: 1\n",
      "\n",
      "Cellularity sample90: 1\n",
      "\n",
      "Cellularity sample91: 1\n",
      "\n",
      "Cellularity sample92: 1\n",
      "\n",
      "Cellularity sample93: 1\n",
      "\n",
      "Cellularity sample94: 1\n",
      "\n",
      "Cellularity sample95: 1\n",
      "\n",
      "Cellularity sample96: 1\n",
      "\n",
      "Cellularity sample97: 1\n",
      "\n",
      "Cellularity sample98: 1\n",
      "\n",
      "Cellularity sample99: 1\n",
      "\n",
      "Cellularity sample100: 1\n",
      "\n",
      "Cellularity sample101: 1\n",
      "\n",
      "Cellularity sample102: 1\n",
      "\n",
      "Cellularity sample103: 1\n",
      "\n",
      "Cellularity sample104: 1\n",
      "\n",
      "Cellularity sample105: 1\n",
      "\n",
      "Cellularity sample106: 1\n",
      "\n",
      "Cellularity sample107: 1\n",
      "\n",
      "Cellularity sample108: 1\n",
      "\n",
      "Cellularity sample109: 1\n",
      "\n",
      "Cellularity sample110: 1\n",
      "\n",
      "Cellularity sample111: 1\n",
      "\n",
      "Cellularity sample112: 1\n",
      "\n",
      "Cellularity sample113: 1\n",
      "\n",
      "Cellularity sample114: 1\n",
      "\n",
      "Cellularity sample115: 1\n",
      "\n",
      "Cellularity sample116: 1\n",
      "\n",
      "Cellularity sample117: 1\n",
      "\n",
      "Cellularity sample118: 1\n",
      "\n",
      "Cellularity sample119: 1\n",
      "\n",
      "Cellularity sample120: 1\n",
      "\n",
      "Cellularity sample121: 1\n",
      "\n",
      "Cellularity sample122: 1\n",
      "\n",
      "Cellularity sample123: 1\n",
      "\n",
      "Cellularity sample124: 1\n",
      "\n",
      "Cellularity sample125: 1\n",
      "\n",
      "Cellularity sample126: 1\n",
      "\n",
      "Cellularity sample127: 1\n",
      "\n",
      "Cellularity sample128: 1\n",
      "\n",
      "Cellularity sample129: 1\n",
      "\n",
      "Cellularity sample130: 1\n",
      "\n",
      "Cellularity sample131: 1\n",
      "\n",
      "Cellularity sample132: 1\n",
      "\n",
      "Cellularity sample133: 1\n",
      "\n",
      "Cellularity sample134: 1\n",
      "\n",
      "Cellularity sample135: 1\n",
      "\n",
      "Cellularity sample136: 1\n",
      "\n",
      "Cellularity sample137: 1\n",
      "\n",
      "Cellularity sample138: 1\n",
      "\n",
      "Cellularity sample139: 1\n",
      "\n",
      "Cellularity sample140: 1\n",
      "\n",
      "Cellularity sample141: 1\n",
      "\n",
      "Cellularity sample142: 1\n",
      "\n",
      "Cellularity sample143: 1\n",
      "\n",
      "Cellularity sample144: 1\n",
      "\n",
      "Cellularity sample145: 1\n",
      "\n",
      "Cellularity sample146: 1\n",
      "\n",
      "Cellularity sample147: 1\n",
      "\n",
      "Cellularity sample148: 1\n",
      "\n",
      "Cellularity sample149: 1\n",
      "\n",
      "Cellularity sample150: 1\n",
      "\n",
      "Cellularity sample151: 1\n",
      "\n",
      "Cellularity sample152: 1\n",
      "\n",
      "Cellularity sample153: 1\n",
      "\n",
      "Cellularity sample154: 1\n",
      "\n",
      "Cellularity sample155: 1\n",
      "\n",
      "Cellularity sample156: 1\n",
      "\n",
      "Cellularity sample157: 1\n",
      "\n",
      "Cellularity sample158: 1\n",
      "\n",
      "Cellularity sample159: 1\n",
      "\n",
      "Cellularity sample160: 1\n",
      "\n",
      "Cellularity sample161: 1\n",
      "\n",
      "Cellularity sample162: 1\n",
      "\n",
      "Cellularity sample163: 1\n",
      "\n",
      "Cellularity sample164: 1\n",
      "\n",
      "Cellularity sample165: 1\n",
      "\n",
      "Cellularity sample166: 1\n",
      "\n",
      "Cellularity sample167: 1\n",
      "\n",
      "Cellularity sample168: 1\n",
      "\n",
      "Cellularity sample169: 1\n",
      "\n",
      "Cellularity sample170: 1\n",
      "\n",
      "Cellularity sample171: 1\n",
      "\n",
      "Cellularity sample172: 1\n",
      "\n",
      "Cellularity sample173: 1\n",
      "\n",
      "Cellularity sample174: 1\n",
      "\n",
      "Cellularity sample175: 1\n",
      "\n",
      "Cellularity sample176: 1\n",
      "\n",
      "Cellularity sample177: 1\n",
      "\n",
      "1\n",
      "386525969632382206.5531.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527769758125206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386527569758122206.5532.37549529111456781403.2850.47549529132621890403.21011.9\n",
      "386530271040622206.55427549529111456781403.2850.47549529132621890403.21011.9\n",
      "386536671040945206.55427549529111456781403.2850.47549529132621890403.21011.9\n",
      "386537571040949206.55427549529111456781403.2850.47549529132621890403.21011.9\n",
      "386538471040953206.55427549529111456781403.2850.47549529132621890403.21011.9\n",
      "386539371040957206.55427549529111456781403.2850.47549529132621890403.21011.9\n",
      "386540271040961206.55427549529111456781403.2850.47549529132621890403.21011.9\n",
      "386540871040964206.55427549529111456781403.2850.47549529132621890403.21011.9\n",
      "386542971041002206.55427549529111456781403.2850.47549529132621890403.21011.9\n",
      "386585271041372206.5542.17549529111456781403.2850.47549529132621890403.21011.9\n",
      "2\n",
      "386587078837081206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587278837084206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587078837081206.5601.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587379944585206.56107549529133828137403.21021.17549529132621890403.21011.9\n",
      "386587379944585206.56107549529133828137403.21021.17549529132621890403.21011.9\n",
      "386588279944589206.56107549529133828137403.21021.17549529132621890403.21011.9\n",
      "386589179944593206.56107549529133828137403.21021.17549529132621890403.21011.9\n",
      "386590079944597206.56107549529133828137403.21021.17549529132621890403.21011.9\n",
      "386590979944601206.56107549529133828137403.21021.17549529132621890403.21011.9\n",
      "386591579944604206.56107549529133828137403.21021.17549529132621890403.21011.9\n",
      "386593679944642206.56107549529133828137403.21021.17549529132621890403.21011.9\n",
      "387008783627181206.7638.17549529133828137403.21021.17549529132621890403.21011.9\n",
      "3\n",
      "387009389184757206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009589184760206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009389184757206.7680.57549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009690290191206.7688.97549529133828137403.21021.17549529132621890403.21011.9\n",
      "387009189158531206.7680.37549529133828137403.21021.17549529132621890403.21011.9\n",
      "387010089158535206.7680.37549529133828137403.21021.17549529132621890403.21011.9\n",
      "387010989158539206.7680.37549529133828137403.21021.17549529132621890403.21011.9\n",
      "387011889158543206.7680.37549529133828137403.21021.17549529132621890403.21011.9\n",
      "387012789158547206.7680.37549529133828137403.21021.17549529132621890403.21011.9\n",
      "387013389158550206.7680.37549529133828137403.21021.17549529132621890403.21011.9\n",
      "387015489158588206.7680.37549529133828137403.21021.17549529132621890403.21011.9\n",
      "387008792478374206.7705.67549529160673764403.21225.97549529132621890403.21011.9\n",
      "4\n",
      "387009398211990206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009598211993206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009398211990206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009699343628206.77587549529160673764403.21225.97549529132621890403.21011.9\n",
      "387009198211973206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387010098211977206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387010998211981206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387011898211985206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387012798211989206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387013398211992206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "387015498212030206.7749.37549529160673764403.21225.97549529132621890403.21011.9\n",
      "3870087101607269206.7775.37549529160673764403.21225.97549529160305308403.21223.1\n",
      "FINISHED!\n",
      "\n",
      "Total time:1minutes\n",
      "\n",
      "  |.....................................                                 |  53%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.......................................                               |  56%\n",
      "label: freqPlots (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "Plotting sample 6322_1_T20-60638_I-1 (4,453,465 reads) (1 of 177) ...\n",
      "Plotting sample 6322_10_T20-60664_I-1 (3,409,233 reads) (2 of 177) ...\n",
      "Plotting sample 6322_100_T20-60765_I-1 (4,154,471 reads) (3 of 177) ...\n",
      "Plotting sample 6322_101_T20-60767_I-1 (4,478,697 reads) (4 of 177) ...\n",
      "Plotting sample 6322_102_T20-60768_I-1 (4,578,519 reads) (5 of 177) ...\n",
      "Plotting sample 6322_103_T20-60769_I-1 (4,131,044 reads) (6 of 177) ...\n",
      "Plotting sample 6322_104_T20-60770_I-1 (4,815,261 reads) (7 of 177) ...\n",
      "Plotting sample 6322_11_T20-60669_I-1 (5,254,711 reads) (8 of 177) ...\n",
      "Plotting sample 6322_12_T20-60672_I-1 (4,221,638 reads) (9 of 177) ...\n",
      "Plotting sample 6322_13_T20-60676_I-1 (4,107,860 reads) (10 of 177) ...\n",
      "Plotting sample 6322_14_T20-60680_I-1 (5,089,680 reads) (11 of 177) ...\n",
      "Plotting sample 6322_15_T20-60652_I-1 (5,106,257 reads) (12 of 177) ...\n",
      "Plotting sample 6322_16_T20-60654_I-1 (4,093,105 reads) (13 of 177) ...\n",
      "Plotting sample 6322_17_T20-60656_I-1 (3,330,248 reads) (14 of 177) ...\n",
      "Plotting sample 6322_18_T20-60657_I-1 (5,791,307 reads) (15 of 177) ...\n",
      "Plotting sample 6322_19_T20-60658_I-1 (5,248,100 reads) (16 of 177) ...\n",
      "Plotting sample 6322_2_T20-60641_I-1 (3,983,862 reads) (17 of 177) ...\n",
      "Plotting sample 6322_20_T20-60661_I-1 (5,425,916 reads) (18 of 177) ...\n",
      "Plotting sample 6322_21_T20-60662_I-1 (5,078,697 reads) (19 of 177) ...\n",
      "Plotting sample 6322_22_T20-60663_I-1 (4,980,715 reads) (20 of 177) ...\n",
      "Plotting sample 6322_23_T20-60666_I-1 (3,377,549 reads) (21 of 177) ...\n",
      "Plotting sample 6322_24_T20-60667_I-1 (4,180,452 reads) (22 of 177) ...\n",
      "Plotting sample 6322_25_T20-60668_I-1 (3,388,804 reads) (23 of 177) ...\n",
      "Plotting sample 6322_26_T20-60702_I-1 (4,143,796 reads) (24 of 177) ...\n",
      "Plotting sample 6322_27_T20-60704_I-1 (6,222,492 reads) (25 of 177) ...\n",
      "Plotting sample 6322_28_T20-60706_I-1 (3,894,082 reads) (26 of 177) ...\n",
      "Plotting sample 6322_29_T20-60708_I-1 (2,915,859 reads) (27 of 177) ...\n",
      "Plotting sample 6322_3_T20-60644_I-1 (3,620,091 reads) (28 of 177) ...\n",
      "Plotting sample 6322_30_T20-60718_I-1 (3,936,392 reads) (29 of 177) ...\n",
      "Plotting sample 6322_31_T20-60671_I-1 (4,355,710 reads) (30 of 177) ...\n",
      "Plotting sample 6322_33_T20-60674_I-1 (3,064,508 reads) (31 of 177) ...\n",
      "Plotting sample 6322_34_T20-60675_I-1 (5,641,231 reads) (32 of 177) ...\n",
      "Plotting sample 6322_35_T20-60677_I-1 (4,520,992 reads) (33 of 177) ...\n",
      "Plotting sample 6322_36_T20-60678_I-1 (4,513,888 reads) (34 of 177) ...\n",
      "Plotting sample 6322_38_T20-60686_I-1 (3,123,633 reads) (35 of 177) ...\n",
      "Plotting sample 6322_39_T20-60688_I-1 (4,183,356 reads) (36 of 177) ...\n",
      "Plotting sample 6322_4_T20-60645_I-1 (3,520,188 reads) (37 of 177) ...\n",
      "Plotting sample 6322_40_T20-60690_I-1 (5,242,536 reads) (38 of 177) ...\n",
      "Plotting sample 6322_43_T20-60694_I-1 (5,347,901 reads) (39 of 177) ...\n",
      "Plotting sample 6322_44_T20-60697_I-1 (4,545,050 reads) (40 of 177) ...\n",
      "Plotting sample 6322_45_T20-60703_I-1 (5,944,356 reads) (41 of 177) ...\n",
      "Plotting sample 6322_46_T20-60705_I-1 (4,931,917 reads) (42 of 177) ...\n",
      "Plotting sample 6322_48_T20-60710_I-1 (4,942,105 reads) (43 of 177) ...\n",
      "Plotting sample 6322_49_T20-60711_I-1 (3,715,599 reads) (44 of 177) ...\n",
      "Plotting sample 6322_5_T20-60647_I-1 (3,865,658 reads) (45 of 177) ...\n",
      "Plotting sample 6322_50_T20-60714_I-1 (4,398,207 reads) (46 of 177) ...\n",
      "Plotting sample 6322_51_T20-60715_I-1 (5,689,785 reads) (47 of 177) ...\n",
      "Plotting sample 6322_52_T20-60717_I-1 (4,775,544 reads) (48 of 177) ...\n",
      "Plotting sample 6322_53_T20-60723_I-1 (4,959,685 reads) (49 of 177) ...\n",
      "Plotting sample 6322_54_T20-60724_I-1 (5,867,399 reads) (50 of 177) ...\n",
      "Plotting sample 6322_55_T20-60709_I-1 (4,293,755 reads) (51 of 177) ...\n",
      "Plotting sample 6322_56_T20-60712_I-1 (3,028,113 reads) (52 of 177) ...\n",
      "Plotting sample 6322_57_T20-60713_I-1 (3,082,543 reads) (53 of 177) ...\n",
      "Plotting sample 6322_58_T20-60719_I-1 (4,418,926 reads) (54 of 177) ...\n",
      "Plotting sample 6322_59_T20-60720_I-1 (4,878,704 reads) (55 of 177) ...\n",
      "Plotting sample 6322_6_T20-60653_I-1 (3,577,353 reads) (56 of 177) ...\n",
      "Plotting sample 6322_60_T20-60721_I-1 (4,772,046 reads) (57 of 177) ...\n",
      "Plotting sample 6322_61_T20-60722_I-1 (4,590,370 reads) (58 of 177) ...\n",
      "Plotting sample 6322_62_T20-60728_I-1 (5,530,619 reads) (59 of 177) ...\n",
      "Plotting sample 6322_63_T20-60734_I-1 (5,393,294 reads) (60 of 177) ...\n",
      "Plotting sample 6322_64_T20-60743_I-1 (4,374,959 reads) (61 of 177) ...\n",
      "Plotting sample 6322_65_T20-60745_I-1 (3,532,716 reads) (62 of 177) ...\n",
      "Plotting sample 6322_66_T20-60746_I-1 (5,897,662 reads) (63 of 177) ...\n",
      "Plotting sample 6322_67_T20-60755_I-1 (4,654,593 reads) (64 of 177) ...\n",
      "Plotting sample 6322_68_T20-60756_I-1 (4,521,287 reads) (65 of 177) ...\n",
      "Plotting sample 6322_69_T20-60760_I-1 (5,386,483 reads) (66 of 177) ...\n",
      "Plotting sample 6322_7_T20-60655_I-1 (3,574,988 reads) (67 of 177) ...\n",
      "Plotting sample 6322_70_T20-60725_I-1 (5,003,086 reads) (68 of 177) ...\n",
      "Plotting sample 6322_71_T20-60726_I-1 (4,729,073 reads) (69 of 177) ...\n",
      "Plotting sample 6322_72_T20-60727_I-1 (4,893,795 reads) (70 of 177) ...\n",
      "Plotting sample 6322_73_T20-60729_I-1 (5,911,980 reads) (71 of 177) ...\n",
      "Plotting sample 6322_74_T20-60733_I-1 (5,831,972 reads) (72 of 177) ...\n",
      "Plotting sample 6322_75_T20-60738_I-1 (5,634,065 reads) (73 of 177) ...\n",
      "Plotting sample 6322_76_T20-60747_I-1 (6,563,253 reads) (74 of 177) ...\n",
      "Plotting sample 6322_77_T20-60748_I-1 (6,506,648 reads) (75 of 177) ...\n",
      "Plotting sample 6322_78_T20-60750_I-1 (5,816,014 reads) (76 of 177) ...\n",
      "Plotting sample 6322_79_T20-60752_I-1 (3,055,244 reads) (77 of 177) ...\n",
      "Plotting sample 6322_8_T20-60659_I-1 (3,707,253 reads) (78 of 177) ...\n",
      "Plotting sample 6322_80_T20-60753_I-1 (4,011,278 reads) (79 of 177) ...\n",
      "Plotting sample 6322_81_T20-60754_I-1 (4,810,508 reads) (80 of 177) ...\n",
      "Plotting sample 6322_82_T20-60757_I-1 (3,984,778 reads) (81 of 177) ...\n",
      "Plotting sample 6322_83_T20-60758_I-1 (4,933,128 reads) (82 of 177) ...\n",
      "Plotting sample 6322_84_T20-60759_I-1 (5,391,429 reads) (83 of 177) ...\n",
      "Plotting sample 6322_85_T20-60764_I-1 (4,197,463 reads) (84 of 177) ...\n",
      "Plotting sample 6322_86_T20-60665_I-1 (4,795,740 reads) (85 of 177) ...\n",
      "Plotting sample 6322_87_T20-60665_I-2 (4,503,411 reads) (86 of 177) ...\n",
      "Plotting sample 6322_88_T20-60700_I-1 (4,829,437 reads) (87 of 177) ...\n",
      "Plotting sample 6322_89_T20-60744_I-1 (5,085,385 reads) (88 of 177) ...\n",
      "Plotting sample 6322_9_T20-60660_I-1 (4,292,273 reads) (89 of 177) ...\n",
      "Plotting sample 6322_90_T20-60744_I-2 (3,854,691 reads) (90 of 177) ...\n",
      "Plotting sample 6322_91_T20-60761_I-1 (4,713,473 reads) (91 of 177) ...\n",
      "Plotting sample 6322_92_T20-60762_I-1 (3,712,206 reads) (92 of 177) ...\n",
      "Plotting sample 6322_93_T20-60763_I-1 (4,363,865 reads) (93 of 177) ...\n",
      "Plotting sample 6322_94_T20-60766_I-1 (3,907,462 reads) (94 of 177) ...\n",
      "Plotting sample 6322_95_T20-60444_I-1 (2,170,310 reads) (95 of 177) ...\n",
      "Plotting sample 6322_96_T20-60453_I-1 (4,149,974 reads) (96 of 177) ...\n",
      "Plotting sample 6322_97_T20-60598_I-1 (4,218,808 reads) (97 of 177) ...\n",
      "Plotting sample 6322_98_T20-60598_I-2 (5,633,444 reads) (98 of 177) ...\n",
      "Plotting sample 6322_99_T20-60626_I-1 (3,464,626 reads) (99 of 177) ...\n",
      "Plotting sample RUN1097_6309_1_T20-60470_I-1_TAACTTGGTC-GTCGTGAATC_S2 (8,547,119 reads) (100 of 177) ...\n",
      "Plotting sample RUN1097_6309_10_T20-60558_I-1_TCCAACTGAA-TGTCCGTAGG_S11 (6,460,784 reads) (101 of 177) ...\n",
      "Plotting sample RUN1097_6309_11_T20-60564_I-1_ATTCGGCTTA-TTAACGTCCG_S17 (6,259,555 reads) (102 of 177) ...\n",
      "Plotting sample RUN1097_6309_12_T20-60570_I-1_TATGGCAAGC-AACAGCAATG_S18 (6,816,988 reads) (103 of 177) ...\n",
      "Plotting sample RUN1097_6309_13_T20-60572_I-1_CCTAAGCTGA-CGTCGCATCT_S19 (5,696,934 reads) (104 of 177) ...\n",
      "Plotting sample RUN1097_6309_14_T20-60580_I-1_GCCACCTAGT-TGCGATGTAT_S20 (8,013,181 reads) (105 of 177) ...\n",
      "Plotting sample RUN1097_6309_15_T20-60475_I-1_AGTATCCGAT-TATTGCGCTA_S16 (5,499,468 reads) (106 of 177) ...\n",
      "Plotting sample RUN1097_6309_16_T20-60476_I-1_CCTCGTTGGA-GACTGCGTTA_S23 (7,339,720 reads) (107 of 177) ...\n",
      "Plotting sample RUN1097_6309_17_T20-60596_I-1_CGACCTTAAT-GCTCTTAGGT_S24 (2,714,431 reads) (108 of 177) ...\n",
      "Plotting sample RUN1097_6309_18_T20-60597_I-1_GAACCGCATT-CATTATGCCT_S25 (5,582,413 reads) (109 of 177) ...\n",
      "Plotting sample RUN1097_6309_19_T20-60600_I-1_CGCGGTATCT-ATTACCGATC_S26 (3,386,410 reads) (110 of 177) ...\n",
      "Plotting sample RUN1097_6309_2_T20-60501_I-1_TCGAACACGA-GCGAGGTTCT_S5 (5,107,330 reads) (111 of 177) ...\n",
      "Plotting sample RUN1097_6309_20_T20-60603_I-1_ACTCCGAAGC-TCCGTGTTGC_S27 (5,526,299 reads) (112 of 177) ...\n",
      "Plotting sample RUN1097_6309_21_T20-60608_I-1_CGATGTCTTG-CCGAACGTTC_S22 (5,646,575 reads) (113 of 177) ...\n",
      "Plotting sample RUN1097_6309_22_T20-60609_I-1_CTCCGCCATA-GCGTTCAGCA_S30 (4,490,486 reads) (114 of 177) ...\n",
      "Plotting sample RUN1097_6309_23_T20-60611_I-1_AACGAAGATC-TGTGGTCTAA_S31 (5,692,110 reads) (115 of 177) ...\n",
      "Plotting sample RUN1097_6309_24_T20-60612_I-1_AATAGGTAGG-TGGCTTATGG_S32 (7,678,770 reads) (116 of 177) ...\n",
      "Plotting sample RUN1097_6309_25_T20-60613_I-1_ATGTTGCCAC-TAGAAGCCAT_S33 (5,129,410 reads) (117 of 177) ...\n",
      "Plotting sample RUN1097_6309_26_T20-60615_I-1_ACAACAGGCT-TCCACTGTTG_S34 (6,408,364 reads) (118 of 177) ...\n",
      "Plotting sample RUN1097_6309_27_T20-60616_I-1_TATCCGCGTC-GTTGCATCGT_S35 (5,983,595 reads) (119 of 177) ...\n",
      "Plotting sample RUN1097_6309_28_T20-60618_I-1_AATCTGCAGT-TGCGAAGTGA_S29 (6,091,415 reads) (120 of 177) ...\n",
      "Plotting sample RUN1097_6309_29_T20-60619_I-1_GAGATGAGAG-GAATGGCGTT_S38 (5,352,997 reads) (121 of 177) ...\n",
      "Plotting sample RUN1097_6309_3_T20-60502_I-1_TTACTTCGTG-AAGATCGGAT_S4 (4,434,309 reads) (122 of 177) ...\n",
      "Plotting sample RUN1097_6309_30_T20-60620_I-1_TGACACGATT-GCCTAGGAAG_S39 (4,650,206 reads) (123 of 177) ...\n",
      "Plotting sample RUN1097_6309_31_T20-60622_I-1_CCTCCTGTGA-CACGTCTACG_S43 (6,817,862 reads) (124 of 177) ...\n",
      "Plotting sample RUN1097_6309_32_T20-60623_I-1_TATGCGTAGC-TCAGAGGCAC_S44 (6,937,203 reads) (125 of 177) ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Plotting sample RUN1097_6309_33_T20-60628_I-1_CAGCAGGTCA-CACTTAGGTA_S42 (2,552,834 reads) (126 of 177) ...\n",
      "Plotting sample RUN1097_6309_34_T20-60681_I-1_TTACGCGCAT-CTTAACGACA_S41 (4,636,639 reads) (127 of 177) ...\n",
      "Plotting sample RUN1097_6309_35_T20-60682_I-1_AGGATGTCCA-ACTGGACGTT_S40 (5,506,862 reads) (128 of 177) ...\n",
      "Plotting sample RUN1097_6309_36_T20-60683_I-1_AGTTCAGGCG-CACAGCTATT_S37 (5,083,786 reads) (129 of 177) ...\n",
      "Plotting sample RUN1097_6309_37_T20-60685_I-1_TCACCATAAG-AAGTTCAGGA_S48 (5,259,450 reads) (130 of 177) ...\n",
      "Plotting sample RUN1097_6309_38_T20-60687_I-1_GTGCTCTAGG-GTTACCAACG_S49 (2,189,340 reads) (131 of 177) ...\n",
      "Plotting sample RUN1097_6309_39_T20-60689_I-1_TAGGCGAATG-AGTCCAGTGA_S50 (7,359,714 reads) (132 of 177) ...\n",
      "Plotting sample RUN1097_6309_4_T20-60504_I-1_ACGAGACGTC-CGGTCTCTCT_S8 (6,394,710 reads) (133 of 177) ...\n",
      "Plotting sample RUN1097_6309_40_T20-60691_I-1_CCAGTCGTCA-CGCTAGGCTA_S1 (5,649,008 reads) (134 of 177) ...\n",
      "Plotting sample RUN1097_6309_41_T20-60695_I-1_TCTAGCTGCG-ATAGGTAAGG_S3 (4,858,668 reads) (135 of 177) ...\n",
      "Plotting sample RUN1097_6309_42_T20-60696_I-1_TTCTCTTAGC-TCCTCGAGCT_S6 (5,083,669 reads) (136 of 177) ...\n",
      "Plotting sample RUN1097_6309_43_T20-60698_I-1_GCCAGCTATC-TTGTTCGTGT_S47 (5,602,435 reads) (137 of 177) ...\n",
      "Plotting sample RUN1097_6309_44_T20-60699_I-1_CGCCAAGGTA-GATAGTGTTC_S10 (6,664,843 reads) (138 of 177) ...\n",
      "Plotting sample RUN1097_6309_45_T20-60701_I-1_ACTTCCGGCA-GACCAATTGT_S46 (8,081,113 reads) (139 of 177) ...\n",
      "Plotting sample RUN1097_6309_46_T20-60437_I-1_GCAATCTGCT-ACGAACCAAG_S54 (8,354,592 reads) (140 of 177) ...\n",
      "Plotting sample RUN1097_6309_47_T20-60446_I-1_AGTAAGGAGA-ATGCTTGGTC_S15 (6,153,409 reads) (141 of 177) ...\n",
      "Plotting sample RUN1097_6309_48_T20-60455_I-1_AGAAGACCTA-AACCGTAACA_S55 (5,509,151 reads) (142 of 177) ...\n",
      "Plotting sample RUN1097_6309_49_T20-60456_I-1_AGATTAGCGT-CGCCTTATAT_S53 (4,707,722 reads) (143 of 177) ...\n",
      "Plotting sample RUN1097_6309_5_T20-60505_I-1_CATAAGCGAC-CCTCTTCAAG_S9 (4,817,992 reads) (144 of 177) ...\n",
      "Plotting sample RUN1097_6309_50_T20-60457_I-1_TCGCGACACT-CGAGTAATGT_S21 (5,396,764 reads) (145 of 177) ...\n",
      "Plotting sample RUN1097_6309_51_T20-60462_I-1_CTTACGAACT-ACGGACTGAG_S52 (2,768,714 reads) (146 of 177) ...\n",
      "Plotting sample RUN1097_6309_52_T20-60466_I-1_CCTGACATCT-TAGCTACCAG_S59 (4,098,151 reads) (147 of 177) ...\n",
      "Plotting sample RUN1097_6309_53_T20-60468_I-1_AAGCGCTTAG-TGTCTGTGAC_S28 (4,495,167 reads) (148 of 177) ...\n",
      "Plotting sample RUN1097_6309_54_T20-60469_I-1_CTCTATTGAG-CACGCAAGTC_S60 (4,482,081 reads) (149 of 177) ...\n",
      "Plotting sample RUN1097_6309_55_T20-60474_I-1_CCGAACAGAG-TGATACTCGC_S58 (5,155,580 reads) (150 of 177) ...\n",
      "Plotting sample RUN1097_6309_56_T20-60491_I-1_CCGTGAGTTC-CTTGGTATGA_S57 (4,846,446 reads) (151 of 177) ...\n",
      "Plotting sample RUN1097_6309_57_T20-60542_I-1_AGGTAACAAC-GCACGATGCT_S36 (5,127,893 reads) (152 of 177) ...\n",
      "Plotting sample RUN1097_6309_58_T20-60629_I-1_TGGAGATCGA-CAGCGGTATG_S64 (4,076,560 reads) (153 of 177) ...\n",
      "Plotting sample RUN1097_6309_59_T20-60634_I-1_CAAGACTCAA-CACCGTCATA_S65 (6,591,209 reads) (154 of 177) ...\n",
      "Plotting sample RUN1097_6309_6_T20-60527_I-1_AACTCTCTAC-ACGGCTTCTC_S7 (6,661,216 reads) (155 of 177) ...\n",
      "Plotting sample RUN1097_6309_60_T20-60635_I-1_CGAGCGTAGT-TGCTCACGCA_S63 (6,576,512 reads) (156 of 177) ...\n",
      "Plotting sample RUN1097_6309_61_T20-60636_I-1_AACAACACAG-GAACATTGGC_S62 (6,181,228 reads) (157 of 177) ...\n",
      "Plotting sample RUN1097_6309_62_T20-60639_I-1_CACTTACGGC-GCTGGAGAAG_S45 (5,773,201 reads) (158 of 177) ...\n",
      "Plotting sample RUN1097_6309_63_T20-60640_I-1_AGAACTAACG-TTATCGAGAG_S69 (5,292,172 reads) (159 of 177) ...\n",
      "Plotting sample RUN1097_6309_64_T20-60642_I-1_TCAATTCCGA-TGTAGATCCG_S68 (8,319,723 reads) (160 of 177) ...\n",
      "Plotting sample RUN1097_6309_65_T20-60643_I-1_CGCACTTCGT-AATATGCCAG_S51 (4,327,122 reads) (161 of 177) ...\n",
      "Plotting sample RUN1097_6309_66_T20-60646_I-1_CCAGACCGAT-GCTACTTGCA_S67 (5,101,116 reads) (162 of 177) ...\n",
      "Plotting sample RUN1097_6309_67_T20-60648_I-1_TCGACGGCAA-GTAGTTGTGT_S56 (5,158,845 reads) (163 of 177) ...\n",
      "Plotting sample RUN1097_6309_68_T20-60651_I-1_CTGGATTACT-AAGTGTGCTA_S70 (3,275,709 reads) (164 of 177) ...\n",
      "Plotting sample RUN1097_6309_69_T20-60561_I-1_AGACTAGATG-CGTAACACTA_S71 (308,503 reads) (165 of 177) ...\n",
      "Plotting sample RUN1097_6309_7_T20-60540_I-1_GTATGCCACA-CCGAGTCTTC_S12 (6,305,716 reads) (166 of 177) ...\n",
      "Plotting sample RUN1097_6309_70_T20-60599_I-1_TCAGCATCTA-CCAGTGTACG_S61 (5,063,809 reads) (167 of 177) ...\n",
      "Plotting sample RUN1097_6309_71_T20-60605_I-1_GAGCAGCTTG-TGCCAGTATT_S72 (4,577,653 reads) (168 of 177) ...\n",
      "Plotting sample RUN1097_6309_72_T20-60606_I-1_CGGAACACAA-AACTCTACCG_S66 (4,902,901 reads) (169 of 177) ...\n",
      "Plotting sample RUN1097_6309_73_T20-60610_I-1_CTTGCTATTC-TAGAGTCACG_S73 (5,355,983 reads) (170 of 177) ...\n",
      "Plotting sample RUN1097_6309_74_T20-60617_I-1_CCACAGCCTT-GCTATTGAAG_S74 (5,646,499 reads) (171 of 177) ...\n",
      "Plotting sample RUN1097_6309_75_T20-60621_I-1_TCAGTTCGAT-CTGAGTCCTT_S75 (7,014,474 reads) (172 of 177) ...\n",
      "Plotting sample RUN1097_6309_76_T20-60625_I-1_AGCAATTCAC-TCATCGTCTG_S76 (4,612,476 reads) (173 of 177) ...\n",
      "Plotting sample RUN1097_6309_77_T20-60627_I-1_CCTCTATAGA-GTACTTCCAA_S77 (5,962,826 reads) (174 of 177) ...\n",
      "Plotting sample RUN1097_6309_78_T20-60637_I-1_GTTGCATCAA-GTTGATGCGA_S78 (5,814,752 reads) (175 of 177) ...\n",
      "Plotting sample RUN1097_6309_8_T20-60541_I-1_GCCAACAGAC-CGTTCACGTT_S13 (5,817,910 reads) (176 of 177) ...\n",
      "Plotting sample RUN1097_6309_9_T20-60546_I-1_ATAAGCGGAG-CTGTTCTACG_S14 (5,054,885 reads) (177 of 177) ...\n",
      "Plotting sample 6322_1_T20-60638_I-1 (4,453,465 reads) (1 of 177) ...\n",
      "Plotting sample 6322_10_T20-60664_I-1 (3,409,233 reads) (2 of 177) ...\n",
      "Plotting sample 6322_100_T20-60765_I-1 (4,154,471 reads) (3 of 177) ...\n",
      "Plotting sample 6322_101_T20-60767_I-1 (4,478,697 reads) (4 of 177) ...\n",
      "Plotting sample 6322_102_T20-60768_I-1 (4,578,519 reads) (5 of 177) ...\n",
      "Plotting sample 6322_103_T20-60769_I-1 (4,131,044 reads) (6 of 177) ...\n",
      "Plotting sample 6322_104_T20-60770_I-1 (4,815,261 reads) (7 of 177) ...\n",
      "Plotting sample 6322_11_T20-60669_I-1 (5,254,711 reads) (8 of 177) ...\n",
      "Plotting sample 6322_12_T20-60672_I-1 (4,221,638 reads) (9 of 177) ...\n",
      "Plotting sample 6322_13_T20-60676_I-1 (4,107,860 reads) (10 of 177) ...\n",
      "Plotting sample 6322_14_T20-60680_I-1 (5,089,680 reads) (11 of 177) ...\n",
      "Plotting sample 6322_15_T20-60652_I-1 (5,106,257 reads) (12 of 177) ...\n",
      "Plotting sample 6322_16_T20-60654_I-1 (4,093,105 reads) (13 of 177) ...\n",
      "Plotting sample 6322_17_T20-60656_I-1 (3,330,248 reads) (14 of 177) ...\n",
      "Plotting sample 6322_18_T20-60657_I-1 (5,791,307 reads) (15 of 177) ...\n",
      "Plotting sample 6322_19_T20-60658_I-1 (5,248,100 reads) (16 of 177) ...\n",
      "Plotting sample 6322_2_T20-60641_I-1 (3,983,862 reads) (17 of 177) ...\n",
      "Plotting sample 6322_20_T20-60661_I-1 (5,425,916 reads) (18 of 177) ...\n",
      "Plotting sample 6322_21_T20-60662_I-1 (5,078,697 reads) (19 of 177) ...\n",
      "Plotting sample 6322_22_T20-60663_I-1 (4,980,715 reads) (20 of 177) ...\n",
      "Plotting sample 6322_23_T20-60666_I-1 (3,377,549 reads) (21 of 177) ...\n",
      "Plotting sample 6322_24_T20-60667_I-1 (4,180,452 reads) (22 of 177) ...\n",
      "Plotting sample 6322_25_T20-60668_I-1 (3,388,804 reads) (23 of 177) ...\n",
      "Plotting sample 6322_26_T20-60702_I-1 (4,143,796 reads) (24 of 177) ...\n",
      "Plotting sample 6322_27_T20-60704_I-1 (6,222,492 reads) (25 of 177) ...\n",
      "Plotting sample 6322_28_T20-60706_I-1 (3,894,082 reads) (26 of 177) ...\n",
      "Plotting sample 6322_29_T20-60708_I-1 (2,915,859 reads) (27 of 177) ...\n",
      "Plotting sample 6322_3_T20-60644_I-1 (3,620,091 reads) (28 of 177) ...\n",
      "Plotting sample 6322_30_T20-60718_I-1 (3,936,392 reads) (29 of 177) ...\n",
      "Plotting sample 6322_31_T20-60671_I-1 (4,355,710 reads) (30 of 177) ...\n",
      "Plotting sample 6322_33_T20-60674_I-1 (3,064,508 reads) (31 of 177) ...\n",
      "Plotting sample 6322_34_T20-60675_I-1 (5,641,231 reads) (32 of 177) ...\n",
      "Plotting sample 6322_35_T20-60677_I-1 (4,520,992 reads) (33 of 177) ...\n",
      "Plotting sample 6322_36_T20-60678_I-1 (4,513,888 reads) (34 of 177) ...\n",
      "Plotting sample 6322_38_T20-60686_I-1 (3,123,633 reads) (35 of 177) ...\n",
      "Plotting sample 6322_39_T20-60688_I-1 (4,183,356 reads) (36 of 177) ...\n",
      "Plotting sample 6322_4_T20-60645_I-1 (3,520,188 reads) (37 of 177) ...\n",
      "Plotting sample 6322_40_T20-60690_I-1 (5,242,536 reads) (38 of 177) ...\n",
      "Plotting sample 6322_43_T20-60694_I-1 (5,347,901 reads) (39 of 177) ...\n",
      "Plotting sample 6322_44_T20-60697_I-1 (4,545,050 reads) (40 of 177) ...\n",
      "Plotting sample 6322_45_T20-60703_I-1 (5,944,356 reads) (41 of 177) ...\n",
      "Plotting sample 6322_46_T20-60705_I-1 (4,931,917 reads) (42 of 177) ...\n",
      "Plotting sample 6322_48_T20-60710_I-1 (4,942,105 reads) (43 of 177) ...\n",
      "Plotting sample 6322_49_T20-60711_I-1 (3,715,599 reads) (44 of 177) ...\n",
      "Plotting sample 6322_5_T20-60647_I-1 (3,865,658 reads) (45 of 177) ...\n",
      "Plotting sample 6322_50_T20-60714_I-1 (4,398,207 reads) (46 of 177) ...\n",
      "Plotting sample 6322_51_T20-60715_I-1 (5,689,785 reads) (47 of 177) ...\n",
      "Plotting sample 6322_52_T20-60717_I-1 (4,775,544 reads) (48 of 177) ...\n",
      "Plotting sample 6322_53_T20-60723_I-1 (4,959,685 reads) (49 of 177) ...\n",
      "Plotting sample 6322_54_T20-60724_I-1 (5,867,399 reads) (50 of 177) ...\n",
      "Plotting sample 6322_55_T20-60709_I-1 (4,293,755 reads) (51 of 177) ...\n",
      "Plotting sample 6322_56_T20-60712_I-1 (3,028,113 reads) (52 of 177) ...\n",
      "Plotting sample 6322_57_T20-60713_I-1 (3,082,543 reads) (53 of 177) ...\n",
      "Plotting sample 6322_58_T20-60719_I-1 (4,418,926 reads) (54 of 177) ...\n",
      "Plotting sample 6322_59_T20-60720_I-1 (4,878,704 reads) (55 of 177) ...\n",
      "Plotting sample 6322_6_T20-60653_I-1 (3,577,353 reads) (56 of 177) ...\n",
      "Plotting sample 6322_60_T20-60721_I-1 (4,772,046 reads) (57 of 177) ...\n",
      "Plotting sample 6322_61_T20-60722_I-1 (4,590,370 reads) (58 of 177) ...\n",
      "Plotting sample 6322_62_T20-60728_I-1 (5,530,619 reads) (59 of 177) ...\n",
      "Plotting sample 6322_63_T20-60734_I-1 (5,393,294 reads) (60 of 177) ...\n",
      "Plotting sample 6322_64_T20-60743_I-1 (4,374,959 reads) (61 of 177) ...\n",
      "Plotting sample 6322_65_T20-60745_I-1 (3,532,716 reads) (62 of 177) ...\n",
      "Plotting sample 6322_66_T20-60746_I-1 (5,897,662 reads) (63 of 177) ...\n",
      "Plotting sample 6322_67_T20-60755_I-1 (4,654,593 reads) (64 of 177) ...\n",
      "Plotting sample 6322_68_T20-60756_I-1 (4,521,287 reads) (65 of 177) ...\n",
      "Plotting sample 6322_69_T20-60760_I-1 (5,386,483 reads) (66 of 177) ...\n",
      "Plotting sample 6322_7_T20-60655_I-1 (3,574,988 reads) (67 of 177) ...\n",
      "Plotting sample 6322_70_T20-60725_I-1 (5,003,086 reads) (68 of 177) ...\n",
      "Plotting sample 6322_71_T20-60726_I-1 (4,729,073 reads) (69 of 177) ...\n",
      "Plotting sample 6322_72_T20-60727_I-1 (4,893,795 reads) (70 of 177) ...\n",
      "Plotting sample 6322_73_T20-60729_I-1 (5,911,980 reads) (71 of 177) ...\n",
      "Plotting sample 6322_74_T20-60733_I-1 (5,831,972 reads) (72 of 177) ...\n",
      "Plotting sample 6322_75_T20-60738_I-1 (5,634,065 reads) (73 of 177) ...\n",
      "Plotting sample 6322_76_T20-60747_I-1 (6,563,253 reads) (74 of 177) ...\n",
      "Plotting sample 6322_77_T20-60748_I-1 (6,506,648 reads) (75 of 177) ...\n",
      "Plotting sample 6322_78_T20-60750_I-1 (5,816,014 reads) (76 of 177) ...\n",
      "Plotting sample 6322_79_T20-60752_I-1 (3,055,244 reads) (77 of 177) ...\n",
      "Plotting sample 6322_8_T20-60659_I-1 (3,707,253 reads) (78 of 177) ...\n",
      "Plotting sample 6322_80_T20-60753_I-1 (4,011,278 reads) (79 of 177) ...\n",
      "Plotting sample 6322_81_T20-60754_I-1 (4,810,508 reads) (80 of 177) ...\n",
      "Plotting sample 6322_82_T20-60757_I-1 (3,984,778 reads) (81 of 177) ...\n",
      "Plotting sample 6322_83_T20-60758_I-1 (4,933,128 reads) (82 of 177) ...\n",
      "Plotting sample 6322_84_T20-60759_I-1 (5,391,429 reads) (83 of 177) ...\n",
      "Plotting sample 6322_85_T20-60764_I-1 (4,197,463 reads) (84 of 177) ...\n",
      "Plotting sample 6322_86_T20-60665_I-1 (4,795,740 reads) (85 of 177) ...\n",
      "Plotting sample 6322_87_T20-60665_I-2 (4,503,411 reads) (86 of 177) ...\n",
      "Plotting sample 6322_88_T20-60700_I-1 (4,829,437 reads) (87 of 177) ...\n",
      "Plotting sample 6322_89_T20-60744_I-1 (5,085,385 reads) (88 of 177) ...\n",
      "Plotting sample 6322_9_T20-60660_I-1 (4,292,273 reads) (89 of 177) ...\n",
      "Plotting sample 6322_90_T20-60744_I-2 (3,854,691 reads) (90 of 177) ...\n",
      "Plotting sample 6322_91_T20-60761_I-1 (4,713,473 reads) (91 of 177) ...\n",
      "Plotting sample 6322_92_T20-60762_I-1 (3,712,206 reads) (92 of 177) ...\n",
      "Plotting sample 6322_93_T20-60763_I-1 (4,363,865 reads) (93 of 177) ...\n",
      "Plotting sample 6322_94_T20-60766_I-1 (3,907,462 reads) (94 of 177) ...\n",
      "Plotting sample 6322_95_T20-60444_I-1 (2,170,310 reads) (95 of 177) ...\n",
      "Plotting sample 6322_96_T20-60453_I-1 (4,149,974 reads) (96 of 177) ...\n",
      "Plotting sample 6322_97_T20-60598_I-1 (4,218,808 reads) (97 of 177) ...\n",
      "Plotting sample 6322_98_T20-60598_I-2 (5,633,444 reads) (98 of 177) ...\n",
      "Plotting sample 6322_99_T20-60626_I-1 (3,464,626 reads) (99 of 177) ...\n",
      "Plotting sample RUN1097_6309_1_T20-60470_I-1_TAACTTGGTC-GTCGTGAATC_S2 (8,547,119 reads) (100 of 177) ...\n",
      "Plotting sample RUN1097_6309_10_T20-60558_I-1_TCCAACTGAA-TGTCCGTAGG_S11 (6,460,784 reads) (101 of 177) ...\n",
      "Plotting sample RUN1097_6309_11_T20-60564_I-1_ATTCGGCTTA-TTAACGTCCG_S17 (6,259,555 reads) (102 of 177) ...\n",
      "Plotting sample RUN1097_6309_12_T20-60570_I-1_TATGGCAAGC-AACAGCAATG_S18 (6,816,988 reads) (103 of 177) ...\n",
      "Plotting sample RUN1097_6309_13_T20-60572_I-1_CCTAAGCTGA-CGTCGCATCT_S19 (5,696,934 reads) (104 of 177) ...\n",
      "Plotting sample RUN1097_6309_14_T20-60580_I-1_GCCACCTAGT-TGCGATGTAT_S20 (8,013,181 reads) (105 of 177) ...\n",
      "Plotting sample RUN1097_6309_15_T20-60475_I-1_AGTATCCGAT-TATTGCGCTA_S16 (5,499,468 reads) (106 of 177) ...\n",
      "Plotting sample RUN1097_6309_16_T20-60476_I-1_CCTCGTTGGA-GACTGCGTTA_S23 (7,339,720 reads) (107 of 177) ...\n",
      "Plotting sample RUN1097_6309_17_T20-60596_I-1_CGACCTTAAT-GCTCTTAGGT_S24 (2,714,431 reads) (108 of 177) ...\n",
      "Plotting sample RUN1097_6309_18_T20-60597_I-1_GAACCGCATT-CATTATGCCT_S25 (5,582,413 reads) (109 of 177) ...\n",
      "Plotting sample RUN1097_6309_19_T20-60600_I-1_CGCGGTATCT-ATTACCGATC_S26 (3,386,410 reads) (110 of 177) ...\n",
      "Plotting sample RUN1097_6309_2_T20-60501_I-1_TCGAACACGA-GCGAGGTTCT_S5 (5,107,330 reads) (111 of 177) ...\n",
      "Plotting sample RUN1097_6309_20_T20-60603_I-1_ACTCCGAAGC-TCCGTGTTGC_S27 (5,526,299 reads) (112 of 177) ...\n",
      "Plotting sample RUN1097_6309_21_T20-60608_I-1_CGATGTCTTG-CCGAACGTTC_S22 (5,646,575 reads) (113 of 177) ...\n",
      "Plotting sample RUN1097_6309_22_T20-60609_I-1_CTCCGCCATA-GCGTTCAGCA_S30 (4,490,486 reads) (114 of 177) ...\n",
      "Plotting sample RUN1097_6309_23_T20-60611_I-1_AACGAAGATC-TGTGGTCTAA_S31 (5,692,110 reads) (115 of 177) ...\n",
      "Plotting sample RUN1097_6309_24_T20-60612_I-1_AATAGGTAGG-TGGCTTATGG_S32 (7,678,770 reads) (116 of 177) ...\n",
      "Plotting sample RUN1097_6309_25_T20-60613_I-1_ATGTTGCCAC-TAGAAGCCAT_S33 (5,129,410 reads) (117 of 177) ...\n",
      "Plotting sample RUN1097_6309_26_T20-60615_I-1_ACAACAGGCT-TCCACTGTTG_S34 (6,408,364 reads) (118 of 177) ...\n",
      "Plotting sample RUN1097_6309_27_T20-60616_I-1_TATCCGCGTC-GTTGCATCGT_S35 (5,983,595 reads) (119 of 177) ...\n",
      "Plotting sample RUN1097_6309_28_T20-60618_I-1_AATCTGCAGT-TGCGAAGTGA_S29 (6,091,415 reads) (120 of 177) ...\n",
      "Plotting sample RUN1097_6309_29_T20-60619_I-1_GAGATGAGAG-GAATGGCGTT_S38 (5,352,997 reads) (121 of 177) ...\n",
      "Plotting sample RUN1097_6309_3_T20-60502_I-1_TTACTTCGTG-AAGATCGGAT_S4 (4,434,309 reads) (122 of 177) ...\n",
      "Plotting sample RUN1097_6309_30_T20-60620_I-1_TGACACGATT-GCCTAGGAAG_S39 (4,650,206 reads) (123 of 177) ...\n",
      "Plotting sample RUN1097_6309_31_T20-60622_I-1_CCTCCTGTGA-CACGTCTACG_S43 (6,817,862 reads) (124 of 177) ...\n",
      "Plotting sample RUN1097_6309_32_T20-60623_I-1_TATGCGTAGC-TCAGAGGCAC_S44 (6,937,203 reads) (125 of 177) ...\n",
      "Plotting sample RUN1097_6309_33_T20-60628_I-1_CAGCAGGTCA-CACTTAGGTA_S42 (2,552,834 reads) (126 of 177) ...\n",
      "Plotting sample RUN1097_6309_34_T20-60681_I-1_TTACGCGCAT-CTTAACGACA_S41 (4,636,639 reads) (127 of 177) ...\n",
      "Plotting sample RUN1097_6309_35_T20-60682_I-1_AGGATGTCCA-ACTGGACGTT_S40 (5,506,862 reads) (128 of 177) ...\n",
      "Plotting sample RUN1097_6309_36_T20-60683_I-1_AGTTCAGGCG-CACAGCTATT_S37 (5,083,786 reads) (129 of 177) ...\n",
      "Plotting sample RUN1097_6309_37_T20-60685_I-1_TCACCATAAG-AAGTTCAGGA_S48 (5,259,450 reads) (130 of 177) ...\n",
      "Plotting sample RUN1097_6309_38_T20-60687_I-1_GTGCTCTAGG-GTTACCAACG_S49 (2,189,340 reads) (131 of 177) ...\n",
      "Plotting sample RUN1097_6309_39_T20-60689_I-1_TAGGCGAATG-AGTCCAGTGA_S50 (7,359,714 reads) (132 of 177) ...\n",
      "Plotting sample RUN1097_6309_4_T20-60504_I-1_ACGAGACGTC-CGGTCTCTCT_S8 (6,394,710 reads) (133 of 177) ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Plotting sample RUN1097_6309_40_T20-60691_I-1_CCAGTCGTCA-CGCTAGGCTA_S1 (5,649,008 reads) (134 of 177) ...\n",
      "Plotting sample RUN1097_6309_41_T20-60695_I-1_TCTAGCTGCG-ATAGGTAAGG_S3 (4,858,668 reads) (135 of 177) ...\n",
      "Plotting sample RUN1097_6309_42_T20-60696_I-1_TTCTCTTAGC-TCCTCGAGCT_S6 (5,083,669 reads) (136 of 177) ...\n",
      "Plotting sample RUN1097_6309_43_T20-60698_I-1_GCCAGCTATC-TTGTTCGTGT_S47 (5,602,435 reads) (137 of 177) ...\n",
      "Plotting sample RUN1097_6309_44_T20-60699_I-1_CGCCAAGGTA-GATAGTGTTC_S10 (6,664,843 reads) (138 of 177) ...\n",
      "Plotting sample RUN1097_6309_45_T20-60701_I-1_ACTTCCGGCA-GACCAATTGT_S46 (8,081,113 reads) (139 of 177) ...\n",
      "Plotting sample RUN1097_6309_46_T20-60437_I-1_GCAATCTGCT-ACGAACCAAG_S54 (8,354,592 reads) (140 of 177) ...\n",
      "Plotting sample RUN1097_6309_47_T20-60446_I-1_AGTAAGGAGA-ATGCTTGGTC_S15 (6,153,409 reads) (141 of 177) ...\n",
      "Plotting sample RUN1097_6309_48_T20-60455_I-1_AGAAGACCTA-AACCGTAACA_S55 (5,509,151 reads) (142 of 177) ...\n",
      "Plotting sample RUN1097_6309_49_T20-60456_I-1_AGATTAGCGT-CGCCTTATAT_S53 (4,707,722 reads) (143 of 177) ...\n",
      "Plotting sample RUN1097_6309_5_T20-60505_I-1_CATAAGCGAC-CCTCTTCAAG_S9 (4,817,992 reads) (144 of 177) ...\n",
      "Plotting sample RUN1097_6309_50_T20-60457_I-1_TCGCGACACT-CGAGTAATGT_S21 (5,396,764 reads) (145 of 177) ...\n",
      "Plotting sample RUN1097_6309_51_T20-60462_I-1_CTTACGAACT-ACGGACTGAG_S52 (2,768,714 reads) (146 of 177) ...\n",
      "Plotting sample RUN1097_6309_52_T20-60466_I-1_CCTGACATCT-TAGCTACCAG_S59 (4,098,151 reads) (147 of 177) ...\n",
      "Plotting sample RUN1097_6309_53_T20-60468_I-1_AAGCGCTTAG-TGTCTGTGAC_S28 (4,495,167 reads) (148 of 177) ...\n",
      "Plotting sample RUN1097_6309_54_T20-60469_I-1_CTCTATTGAG-CACGCAAGTC_S60 (4,482,081 reads) (149 of 177) ...\n",
      "Plotting sample RUN1097_6309_55_T20-60474_I-1_CCGAACAGAG-TGATACTCGC_S58 (5,155,580 reads) (150 of 177) ...\n",
      "Plotting sample RUN1097_6309_56_T20-60491_I-1_CCGTGAGTTC-CTTGGTATGA_S57 (4,846,446 reads) (151 of 177) ...\n",
      "Plotting sample RUN1097_6309_57_T20-60542_I-1_AGGTAACAAC-GCACGATGCT_S36 (5,127,893 reads) (152 of 177) ...\n",
      "Plotting sample RUN1097_6309_58_T20-60629_I-1_TGGAGATCGA-CAGCGGTATG_S64 (4,076,560 reads) (153 of 177) ...\n",
      "Plotting sample RUN1097_6309_59_T20-60634_I-1_CAAGACTCAA-CACCGTCATA_S65 (6,591,209 reads) (154 of 177) ...\n",
      "Plotting sample RUN1097_6309_6_T20-60527_I-1_AACTCTCTAC-ACGGCTTCTC_S7 (6,661,216 reads) (155 of 177) ...\n",
      "Plotting sample RUN1097_6309_60_T20-60635_I-1_CGAGCGTAGT-TGCTCACGCA_S63 (6,576,512 reads) (156 of 177) ...\n",
      "Plotting sample RUN1097_6309_61_T20-60636_I-1_AACAACACAG-GAACATTGGC_S62 (6,181,228 reads) (157 of 177) ...\n",
      "Plotting sample RUN1097_6309_62_T20-60639_I-1_CACTTACGGC-GCTGGAGAAG_S45 (5,773,201 reads) (158 of 177) ...\n",
      "Plotting sample RUN1097_6309_63_T20-60640_I-1_AGAACTAACG-TTATCGAGAG_S69 (5,292,172 reads) (159 of 177) ...\n",
      "Plotting sample RUN1097_6309_64_T20-60642_I-1_TCAATTCCGA-TGTAGATCCG_S68 (8,319,723 reads) (160 of 177) ...\n",
      "Plotting sample RUN1097_6309_65_T20-60643_I-1_CGCACTTCGT-AATATGCCAG_S51 (4,327,122 reads) (161 of 177) ...\n",
      "Plotting sample RUN1097_6309_66_T20-60646_I-1_CCAGACCGAT-GCTACTTGCA_S67 (5,101,116 reads) (162 of 177) ...\n",
      "Plotting sample RUN1097_6309_67_T20-60648_I-1_TCGACGGCAA-GTAGTTGTGT_S56 (5,158,845 reads) (163 of 177) ...\n",
      "Plotting sample RUN1097_6309_68_T20-60651_I-1_CTGGATTACT-AAGTGTGCTA_S70 (3,275,709 reads) (164 of 177) ...\n",
      "Plotting sample RUN1097_6309_69_T20-60561_I-1_AGACTAGATG-CGTAACACTA_S71 (308,503 reads) (165 of 177) ...\n",
      "Plotting sample RUN1097_6309_7_T20-60540_I-1_GTATGCCACA-CCGAGTCTTC_S12 (6,305,716 reads) (166 of 177) ...\n",
      "Plotting sample RUN1097_6309_70_T20-60599_I-1_TCAGCATCTA-CCAGTGTACG_S61 (5,063,809 reads) (167 of 177) ...\n",
      "Plotting sample RUN1097_6309_71_T20-60605_I-1_GAGCAGCTTG-TGCCAGTATT_S72 (4,577,653 reads) (168 of 177) ...\n",
      "Plotting sample RUN1097_6309_72_T20-60606_I-1_CGGAACACAA-AACTCTACCG_S66 (4,902,901 reads) (169 of 177) ...\n",
      "Plotting sample RUN1097_6309_73_T20-60610_I-1_CTTGCTATTC-TAGAGTCACG_S73 (5,355,983 reads) (170 of 177) ...\n",
      "Plotting sample RUN1097_6309_74_T20-60617_I-1_CCACAGCCTT-GCTATTGAAG_S74 (5,646,499 reads) (171 of 177) ...\n",
      "Plotting sample RUN1097_6309_75_T20-60621_I-1_TCAGTTCGAT-CTGAGTCCTT_S75 (7,014,474 reads) (172 of 177) ...\n",
      "Plotting sample RUN1097_6309_76_T20-60625_I-1_AGCAATTCAC-TCATCGTCTG_S76 (4,612,476 reads) (173 of 177) ...\n",
      "Plotting sample RUN1097_6309_77_T20-60627_I-1_CCTCTATAGA-GTACTTCCAA_S77 (5,962,826 reads) (174 of 177) ...\n",
      "Plotting sample RUN1097_6309_78_T20-60637_I-1_GTTGCATCAA-GTTGATGCGA_S78 (5,814,752 reads) (175 of 177) ...\n",
      "Plotting sample RUN1097_6309_8_T20-60541_I-1_GCCAACAGAC-CGTTCACGTT_S13 (5,817,910 reads) (176 of 177) ...\n",
      "Plotting sample RUN1097_6309_9_T20-60546_I-1_ATAAGCGGAG-CTGTTCTACG_S14 (5,054,885 reads) (177 of 177) ...\n",
      "  |..........................................                            |  59%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |............................................                          |  62%\n",
      "label: calledPlots (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "Plotting sample 6322_1_T20-60638_I-1 (4,453,465 reads) (1 of 1) ...\n",
      "Plotting sample 6322_10_T20-60664_I-1 (3,409,233 reads) (1 of 1) ...\n",
      "Plotting sample 6322_100_T20-60765_I-1 (4,154,471 reads) (1 of 1) ...\n",
      "Plotting sample 6322_101_T20-60767_I-1 (4,478,697 reads) (1 of 1) ...\n",
      "Plotting sample 6322_102_T20-60768_I-1 (4,578,519 reads) (1 of 1) ...\n",
      "Plotting sample 6322_103_T20-60769_I-1 (4,131,044 reads) (1 of 1) ...\n",
      "Plotting sample 6322_104_T20-60770_I-1 (4,815,261 reads) (1 of 1) ...\n",
      "Plotting sample 6322_11_T20-60669_I-1 (5,254,711 reads) (1 of 1) ...\n",
      "Plotting sample 6322_12_T20-60672_I-1 (4,221,638 reads) (1 of 1) ...\n",
      "Plotting sample 6322_13_T20-60676_I-1 (4,107,860 reads) (1 of 1) ...\n",
      "Plotting sample 6322_14_T20-60680_I-1 (5,089,680 reads) (1 of 1) ...\n",
      "Plotting sample 6322_15_T20-60652_I-1 (5,106,257 reads) (1 of 1) ...\n",
      "Plotting sample 6322_16_T20-60654_I-1 (4,093,105 reads) (1 of 1) ...\n",
      "Plotting sample 6322_17_T20-60656_I-1 (3,330,248 reads) (1 of 1) ...\n",
      "Plotting sample 6322_18_T20-60657_I-1 (5,791,307 reads) (1 of 1) ...\n",
      "Plotting sample 6322_19_T20-60658_I-1 (5,248,100 reads) (1 of 1) ...\n",
      "Plotting sample 6322_2_T20-60641_I-1 (3,983,862 reads) (1 of 1) ...\n",
      "Plotting sample 6322_20_T20-60661_I-1 (5,425,916 reads) (1 of 1) ...\n",
      "Plotting sample 6322_21_T20-60662_I-1 (5,078,697 reads) (1 of 1) ...\n",
      "Plotting sample 6322_22_T20-60663_I-1 (4,980,715 reads) (1 of 1) ...\n",
      "Plotting sample 6322_23_T20-60666_I-1 (3,377,549 reads) (1 of 1) ...\n",
      "Plotting sample 6322_24_T20-60667_I-1 (4,180,452 reads) (1 of 1) ...\n",
      "Plotting sample 6322_25_T20-60668_I-1 (3,388,804 reads) (1 of 1) ...\n",
      "Plotting sample 6322_26_T20-60702_I-1 (4,143,796 reads) (1 of 1) ...\n",
      "Plotting sample 6322_27_T20-60704_I-1 (6,222,492 reads) (1 of 1) ...\n",
      "Plotting sample 6322_28_T20-60706_I-1 (3,894,082 reads) (1 of 1) ...\n",
      "Plotting sample 6322_29_T20-60708_I-1 (2,915,859 reads) (1 of 1) ...\n",
      "Plotting sample 6322_3_T20-60644_I-1 (3,620,091 reads) (1 of 1) ...\n",
      "Plotting sample 6322_30_T20-60718_I-1 (3,936,392 reads) (1 of 1) ...\n",
      "Plotting sample 6322_31_T20-60671_I-1 (4,355,710 reads) (1 of 1) ...\n",
      "Plotting sample 6322_33_T20-60674_I-1 (3,064,508 reads) (1 of 1) ...\n",
      "Plotting sample 6322_34_T20-60675_I-1 (5,641,231 reads) (1 of 1) ...\n",
      "Plotting sample 6322_35_T20-60677_I-1 (4,520,992 reads) (1 of 1) ...\n",
      "Plotting sample 6322_36_T20-60678_I-1 (4,513,888 reads) (1 of 1) ...\n",
      "Plotting sample 6322_38_T20-60686_I-1 (3,123,633 reads) (1 of 1) ...\n",
      "Plotting sample 6322_39_T20-60688_I-1 (4,183,356 reads) (1 of 1) ...\n",
      "Plotting sample 6322_4_T20-60645_I-1 (3,520,188 reads) (1 of 1) ...\n",
      "Plotting sample 6322_40_T20-60690_I-1 (5,242,536 reads) (1 of 1) ...\n",
      "Plotting sample 6322_43_T20-60694_I-1 (5,347,901 reads) (1 of 1) ...\n",
      "Plotting sample 6322_44_T20-60697_I-1 (4,545,050 reads) (1 of 1) ...\n",
      "Plotting sample 6322_45_T20-60703_I-1 (5,944,356 reads) (1 of 1) ...\n",
      "Plotting sample 6322_46_T20-60705_I-1 (4,931,917 reads) (1 of 1) ...\n",
      "Plotting sample 6322_48_T20-60710_I-1 (4,942,105 reads) (1 of 1) ...\n",
      "Plotting sample 6322_49_T20-60711_I-1 (3,715,599 reads) (1 of 1) ...\n",
      "Plotting sample 6322_5_T20-60647_I-1 (3,865,658 reads) (1 of 1) ...\n",
      "Plotting sample 6322_50_T20-60714_I-1 (4,398,207 reads) (1 of 1) ...\n",
      "Plotting sample 6322_51_T20-60715_I-1 (5,689,785 reads) (1 of 1) ...\n",
      "Plotting sample 6322_52_T20-60717_I-1 (4,775,544 reads) (1 of 1) ...\n",
      "Plotting sample 6322_53_T20-60723_I-1 (4,959,685 reads) (1 of 1) ...\n",
      "Plotting sample 6322_54_T20-60724_I-1 (5,867,399 reads) (1 of 1) ...\n",
      "Plotting sample 6322_55_T20-60709_I-1 (4,293,755 reads) (1 of 1) ...\n",
      "Plotting sample 6322_56_T20-60712_I-1 (3,028,113 reads) (1 of 1) ...\n",
      "Plotting sample 6322_57_T20-60713_I-1 (3,082,543 reads) (1 of 1) ...\n",
      "Plotting sample 6322_58_T20-60719_I-1 (4,418,926 reads) (1 of 1) ...\n",
      "Plotting sample 6322_59_T20-60720_I-1 (4,878,704 reads) (1 of 1) ...\n",
      "Plotting sample 6322_6_T20-60653_I-1 (3,577,353 reads) (1 of 1) ...\n",
      "Plotting sample 6322_60_T20-60721_I-1 (4,772,046 reads) (1 of 1) ...\n",
      "Plotting sample 6322_61_T20-60722_I-1 (4,590,370 reads) (1 of 1) ...\n",
      "Plotting sample 6322_62_T20-60728_I-1 (5,530,619 reads) (1 of 1) ...\n",
      "Plotting sample 6322_63_T20-60734_I-1 (5,393,294 reads) (1 of 1) ...\n",
      "Plotting sample 6322_64_T20-60743_I-1 (4,374,959 reads) (1 of 1) ...\n",
      "Plotting sample 6322_65_T20-60745_I-1 (3,532,716 reads) (1 of 1) ...\n",
      "Plotting sample 6322_66_T20-60746_I-1 (5,897,662 reads) (1 of 1) ...\n",
      "Plotting sample 6322_67_T20-60755_I-1 (4,654,593 reads) (1 of 1) ...\n",
      "Plotting sample 6322_68_T20-60756_I-1 (4,521,287 reads) (1 of 1) ...\n",
      "Plotting sample 6322_69_T20-60760_I-1 (5,386,483 reads) (1 of 1) ...\n",
      "Plotting sample 6322_7_T20-60655_I-1 (3,574,988 reads) (1 of 1) ...\n",
      "Plotting sample 6322_70_T20-60725_I-1 (5,003,086 reads) (1 of 1) ...\n",
      "Plotting sample 6322_71_T20-60726_I-1 (4,729,073 reads) (1 of 1) ...\n",
      "Plotting sample 6322_72_T20-60727_I-1 (4,893,795 reads) (1 of 1) ...\n",
      "Plotting sample 6322_73_T20-60729_I-1 (5,911,980 reads) (1 of 1) ...\n",
      "Plotting sample 6322_74_T20-60733_I-1 (5,831,972 reads) (1 of 1) ...\n",
      "Plotting sample 6322_75_T20-60738_I-1 (5,634,065 reads) (1 of 1) ...\n",
      "Plotting sample 6322_76_T20-60747_I-1 (6,563,253 reads) (1 of 1) ...\n",
      "Plotting sample 6322_77_T20-60748_I-1 (6,506,648 reads) (1 of 1) ...\n",
      "Plotting sample 6322_78_T20-60750_I-1 (5,816,014 reads) (1 of 1) ...\n",
      "Plotting sample 6322_79_T20-60752_I-1 (3,055,244 reads) (1 of 1) ...\n",
      "Plotting sample 6322_8_T20-60659_I-1 (3,707,253 reads) (1 of 1) ...\n",
      "Plotting sample 6322_80_T20-60753_I-1 (4,011,278 reads) (1 of 1) ...\n",
      "Plotting sample 6322_81_T20-60754_I-1 (4,810,508 reads) (1 of 1) ...\n",
      "Plotting sample 6322_82_T20-60757_I-1 (3,984,778 reads) (1 of 1) ...\n",
      "Plotting sample 6322_83_T20-60758_I-1 (4,933,128 reads) (1 of 1) ...\n",
      "Plotting sample 6322_84_T20-60759_I-1 (5,391,429 reads) (1 of 1) ...\n",
      "Plotting sample 6322_85_T20-60764_I-1 (4,197,463 reads) (1 of 1) ...\n",
      "Plotting sample 6322_86_T20-60665_I-1 (4,795,740 reads) (1 of 1) ...\n",
      "Plotting sample 6322_87_T20-60665_I-2 (4,503,411 reads) (1 of 1) ...\n",
      "Plotting sample 6322_88_T20-60700_I-1 (4,829,437 reads) (1 of 1) ...\n",
      "Plotting sample 6322_89_T20-60744_I-1 (5,085,385 reads) (1 of 1) ...\n",
      "Plotting sample 6322_9_T20-60660_I-1 (4,292,273 reads) (1 of 1) ...\n",
      "Plotting sample 6322_90_T20-60744_I-2 (3,854,691 reads) (1 of 1) ...\n",
      "Plotting sample 6322_91_T20-60761_I-1 (4,713,473 reads) (1 of 1) ...\n",
      "Plotting sample 6322_92_T20-60762_I-1 (3,712,206 reads) (1 of 1) ...\n",
      "Plotting sample 6322_93_T20-60763_I-1 (4,363,865 reads) (1 of 1) ...\n",
      "Plotting sample 6322_94_T20-60766_I-1 (3,907,462 reads) (1 of 1) ...\n",
      "Plotting sample 6322_95_T20-60444_I-1 (2,170,310 reads) (1 of 1) ...\n",
      "Plotting sample 6322_96_T20-60453_I-1 (4,149,974 reads) (1 of 1) ...\n",
      "Plotting sample 6322_97_T20-60598_I-1 (4,218,808 reads) (1 of 1) ...\n",
      "Plotting sample 6322_98_T20-60598_I-2 (5,633,444 reads) (1 of 1) ...\n",
      "Plotting sample 6322_99_T20-60626_I-1 (3,464,626 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_1_T20-60470_I-1_TAACTTGGTC-GTCGTGAATC_S2 (8,547,119 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_10_T20-60558_I-1_TCCAACTGAA-TGTCCGTAGG_S11 (6,460,784 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_11_T20-60564_I-1_ATTCGGCTTA-TTAACGTCCG_S17 (6,259,555 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_12_T20-60570_I-1_TATGGCAAGC-AACAGCAATG_S18 (6,816,988 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_13_T20-60572_I-1_CCTAAGCTGA-CGTCGCATCT_S19 (5,696,934 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_14_T20-60580_I-1_GCCACCTAGT-TGCGATGTAT_S20 (8,013,181 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_15_T20-60475_I-1_AGTATCCGAT-TATTGCGCTA_S16 (5,499,468 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_16_T20-60476_I-1_CCTCGTTGGA-GACTGCGTTA_S23 (7,339,720 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_17_T20-60596_I-1_CGACCTTAAT-GCTCTTAGGT_S24 (2,714,431 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_18_T20-60597_I-1_GAACCGCATT-CATTATGCCT_S25 (5,582,413 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_19_T20-60600_I-1_CGCGGTATCT-ATTACCGATC_S26 (3,386,410 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_2_T20-60501_I-1_TCGAACACGA-GCGAGGTTCT_S5 (5,107,330 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_20_T20-60603_I-1_ACTCCGAAGC-TCCGTGTTGC_S27 (5,526,299 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_21_T20-60608_I-1_CGATGTCTTG-CCGAACGTTC_S22 (5,646,575 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_22_T20-60609_I-1_CTCCGCCATA-GCGTTCAGCA_S30 (4,490,486 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_23_T20-60611_I-1_AACGAAGATC-TGTGGTCTAA_S31 (5,692,110 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_24_T20-60612_I-1_AATAGGTAGG-TGGCTTATGG_S32 (7,678,770 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_25_T20-60613_I-1_ATGTTGCCAC-TAGAAGCCAT_S33 (5,129,410 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_26_T20-60615_I-1_ACAACAGGCT-TCCACTGTTG_S34 (6,408,364 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_27_T20-60616_I-1_TATCCGCGTC-GTTGCATCGT_S35 (5,983,595 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_28_T20-60618_I-1_AATCTGCAGT-TGCGAAGTGA_S29 (6,091,415 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_29_T20-60619_I-1_GAGATGAGAG-GAATGGCGTT_S38 (5,352,997 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_3_T20-60502_I-1_TTACTTCGTG-AAGATCGGAT_S4 (4,434,309 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_30_T20-60620_I-1_TGACACGATT-GCCTAGGAAG_S39 (4,650,206 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_31_T20-60622_I-1_CCTCCTGTGA-CACGTCTACG_S43 (6,817,862 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_32_T20-60623_I-1_TATGCGTAGC-TCAGAGGCAC_S44 (6,937,203 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_33_T20-60628_I-1_CAGCAGGTCA-CACTTAGGTA_S42 (2,552,834 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_34_T20-60681_I-1_TTACGCGCAT-CTTAACGACA_S41 (4,636,639 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_35_T20-60682_I-1_AGGATGTCCA-ACTGGACGTT_S40 (5,506,862 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_36_T20-60683_I-1_AGTTCAGGCG-CACAGCTATT_S37 (5,083,786 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_37_T20-60685_I-1_TCACCATAAG-AAGTTCAGGA_S48 (5,259,450 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_38_T20-60687_I-1_GTGCTCTAGG-GTTACCAACG_S49 (2,189,340 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_39_T20-60689_I-1_TAGGCGAATG-AGTCCAGTGA_S50 (7,359,714 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_4_T20-60504_I-1_ACGAGACGTC-CGGTCTCTCT_S8 (6,394,710 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_40_T20-60691_I-1_CCAGTCGTCA-CGCTAGGCTA_S1 (5,649,008 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_41_T20-60695_I-1_TCTAGCTGCG-ATAGGTAAGG_S3 (4,858,668 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_42_T20-60696_I-1_TTCTCTTAGC-TCCTCGAGCT_S6 (5,083,669 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_43_T20-60698_I-1_GCCAGCTATC-TTGTTCGTGT_S47 (5,602,435 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_44_T20-60699_I-1_CGCCAAGGTA-GATAGTGTTC_S10 (6,664,843 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_45_T20-60701_I-1_ACTTCCGGCA-GACCAATTGT_S46 (8,081,113 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_46_T20-60437_I-1_GCAATCTGCT-ACGAACCAAG_S54 (8,354,592 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_47_T20-60446_I-1_AGTAAGGAGA-ATGCTTGGTC_S15 (6,153,409 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_48_T20-60455_I-1_AGAAGACCTA-AACCGTAACA_S55 (5,509,151 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_49_T20-60456_I-1_AGATTAGCGT-CGCCTTATAT_S53 (4,707,722 reads) (1 of 1) ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Plotting sample RUN1097_6309_5_T20-60505_I-1_CATAAGCGAC-CCTCTTCAAG_S9 (4,817,992 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_50_T20-60457_I-1_TCGCGACACT-CGAGTAATGT_S21 (5,396,764 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_51_T20-60462_I-1_CTTACGAACT-ACGGACTGAG_S52 (2,768,714 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_52_T20-60466_I-1_CCTGACATCT-TAGCTACCAG_S59 (4,098,151 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_53_T20-60468_I-1_AAGCGCTTAG-TGTCTGTGAC_S28 (4,495,167 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_54_T20-60469_I-1_CTCTATTGAG-CACGCAAGTC_S60 (4,482,081 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_55_T20-60474_I-1_CCGAACAGAG-TGATACTCGC_S58 (5,155,580 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_56_T20-60491_I-1_CCGTGAGTTC-CTTGGTATGA_S57 (4,846,446 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_57_T20-60542_I-1_AGGTAACAAC-GCACGATGCT_S36 (5,127,893 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_58_T20-60629_I-1_TGGAGATCGA-CAGCGGTATG_S64 (4,076,560 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_59_T20-60634_I-1_CAAGACTCAA-CACCGTCATA_S65 (6,591,209 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_6_T20-60527_I-1_AACTCTCTAC-ACGGCTTCTC_S7 (6,661,216 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_60_T20-60635_I-1_CGAGCGTAGT-TGCTCACGCA_S63 (6,576,512 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_61_T20-60636_I-1_AACAACACAG-GAACATTGGC_S62 (6,181,228 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_62_T20-60639_I-1_CACTTACGGC-GCTGGAGAAG_S45 (5,773,201 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_63_T20-60640_I-1_AGAACTAACG-TTATCGAGAG_S69 (5,292,172 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_64_T20-60642_I-1_TCAATTCCGA-TGTAGATCCG_S68 (8,319,723 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_65_T20-60643_I-1_CGCACTTCGT-AATATGCCAG_S51 (4,327,122 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_66_T20-60646_I-1_CCAGACCGAT-GCTACTTGCA_S67 (5,101,116 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_67_T20-60648_I-1_TCGACGGCAA-GTAGTTGTGT_S56 (5,158,845 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_68_T20-60651_I-1_CTGGATTACT-AAGTGTGCTA_S70 (3,275,709 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_69_T20-60561_I-1_AGACTAGATG-CGTAACACTA_S71 (308,503 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_7_T20-60540_I-1_GTATGCCACA-CCGAGTCTTC_S12 (6,305,716 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_70_T20-60599_I-1_TCAGCATCTA-CCAGTGTACG_S61 (5,063,809 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_71_T20-60605_I-1_GAGCAGCTTG-TGCCAGTATT_S72 (4,577,653 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_72_T20-60606_I-1_CGGAACACAA-AACTCTACCG_S66 (4,902,901 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_73_T20-60610_I-1_CTTGCTATTC-TAGAGTCACG_S73 (5,355,983 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_74_T20-60617_I-1_CCACAGCCTT-GCTATTGAAG_S74 (5,646,499 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_75_T20-60621_I-1_TCAGTTCGAT-CTGAGTCCTT_S75 (7,014,474 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_76_T20-60625_I-1_AGCAATTCAC-TCATCGTCTG_S76 (4,612,476 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_77_T20-60627_I-1_CCTCTATAGA-GTACTTCCAA_S77 (5,962,826 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_78_T20-60637_I-1_GTTGCATCAA-GTTGATGCGA_S78 (5,814,752 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_8_T20-60541_I-1_GCCAACAGAC-CGTTCACGTT_S13 (5,817,910 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_9_T20-60546_I-1_ATAAGCGGAG-CTGTTCTACG_S14 (5,054,885 reads) (1 of 1) ...\n",
      "  |..............................................                        |  66%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |................................................                      |  69%\n",
      "label: segmentsPlots (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "Plotting sample 6322_1_T20-60638_I-1 (4,453,465 reads) (1 of 1) ...\n",
      "Plotting sample 6322_10_T20-60664_I-1 (3,409,233 reads) (1 of 1) ...\n",
      "Plotting sample 6322_100_T20-60765_I-1 (4,154,471 reads) (1 of 1) ...\n",
      "Plotting sample 6322_101_T20-60767_I-1 (4,478,697 reads) (1 of 1) ...\n",
      "Plotting sample 6322_102_T20-60768_I-1 (4,578,519 reads) (1 of 1) ...\n",
      "Plotting sample 6322_103_T20-60769_I-1 (4,131,044 reads) (1 of 1) ...\n",
      "Plotting sample 6322_104_T20-60770_I-1 (4,815,261 reads) (1 of 1) ...\n",
      "Plotting sample 6322_11_T20-60669_I-1 (5,254,711 reads) (1 of 1) ...\n",
      "Plotting sample 6322_12_T20-60672_I-1 (4,221,638 reads) (1 of 1) ...\n",
      "Plotting sample 6322_13_T20-60676_I-1 (4,107,860 reads) (1 of 1) ...\n",
      "Plotting sample 6322_14_T20-60680_I-1 (5,089,680 reads) (1 of 1) ...\n",
      "Plotting sample 6322_15_T20-60652_I-1 (5,106,257 reads) (1 of 1) ...\n",
      "Plotting sample 6322_16_T20-60654_I-1 (4,093,105 reads) (1 of 1) ...\n",
      "Plotting sample 6322_17_T20-60656_I-1 (3,330,248 reads) (1 of 1) ...\n",
      "Plotting sample 6322_18_T20-60657_I-1 (5,791,307 reads) (1 of 1) ...\n",
      "Plotting sample 6322_19_T20-60658_I-1 (5,248,100 reads) (1 of 1) ...\n",
      "Plotting sample 6322_2_T20-60641_I-1 (3,983,862 reads) (1 of 1) ...\n",
      "Plotting sample 6322_20_T20-60661_I-1 (5,425,916 reads) (1 of 1) ...\n",
      "Plotting sample 6322_21_T20-60662_I-1 (5,078,697 reads) (1 of 1) ...\n",
      "Plotting sample 6322_22_T20-60663_I-1 (4,980,715 reads) (1 of 1) ...\n",
      "Plotting sample 6322_23_T20-60666_I-1 (3,377,549 reads) (1 of 1) ...\n",
      "Plotting sample 6322_24_T20-60667_I-1 (4,180,452 reads) (1 of 1) ...\n",
      "Plotting sample 6322_25_T20-60668_I-1 (3,388,804 reads) (1 of 1) ...\n",
      "Plotting sample 6322_26_T20-60702_I-1 (4,143,796 reads) (1 of 1) ...\n",
      "Plotting sample 6322_27_T20-60704_I-1 (6,222,492 reads) (1 of 1) ...\n",
      "Plotting sample 6322_28_T20-60706_I-1 (3,894,082 reads) (1 of 1) ...\n",
      "Plotting sample 6322_29_T20-60708_I-1 (2,915,859 reads) (1 of 1) ...\n",
      "Plotting sample 6322_3_T20-60644_I-1 (3,620,091 reads) (1 of 1) ...\n",
      "Plotting sample 6322_30_T20-60718_I-1 (3,936,392 reads) (1 of 1) ...\n",
      "Plotting sample 6322_31_T20-60671_I-1 (4,355,710 reads) (1 of 1) ...\n",
      "Plotting sample 6322_33_T20-60674_I-1 (3,064,508 reads) (1 of 1) ...\n",
      "Plotting sample 6322_34_T20-60675_I-1 (5,641,231 reads) (1 of 1) ...\n",
      "Plotting sample 6322_35_T20-60677_I-1 (4,520,992 reads) (1 of 1) ...\n",
      "Plotting sample 6322_36_T20-60678_I-1 (4,513,888 reads) (1 of 1) ...\n",
      "Plotting sample 6322_38_T20-60686_I-1 (3,123,633 reads) (1 of 1) ...\n",
      "Plotting sample 6322_39_T20-60688_I-1 (4,183,356 reads) (1 of 1) ...\n",
      "Plotting sample 6322_4_T20-60645_I-1 (3,520,188 reads) (1 of 1) ...\n",
      "Plotting sample 6322_40_T20-60690_I-1 (5,242,536 reads) (1 of 1) ...\n",
      "Plotting sample 6322_43_T20-60694_I-1 (5,347,901 reads) (1 of 1) ...\n",
      "Plotting sample 6322_44_T20-60697_I-1 (4,545,050 reads) (1 of 1) ...\n",
      "Plotting sample 6322_45_T20-60703_I-1 (5,944,356 reads) (1 of 1) ...\n",
      "Plotting sample 6322_46_T20-60705_I-1 (4,931,917 reads) (1 of 1) ...\n",
      "Plotting sample 6322_48_T20-60710_I-1 (4,942,105 reads) (1 of 1) ...\n",
      "Plotting sample 6322_49_T20-60711_I-1 (3,715,599 reads) (1 of 1) ...\n",
      "Plotting sample 6322_5_T20-60647_I-1 (3,865,658 reads) (1 of 1) ...\n",
      "Plotting sample 6322_50_T20-60714_I-1 (4,398,207 reads) (1 of 1) ...\n",
      "Plotting sample 6322_51_T20-60715_I-1 (5,689,785 reads) (1 of 1) ...\n",
      "Plotting sample 6322_52_T20-60717_I-1 (4,775,544 reads) (1 of 1) ...\n",
      "Plotting sample 6322_53_T20-60723_I-1 (4,959,685 reads) (1 of 1) ...\n",
      "Plotting sample 6322_54_T20-60724_I-1 (5,867,399 reads) (1 of 1) ...\n",
      "Plotting sample 6322_55_T20-60709_I-1 (4,293,755 reads) (1 of 1) ...\n",
      "Plotting sample 6322_56_T20-60712_I-1 (3,028,113 reads) (1 of 1) ...\n",
      "Plotting sample 6322_57_T20-60713_I-1 (3,082,543 reads) (1 of 1) ...\n",
      "Plotting sample 6322_58_T20-60719_I-1 (4,418,926 reads) (1 of 1) ...\n",
      "Plotting sample 6322_59_T20-60720_I-1 (4,878,704 reads) (1 of 1) ...\n",
      "Plotting sample 6322_6_T20-60653_I-1 (3,577,353 reads) (1 of 1) ...\n",
      "Plotting sample 6322_60_T20-60721_I-1 (4,772,046 reads) (1 of 1) ...\n",
      "Plotting sample 6322_61_T20-60722_I-1 (4,590,370 reads) (1 of 1) ...\n",
      "Plotting sample 6322_62_T20-60728_I-1 (5,530,619 reads) (1 of 1) ...\n",
      "Plotting sample 6322_63_T20-60734_I-1 (5,393,294 reads) (1 of 1) ...\n",
      "Plotting sample 6322_64_T20-60743_I-1 (4,374,959 reads) (1 of 1) ...\n",
      "Plotting sample 6322_65_T20-60745_I-1 (3,532,716 reads) (1 of 1) ...\n",
      "Plotting sample 6322_66_T20-60746_I-1 (5,897,662 reads) (1 of 1) ...\n",
      "Plotting sample 6322_67_T20-60755_I-1 (4,654,593 reads) (1 of 1) ...\n",
      "Plotting sample 6322_68_T20-60756_I-1 (4,521,287 reads) (1 of 1) ...\n",
      "Plotting sample 6322_69_T20-60760_I-1 (5,386,483 reads) (1 of 1) ...\n",
      "Plotting sample 6322_7_T20-60655_I-1 (3,574,988 reads) (1 of 1) ...\n",
      "Plotting sample 6322_70_T20-60725_I-1 (5,003,086 reads) (1 of 1) ...\n",
      "Plotting sample 6322_71_T20-60726_I-1 (4,729,073 reads) (1 of 1) ...\n",
      "Plotting sample 6322_72_T20-60727_I-1 (4,893,795 reads) (1 of 1) ...\n",
      "Plotting sample 6322_73_T20-60729_I-1 (5,911,980 reads) (1 of 1) ...\n",
      "Plotting sample 6322_74_T20-60733_I-1 (5,831,972 reads) (1 of 1) ...\n",
      "Plotting sample 6322_75_T20-60738_I-1 (5,634,065 reads) (1 of 1) ...\n",
      "Plotting sample 6322_76_T20-60747_I-1 (6,563,253 reads) (1 of 1) ...\n",
      "Plotting sample 6322_77_T20-60748_I-1 (6,506,648 reads) (1 of 1) ...\n",
      "Plotting sample 6322_78_T20-60750_I-1 (5,816,014 reads) (1 of 1) ...\n",
      "Plotting sample 6322_79_T20-60752_I-1 (3,055,244 reads) (1 of 1) ...\n",
      "Plotting sample 6322_8_T20-60659_I-1 (3,707,253 reads) (1 of 1) ...\n",
      "Plotting sample 6322_80_T20-60753_I-1 (4,011,278 reads) (1 of 1) ...\n",
      "Plotting sample 6322_81_T20-60754_I-1 (4,810,508 reads) (1 of 1) ...\n",
      "Plotting sample 6322_82_T20-60757_I-1 (3,984,778 reads) (1 of 1) ...\n",
      "Plotting sample 6322_83_T20-60758_I-1 (4,933,128 reads) (1 of 1) ...\n",
      "Plotting sample 6322_84_T20-60759_I-1 (5,391,429 reads) (1 of 1) ...\n",
      "Plotting sample 6322_85_T20-60764_I-1 (4,197,463 reads) (1 of 1) ...\n",
      "Plotting sample 6322_86_T20-60665_I-1 (4,795,740 reads) (1 of 1) ...\n",
      "Plotting sample 6322_87_T20-60665_I-2 (4,503,411 reads) (1 of 1) ...\n",
      "Plotting sample 6322_88_T20-60700_I-1 (4,829,437 reads) (1 of 1) ...\n",
      "Plotting sample 6322_89_T20-60744_I-1 (5,085,385 reads) (1 of 1) ...\n",
      "Plotting sample 6322_9_T20-60660_I-1 (4,292,273 reads) (1 of 1) ...\n",
      "Plotting sample 6322_90_T20-60744_I-2 (3,854,691 reads) (1 of 1) ...\n",
      "Plotting sample 6322_91_T20-60761_I-1 (4,713,473 reads) (1 of 1) ...\n",
      "Plotting sample 6322_92_T20-60762_I-1 (3,712,206 reads) (1 of 1) ...\n",
      "Plotting sample 6322_93_T20-60763_I-1 (4,363,865 reads) (1 of 1) ...\n",
      "Plotting sample 6322_94_T20-60766_I-1 (3,907,462 reads) (1 of 1) ...\n",
      "Plotting sample 6322_95_T20-60444_I-1 (2,170,310 reads) (1 of 1) ...\n",
      "Plotting sample 6322_96_T20-60453_I-1 (4,149,974 reads) (1 of 1) ...\n",
      "Plotting sample 6322_97_T20-60598_I-1 (4,218,808 reads) (1 of 1) ...\n",
      "Plotting sample 6322_98_T20-60598_I-2 (5,633,444 reads) (1 of 1) ...\n",
      "Plotting sample 6322_99_T20-60626_I-1 (3,464,626 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_1_T20-60470_I-1_TAACTTGGTC-GTCGTGAATC_S2 (8,547,119 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_10_T20-60558_I-1_TCCAACTGAA-TGTCCGTAGG_S11 (6,460,784 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_11_T20-60564_I-1_ATTCGGCTTA-TTAACGTCCG_S17 (6,259,555 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_12_T20-60570_I-1_TATGGCAAGC-AACAGCAATG_S18 (6,816,988 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_13_T20-60572_I-1_CCTAAGCTGA-CGTCGCATCT_S19 (5,696,934 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_14_T20-60580_I-1_GCCACCTAGT-TGCGATGTAT_S20 (8,013,181 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_15_T20-60475_I-1_AGTATCCGAT-TATTGCGCTA_S16 (5,499,468 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_16_T20-60476_I-1_CCTCGTTGGA-GACTGCGTTA_S23 (7,339,720 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_17_T20-60596_I-1_CGACCTTAAT-GCTCTTAGGT_S24 (2,714,431 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_18_T20-60597_I-1_GAACCGCATT-CATTATGCCT_S25 (5,582,413 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_19_T20-60600_I-1_CGCGGTATCT-ATTACCGATC_S26 (3,386,410 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_2_T20-60501_I-1_TCGAACACGA-GCGAGGTTCT_S5 (5,107,330 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_20_T20-60603_I-1_ACTCCGAAGC-TCCGTGTTGC_S27 (5,526,299 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_21_T20-60608_I-1_CGATGTCTTG-CCGAACGTTC_S22 (5,646,575 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_22_T20-60609_I-1_CTCCGCCATA-GCGTTCAGCA_S30 (4,490,486 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_23_T20-60611_I-1_AACGAAGATC-TGTGGTCTAA_S31 (5,692,110 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_24_T20-60612_I-1_AATAGGTAGG-TGGCTTATGG_S32 (7,678,770 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_25_T20-60613_I-1_ATGTTGCCAC-TAGAAGCCAT_S33 (5,129,410 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_26_T20-60615_I-1_ACAACAGGCT-TCCACTGTTG_S34 (6,408,364 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_27_T20-60616_I-1_TATCCGCGTC-GTTGCATCGT_S35 (5,983,595 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_28_T20-60618_I-1_AATCTGCAGT-TGCGAAGTGA_S29 (6,091,415 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_29_T20-60619_I-1_GAGATGAGAG-GAATGGCGTT_S38 (5,352,997 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_3_T20-60502_I-1_TTACTTCGTG-AAGATCGGAT_S4 (4,434,309 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_30_T20-60620_I-1_TGACACGATT-GCCTAGGAAG_S39 (4,650,206 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_31_T20-60622_I-1_CCTCCTGTGA-CACGTCTACG_S43 (6,817,862 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_32_T20-60623_I-1_TATGCGTAGC-TCAGAGGCAC_S44 (6,937,203 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_33_T20-60628_I-1_CAGCAGGTCA-CACTTAGGTA_S42 (2,552,834 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_34_T20-60681_I-1_TTACGCGCAT-CTTAACGACA_S41 (4,636,639 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_35_T20-60682_I-1_AGGATGTCCA-ACTGGACGTT_S40 (5,506,862 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_36_T20-60683_I-1_AGTTCAGGCG-CACAGCTATT_S37 (5,083,786 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_37_T20-60685_I-1_TCACCATAAG-AAGTTCAGGA_S48 (5,259,450 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_38_T20-60687_I-1_GTGCTCTAGG-GTTACCAACG_S49 (2,189,340 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_39_T20-60689_I-1_TAGGCGAATG-AGTCCAGTGA_S50 (7,359,714 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_4_T20-60504_I-1_ACGAGACGTC-CGGTCTCTCT_S8 (6,394,710 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_40_T20-60691_I-1_CCAGTCGTCA-CGCTAGGCTA_S1 (5,649,008 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_41_T20-60695_I-1_TCTAGCTGCG-ATAGGTAAGG_S3 (4,858,668 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_42_T20-60696_I-1_TTCTCTTAGC-TCCTCGAGCT_S6 (5,083,669 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_43_T20-60698_I-1_GCCAGCTATC-TTGTTCGTGT_S47 (5,602,435 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_44_T20-60699_I-1_CGCCAAGGTA-GATAGTGTTC_S10 (6,664,843 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_45_T20-60701_I-1_ACTTCCGGCA-GACCAATTGT_S46 (8,081,113 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_46_T20-60437_I-1_GCAATCTGCT-ACGAACCAAG_S54 (8,354,592 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_47_T20-60446_I-1_AGTAAGGAGA-ATGCTTGGTC_S15 (6,153,409 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_48_T20-60455_I-1_AGAAGACCTA-AACCGTAACA_S55 (5,509,151 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_49_T20-60456_I-1_AGATTAGCGT-CGCCTTATAT_S53 (4,707,722 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_5_T20-60505_I-1_CATAAGCGAC-CCTCTTCAAG_S9 (4,817,992 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_50_T20-60457_I-1_TCGCGACACT-CGAGTAATGT_S21 (5,396,764 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_51_T20-60462_I-1_CTTACGAACT-ACGGACTGAG_S52 (2,768,714 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_52_T20-60466_I-1_CCTGACATCT-TAGCTACCAG_S59 (4,098,151 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_53_T20-60468_I-1_AAGCGCTTAG-TGTCTGTGAC_S28 (4,495,167 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_54_T20-60469_I-1_CTCTATTGAG-CACGCAAGTC_S60 (4,482,081 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_55_T20-60474_I-1_CCGAACAGAG-TGATACTCGC_S58 (5,155,580 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_56_T20-60491_I-1_CCGTGAGTTC-CTTGGTATGA_S57 (4,846,446 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_57_T20-60542_I-1_AGGTAACAAC-GCACGATGCT_S36 (5,127,893 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_58_T20-60629_I-1_TGGAGATCGA-CAGCGGTATG_S64 (4,076,560 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_59_T20-60634_I-1_CAAGACTCAA-CACCGTCATA_S65 (6,591,209 reads) (1 of 1) ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Plotting sample RUN1097_6309_6_T20-60527_I-1_AACTCTCTAC-ACGGCTTCTC_S7 (6,661,216 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_60_T20-60635_I-1_CGAGCGTAGT-TGCTCACGCA_S63 (6,576,512 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_61_T20-60636_I-1_AACAACACAG-GAACATTGGC_S62 (6,181,228 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_62_T20-60639_I-1_CACTTACGGC-GCTGGAGAAG_S45 (5,773,201 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_63_T20-60640_I-1_AGAACTAACG-TTATCGAGAG_S69 (5,292,172 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_64_T20-60642_I-1_TCAATTCCGA-TGTAGATCCG_S68 (8,319,723 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_65_T20-60643_I-1_CGCACTTCGT-AATATGCCAG_S51 (4,327,122 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_66_T20-60646_I-1_CCAGACCGAT-GCTACTTGCA_S67 (5,101,116 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_67_T20-60648_I-1_TCGACGGCAA-GTAGTTGTGT_S56 (5,158,845 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_68_T20-60651_I-1_CTGGATTACT-AAGTGTGCTA_S70 (3,275,709 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_69_T20-60561_I-1_AGACTAGATG-CGTAACACTA_S71 (308,503 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_7_T20-60540_I-1_GTATGCCACA-CCGAGTCTTC_S12 (6,305,716 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_70_T20-60599_I-1_TCAGCATCTA-CCAGTGTACG_S61 (5,063,809 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_71_T20-60605_I-1_GAGCAGCTTG-TGCCAGTATT_S72 (4,577,653 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_72_T20-60606_I-1_CGGAACACAA-AACTCTACCG_S66 (4,902,901 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_73_T20-60610_I-1_CTTGCTATTC-TAGAGTCACG_S73 (5,355,983 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_74_T20-60617_I-1_CCACAGCCTT-GCTATTGAAG_S74 (5,646,499 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_75_T20-60621_I-1_TCAGTTCGAT-CTGAGTCCTT_S75 (7,014,474 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_76_T20-60625_I-1_AGCAATTCAC-TCATCGTCTG_S76 (4,612,476 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_77_T20-60627_I-1_CCTCTATAGA-GTACTTCCAA_S77 (5,962,826 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_78_T20-60637_I-1_GTTGCATCAA-GTTGATGCGA_S78 (5,814,752 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_8_T20-60541_I-1_GCCAACAGAC-CGTTCACGTT_S13 (5,817,910 reads) (1 of 1) ...\n",
      "Plotting sample RUN1097_6309_9_T20-60546_I-1_ATAAGCGGAG-CTGTTCTACG_S14 (5,054,885 reads) (1 of 1) ...\n",
      "  |..................................................                    |  72%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |....................................................                  |  75%\n",
      "label: Stats (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |.......................................................               |  78%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.........................................................             |  81%\n",
      "label: cghregions (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |...........................................................           |  84%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.............................................................         |  88%\n",
      "label: segments (with options) \n",
      "List of 2\n",
      " $ warning: logi FALSE\n",
      " $ message: logi FALSE\n",
      "\n",
      "  |...............................................................       |  91%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |..................................................................    |  94%\n",
      "label: TBL (with options) \n",
      "List of 2\n",
      " $ warning: logi FALSE\n",
      " $ message: logi FALSE\n",
      "\n",
      "Loading required package: CGHcall\n",
      "Loading required package: DNAcopy\n",
      "Loading required package: snowfall\n",
      "Loading required package: snow\n",
      "\n",
      "Attaching package: 'snow'\n",
      "\n",
      "The following objects are masked from 'package:BiocGenerics':\n",
      "\n",
      "    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,\n",
      "    clusterExport, clusterMap, clusterSplit, parApply, parCapply,\n",
      "    parLapply, parRapply, parSapply\n",
      "\n",
      "The following objects are masked from 'package:parallel':\n",
      "\n",
      "    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,\n",
      "    clusterExport, clusterMap, clusterSplit, makeCluster, parApply,\n",
      "    parCapply, parLapply, parRapply, parSapply, splitIndices,\n",
      "    stopCluster\n",
      "\n",
      "\n",
      "Attaching package: 'CGHcall'\n",
      "\n",
      "The following object is masked from 'package:BiocGenerics':\n",
      "\n",
      "    normalize\n",
      "\n",
      "Loading required package: GenomicRanges\n",
      "Loading required package: stats4\n",
      "Loading required package: S4Vectors\n",
      "\n",
      "Attaching package: 'S4Vectors'\n",
      "\n",
      "The following object is masked from 'package:future':\n",
      "\n",
      "    values\n",
      "\n",
      "The following object is masked from 'package:base':\n",
      "\n",
      "    expand.grid\n",
      "\n",
      "Loading required package: IRanges\n",
      "\n",
      "Attaching package: 'IRanges'\n",
      "\n",
      "The following object is masked _by_ '.GlobalEnv':\n",
      "\n",
      "    shift\n",
      "\n",
      "Loading required package: GenomeInfoDb\n",
      "\n",
      "Attaching package: 'GeneBreak'\n",
      "\n",
      "The following object is masked _by_ '.GlobalEnv':\n",
      "\n",
      "    sampleNames\n",
      "\n",
      "The following object is masked from 'package:CGHcall':\n",
      "\n",
      "    segmentData\n",
      "\n",
      "The following object is masked from 'package:Biobase':\n",
      "\n",
      "    sampleNames\n",
      "\n",
      "── Attaching packages ─────────────────────────────────────── tidyverse 1.3.0 ──\n",
      "✔ tibble  3.1.1     ✔ purrr   0.3.4\n",
      "✔ tidyr   1.1.3     ✔ dplyr   1.0.6\n",
      "✔ readr   1.4.0     ✔ forcats 0.5.1\n",
      "── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──\n",
      "✖ dplyr::collapse()   masks IRanges::collapse()\n",
      "✖ dplyr::combine()    masks Biobase::combine(), BiocGenerics::combine()\n",
      "✖ tidyr::complete()   masks RCurl::complete()\n",
      "✖ dplyr::desc()       masks IRanges::desc()\n",
      "✖ tidyr::expand()     masks S4Vectors::expand()\n",
      "✖ dplyr::filter()     masks stats::filter()\n",
      "✖ dplyr::first()      masks S4Vectors::first()\n",
      "✖ dplyr::lag()        masks stats::lag()\n",
      "✖ ggplot2::Position() masks BiocGenerics::Position(), base::Position()\n",
      "✖ purrr::reduce()     masks GenomicRanges::reduce(), IRanges::reduce()\n",
      "✖ dplyr::rename()     masks S4Vectors::rename()\n",
      "✖ dplyr::select()     masks MASS::select()\n",
      "✖ dplyr::slice()      masks IRanges::slice()\n",
      "  |....................................................................  |  97%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |......................................................................| 100%\n",
      "label: unnamed-chunk-2\n",
      "\n",
      "output file: QDNASeq_hg38_IntEnd_btD_server100kb.knit.md\n",
      "\n",
      "/home/l.gonzalez/anaconda3/bin/pandoc +RTS -K512m -RTS QDNASeq_hg38_IntEnd_btD_server100kb.utf8.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output QDNASeq_hg38_IntEnd_btD_server100kb.html --lua-filter /DATA/home/l.gonzalez/R/x86_64-pc-linux-gnu-library/3.6/rmarkdown/rmarkdown/lua/pagebreak.lua --lua-filter /DATA/home/l.gonzalez/R/x86_64-pc-linux-gnu-library/3.6/rmarkdown/rmarkdown/lua/latex-div.lua --self-contained --variable bs3=TRUE --standalone --section-divs --template /DATA/home/l.gonzalez/R/x86_64-pc-linux-gnu-library/3.6/rmarkdown/rmd/h/default.html --no-highlight --variable highlightjs=1 --variable theme=bootstrap --include-in-header /tmp/RtmpKcXluV/rmarkdown-str10b79d41d0a55e.html --mathjax --variable 'mathjax-url:https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML' \n",
      "\n",
      "Output created: QDNASeq_hg38_IntEnd_btD_server100kb.html\n",
      "There were 50 or more warnings (use warnings() to see the first 50)\n",
      "> \n",
      "> \n"
     ]
    }
   ],
   "source": [
    "# To run as CLI\n",
    "# Here is line of code to run in the Terminal (cliking here it will run it there, as knit)\n",
    "!R -e \"rmarkdown::render('/DATA/share/pipelines/lgleon/IntEnd/DNA/Shallow_Seq/Results_QDNAseqFlow/BatchD_9-10/btD_hg38_100kb/QDNASeq_hg38_IntEnd_btD_server100kb.Rmd')\" \n",
    "\n",
    "###Do not use R4.0###\n",
    "#!/opt/R/4.0.4/bin/Rscript -e \"rmarkdown::render('/DATA/share/pipelines/lgleon/NGS_protocol/results_QDNAseq/normalS/QDNASeqFlow_hg38_30Kb.Rmd')\"\n",
    "\n",
    "#The General Code, which is not, it is at \n",
    "# !R -e \"/home/l.gonzalez/Notebooks/bash_R_scripts/QDNASeqFlow_hg38_NGSprotocol_30Kb.Rmd\"\n",
    "#The best way to work is make a copy of the general one, in the results folder, and run it form there\n",
    "#as a knit it will create a html file at the same time\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "R version 3.6.3 (2020-02-29) -- \"Holding the Windsock\"\n",
      "Copyright (C) 2020 The R Foundation for Statistical Computing\n",
      "Platform: x86_64-pc-linux-gnu (64-bit)\n",
      "\n",
      "R is free software and comes with ABSOLUTELY NO WARRANTY.\n",
      "You are welcome to redistribute it under certain conditions.\n",
      "Type 'license()' or 'licence()' for distribution details.\n",
      "\n",
      "  Natural language support but running in an English locale\n",
      "\n",
      "R is a collaborative project with many contributors.\n",
      "Type 'contributors()' for more information and\n",
      "'citation()' on how to cite R or R packages in publications.\n",
      "\n",
      "Type 'demo()' for some demos, 'help()' for on-line help, or\n",
      "'help.start()' for an HTML browser interface to help.\n",
      "Type 'q()' to quit R.\n",
      "\n",
      "> rmarkdown::render('/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/qdnaseq_out/QDNASeq_hg38_oldOrganoids_30kb.Rmd')\n",
      "\n",
      "\n",
      "processing file: QDNASeq_hg38_oldOrganoids_30kb.Rmd\n",
      "  |..                                                                    |   3%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |....                                                                  |   6%\n",
      "label: unnamed-chunk-1 (with options) \n",
      "List of 1\n",
      " $ include: logi FALSE\n",
      "\n",
      "  |.......                                                               |   9%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.........                                                             |  12%\n",
      "label: setup (with options) \n",
      "List of 1\n",
      " $ include: logi FALSE\n",
      "\n",
      "  |...........                                                           |  16%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.............                                                         |  19%\n",
      "label: functions (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |...............                                                       |  22%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |..................                                                    |  25%\n",
      "label: wd (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |....................                                                  |  28%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |......................                                                |  31%\n",
      "label: paths (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |........................                                              |  34%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |..........................                                            |  38%\n",
      "label: bam2smooth (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "    5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup (1 of 4): extracting reads ... saving in cache ... binning ... saving in cache ...\n",
      "    5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup (2 of 4): extracting reads ... saving in cache ... binning ... saving in cache ...\n",
      "    5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup (3 of 4): extracting reads ... saving in cache ... binning ... saving in cache ...\n",
      "    5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup (4 of 4): extracting reads ... saving in cache ... binning ... saving in cache ...\n",
      "102,954\ttotal bins\n",
      " 95,844\tof which in selected chromosomes\n",
      " 92,067\tof which with reference sequence\n",
      " 82,838\tfinal bins\n",
      "Calculating correction for GC content and mappability\n",
      "    Calculating fit for sample 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup (1 of 4) ...\n",
      "    Calculating fit for sample 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup (2 of 4) ...\n",
      "    Calculating fit for sample 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup (3 of 4) ...\n",
      "    Calculating fit for sample 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup (4 of 4) ...\n",
      "Done.\n",
      "Applying median normalization ...\n",
      "Smoothing outliers ...\n",
      "  |............................                                          |  41%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |...............................                                       |  44%\n",
      "label: dewaving (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "Loading required package: gtools\n",
      "Applying deWaving ... \n",
      "  |.................................                                     |  47%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |...................................                                   |  50%\n",
      "label: calling (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "Performing segmentation:\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "EM algorithm started ... \n",
      "\n",
      "396456817489634211.8133.5757090258587921404.4447757090291525847404.4698.3\n",
      "Calling iteration1:\n",
      "\n",
      "optim results\n",
      "time: 5\n",
      "minimum: 166291.159687242\n",
      "4166222.956616616-0.942417202498039-0.4885436389859240.02026510276985380.4216571805574440.7208277112423691.149409018022460.2543228408899220.138007951881070.08745118237233790.1635645834340240.1811099357506910.372359869605834\n",
      "396607117494564211.9133.5757090258587921404.4447757090291525847404.4698.3\n",
      "Calling iteration2:\n",
      "\n",
      "optim results\n",
      "time: 3\n",
      "minimum: 166233.749610535\n",
      "4166202.654310735-0.913194137208366-0.4826057028706990.02397902869847170.4334344024056070.7409610049725561.165215827578470.3147507272466880.1224182392514990.06348641257488140.1274983422066360.1782651254057860.361680440992586\n",
      "396607117494564211.9133.5757090258587921404.4447757090291525847404.4698.3\n",
      "Calling iteration3:\n",
      "\n",
      "optim results\n",
      "time: 3\n",
      "minimum: 166215.634232757\n",
      "4166200.008672904-0.892964171417285-0.4778746022997270.02393314174626950.433731119810840.7414682467271421.164157507357930.4088585371335470.1190099435944730.05805270463507570.1086165602002820.1453682675935230.329675162709935\n",
      "EM algorithm done ...\n",
      "\n",
      "Computing posterior probabilities for all segments ...\n",
      "\n",
      "Total time:0minutes\n",
      "\n",
      "Adjusting segmented data for cellularity ... \n",
      "\n",
      "Cellularity sample1: 1\n",
      "\n",
      "Cellularity sample2: 1\n",
      "\n",
      "Cellularity sample3: 1\n",
      "\n",
      "Cellularity sample4: 1\n",
      "\n",
      "Adjusting normalized data for cellularity ... \n",
      "\n",
      "Cellularity sample1: 1\n",
      "\n",
      "Cellularity sample2: 1\n",
      "\n",
      "Cellularity sample3: 1\n",
      "\n",
      "Cellularity sample4: 1\n",
      "\n",
      "1\n",
      "396855417943527212136.9757090258587921404.4447757090291525847404.4698.3\n",
      "396857218357725212140.1757090258587921404.4447757090291525847404.4698.3\n",
      "396857018357722212140.1757090258587921404.4447757090291525847404.4698.3\n",
      "396859719020447212145.2757090258587921404.4447757090291525847404.4698.3\n",
      "396866119020558212145.2757090258587921404.4447757090291525847404.4698.3\n",
      "396867019020562212145.2757090258587921404.4447757090291525847404.4698.3\n",
      "396867919020566212145.2757090258587921404.4447757090291525847404.4698.3\n",
      "396868819020570212145.2757090258587921404.4447757090291525847404.4698.3\n",
      "396869719020574212145.2757090258587921404.4447757090291525847404.4698.3\n",
      "396870319020577212145.2757090258587921404.4447757090291525847404.4698.3\n",
      "396872419020615212145.2757090258587921404.4447757090291525847404.4698.3\n",
      "396914719020942212145.2757090258587921404.4447757090291525847404.4698.3\n",
      "2\n",
      "396916519766494212150.9757090258587921404.4447757090291525847404.4698.3\n",
      "396916719766497212150.9757090258587921404.4447757090291525847404.4698.3\n",
      "396916519766494212150.9757090258587921404.4447757090291525847404.4698.3\n",
      "396916819849323212151.5757090258587921404.4447757090291525847404.4698.3\n",
      "396916819849323212151.5757090258587921404.4447757090291525847404.4698.3\n",
      "396917719849327212151.5757090258587921404.4447757090291525847404.4698.3\n",
      "396918619849331212151.5757090258587921404.4447757090291525847404.4698.3\n",
      "396919519849335212151.5757090258587921404.4447757090291525847404.4698.3\n",
      "396920419849339212151.5757090258587921404.4447757090291525847404.4698.3\n",
      "396921019849342212151.5757090258587921404.4447757090291525847404.4698.3\n",
      "396923119849380212151.5757090258587921404.4447757090291525847404.4698.3\n",
      "397337021267881212.3162.3757090258587921404.4447757090291525847404.4698.3\n",
      "3\n",
      "397337621764912212.3166.1757090258587921404.4447757090291525847404.4698.3\n",
      "397337821764915212.3166.1757090258587921404.4447757090291525847404.4698.3\n",
      "397337621764912212.3166.1757090258587921404.4447757090291525847404.4698.3\n",
      "397337921847745212.3166.7757090258587921404.4447757090291525847404.4698.3\n",
      "397337421682067212.3165.5757090258587921404.4447757090291525847404.4698.3\n",
      "397338321682071212.3165.5757090258587921404.4447757090291525847404.4698.3\n",
      "397339221682075212.3165.5757090258587921404.4447757090291525847404.4698.3\n",
      "397340121682079212.3165.5757090258587921404.4447757090291525847404.4698.3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "397341021682083212.3165.5757090258587921404.4447757090291525847404.4698.3\n",
      "397341621682086212.3165.5757090258587921404.4447757090291525847404.4698.3\n",
      "397343721682124212.3165.5757090258587921404.4447757090291525847404.4698.3\n",
      "397337021930601212.3167.4757090258587921404.4447757090291525847404.4698.3\n",
      "4\n",
      "397337622427632212.3171.2757090258587921404.4447757090291525847404.4698.3\n",
      "397337822427635212.3171.2757090258587921404.4447757090291525847404.4698.3\n",
      "397337622427632212.3171.2757090258587921404.4447757090291525847404.4698.3\n",
      "397337922510512212.3171.8757090258587921404.4447757090291525847404.4698.3\n",
      "397337422344834212.3170.5757090258587921404.4447757090291525847404.4698.3\n",
      "397338322344838212.3170.5757090258587921404.4447757090291525847404.4698.3\n",
      "397339222344842212.3170.5757090258587921404.4447757090291525847404.4698.3\n",
      "397340122344846212.3170.5757090258587921404.4447757090291525847404.4698.3\n",
      "397341022344850212.3170.5757090258587921404.4447757090291525847404.4698.3\n",
      "397341622344853212.3170.5757090258587921404.4447757090291525847404.4698.3\n",
      "397343722344891212.3170.5757090258587921404.4447757090291525847404.4698.3\n",
      "397337022593346212.3172.4757090258587921404.4447757090291525847404.4698.3\n",
      "FINISHED!\n",
      "\n",
      "Total time:0minutes\n",
      "\n",
      "  |.....................................                                 |  53%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.......................................                               |  56%\n",
      "label: freqPlots (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "Plotting sample 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup (10,991,657 reads) (1 of 4) ...\n",
      "Plotting sample 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup (5,984,265 reads) (2 of 4) ...\n",
      "Plotting sample 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup (3,006,820 reads) (3 of 4) ...\n",
      "Plotting sample 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup (5,509,831 reads) (4 of 4) ...\n",
      "Plotting sample 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup (10,991,657 reads) (1 of 4) ...\n",
      "Plotting sample 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup (5,984,265 reads) (2 of 4) ...\n",
      "Plotting sample 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup (3,006,820 reads) (3 of 4) ...\n",
      "Plotting sample 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup (5,509,831 reads) (4 of 4) ...\n",
      "  |..........................................                            |  59%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |............................................                          |  62%\n",
      "label: calledPlots (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "Plotting sample 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup (10,991,657 reads) (1 of 1) ...\n",
      "Plotting sample 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup (5,984,265 reads) (1 of 1) ...\n",
      "Plotting sample 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup (3,006,820 reads) (1 of 1) ...\n",
      "Plotting sample 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup (5,509,831 reads) (1 of 1) ...\n",
      "  |..............................................                        |  66%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |................................................                      |  69%\n",
      "label: segmentsPlots (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "Plotting sample 5177_20_B16PON_C-050_p6_ACACAGAA_S39_R1_001.markdup (10,991,657 reads) (1 of 1) ...\n",
      "Plotting sample 5177_21_B16PON_C-051_p9_ACGCTCGA_S33_R1_001.markdup (5,984,265 reads) (1 of 1) ...\n",
      "Plotting sample 5177_22_B16PON_C-052_p6_ACGTATCA_S40_R1_001.markdup (3,006,820 reads) (1 of 1) ...\n",
      "Plotting sample 5177_23_B16PON_C-053_p4_AGATCGCA_S41_R1_001.markdup (5,509,831 reads) (1 of 1) ...\n",
      "  |..................................................                    |  72%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |....................................................                  |  75%\n",
      "label: Stats (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |.......................................................               |  78%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.........................................................             |  81%\n",
      "label: cghregions (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |...........................................................           |  84%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.............................................................         |  88%\n",
      "label: segments (with options) \n",
      "List of 2\n",
      " $ warning: logi FALSE\n",
      " $ message: logi FALSE\n",
      "\n",
      "  |...............................................................       |  91%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |..................................................................    |  94%\n",
      "label: TBL (with options) \n",
      "List of 2\n",
      " $ warning: logi FALSE\n",
      " $ message: logi FALSE\n",
      "\n",
      "Loading required package: CGHcall\n",
      "Loading required package: DNAcopy\n",
      "Loading required package: snowfall\n",
      "Loading required package: snow\n",
      "\n",
      "Attaching package: 'snow'\n",
      "\n",
      "The following objects are masked from 'package:BiocGenerics':\n",
      "\n",
      "    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,\n",
      "    clusterExport, clusterMap, clusterSplit, parApply, parCapply,\n",
      "    parLapply, parRapply, parSapply\n",
      "\n",
      "The following objects are masked from 'package:parallel':\n",
      "\n",
      "    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,\n",
      "    clusterExport, clusterMap, clusterSplit, makeCluster, parApply,\n",
      "    parCapply, parLapply, parRapply, parSapply, splitIndices,\n",
      "    stopCluster\n",
      "\n",
      "\n",
      "Attaching package: 'CGHcall'\n",
      "\n",
      "The following object is masked from 'package:BiocGenerics':\n",
      "\n",
      "    normalize\n",
      "\n",
      "Loading required package: GenomicRanges\n",
      "Loading required package: stats4\n",
      "Loading required package: S4Vectors\n",
      "\n",
      "Attaching package: 'S4Vectors'\n",
      "\n",
      "The following object is masked from 'package:future':\n",
      "\n",
      "    values\n",
      "\n",
      "The following object is masked from 'package:base':\n",
      "\n",
      "    expand.grid\n",
      "\n",
      "Loading required package: IRanges\n",
      "\n",
      "Attaching package: 'IRanges'\n",
      "\n",
      "The following object is masked _by_ '.GlobalEnv':\n",
      "\n",
      "    shift\n",
      "\n",
      "Loading required package: GenomeInfoDb\n",
      "\n",
      "Attaching package: 'GeneBreak'\n",
      "\n",
      "The following object is masked _by_ '.GlobalEnv':\n",
      "\n",
      "    sampleNames\n",
      "\n",
      "The following object is masked from 'package:CGHcall':\n",
      "\n",
      "    segmentData\n",
      "\n",
      "The following object is masked from 'package:Biobase':\n",
      "\n",
      "    sampleNames\n",
      "\n",
      "── Attaching packages ─────────────────────────────────────── tidyverse 1.3.0 ──\n",
      "✔ tibble  3.1.1     ✔ purrr   0.3.4\n",
      "✔ tidyr   1.1.3     ✔ dplyr   1.0.6\n",
      "✔ readr   1.4.0     ✔ forcats 0.5.1\n",
      "── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──\n",
      "✖ dplyr::collapse()   masks IRanges::collapse()\n",
      "✖ dplyr::combine()    masks Biobase::combine(), BiocGenerics::combine()\n",
      "✖ dplyr::desc()       masks IRanges::desc()\n",
      "✖ tidyr::expand()     masks S4Vectors::expand()\n",
      "✖ dplyr::filter()     masks stats::filter()\n",
      "✖ dplyr::first()      masks S4Vectors::first()\n",
      "✖ dplyr::lag()        masks stats::lag()\n",
      "✖ ggplot2::Position() masks BiocGenerics::Position(), base::Position()\n",
      "✖ purrr::reduce()     masks GenomicRanges::reduce(), IRanges::reduce()\n",
      "✖ dplyr::rename()     masks S4Vectors::rename()\n",
      "✖ dplyr::select()     masks MASS::select()\n",
      "✖ dplyr::slice()      masks IRanges::slice()\n",
      "  |....................................................................  |  97%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |......................................................................| 100%\n",
      "label: unnamed-chunk-2\n",
      "\n",
      "output file: QDNASeq_hg38_oldOrganoids_30kb.knit.md\n",
      "\n",
      "/home/l.gonzalez/anaconda3/bin/pandoc +RTS -K512m -RTS QDNASeq_hg38_oldOrganoids_30kb.knit.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output QDNASeq_hg38_oldOrganoids_30kb.html --lua-filter /DATA/home/l.gonzalez/R/x86_64-pc-linux-gnu-library/3.6/rmarkdown/rmarkdown/lua/pagebreak.lua --lua-filter /DATA/home/l.gonzalez/R/x86_64-pc-linux-gnu-library/3.6/rmarkdown/rmarkdown/lua/latex-div.lua --self-contained --variable bs3=TRUE --standalone --section-divs --template /DATA/home/l.gonzalez/R/x86_64-pc-linux-gnu-library/3.6/rmarkdown/rmd/h/default.html --no-highlight --variable highlightjs=1 --variable theme=bootstrap --include-in-header /tmp/RtmppSwiEK/rmarkdown-str22b0f3711cd91.html --mathjax --variable 'mathjax-url:https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML' \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "Output created: QDNASeq_hg38_oldOrganoids_30kb.html\r\n",
      "There were 50 or more warnings (use warnings() to see the first 50)\r\n",
      "> \r\n",
      "> \r\n"
     ]
    }
   ],
   "source": [
    "## To run QDNAseq for the 30kb bins\n",
    "!R -e \"rmarkdown::render('/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/old_organoids/qdnaseq_out/QDNASeq_hg38_oldOrganoids_30kb.Rmd')\" \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] TRUE\r\n"
     ]
    }
   ],
   "source": [
    "#! selectR-config --select 3.6\n",
    "#! selectR-config --select 4.0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "R version 3.6.3 (2020-02-29) -- \"Holding the Windsock\"\n",
      "Copyright (C) 2020 The R Foundation for Statistical Computing\n",
      "Platform: x86_64-pc-linux-gnu (64-bit)\n",
      "\n",
      "R is free software and comes with ABSOLUTELY NO WARRANTY.\n",
      "You are welcome to redistribute it under certain conditions.\n",
      "Type 'license()' or 'licence()' for distribution details.\n",
      "\n",
      "  Natural language support but running in an English locale\n",
      "\n",
      "R is a collaborative project with many contributors.\n",
      "Type 'contributors()' for more information and\n",
      "'citation()' on how to cite R or R packages in publications.\n",
      "\n",
      "Type 'demo()' for some demos, 'help()' for on-line help, or\n",
      "'help.start()' for an HTML browser interface to help.\n",
      "Type 'q()' to quit R.\n",
      "\n",
      "> "
     ]
    }
   ],
   "source": [
    "#!R\n",
    "#!/opt/R/4.0.4/bin/Rscript"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "alignmentSieve\t\t  google-oauthlib-tool\t    pyrsa-keygen\r\n",
      "bamCompare\t\t  import_pb_to_tensorboard  pyrsa-priv2pub\r\n",
      "bamCoverage\t\t  markdown_py\t\t    pyrsa-sign\r\n",
      "bamPEFragmentSize\t  multiBamSummary\t    pyrsa-verify\r\n",
      "bigwigCompare\t\t  multiBigwigSummary\t    saved_model_cli\r\n",
      "computeGCBias\t\t  pbr\t\t\t    sphinx-apidoc\r\n",
      "computeMatrix\t\t  plotCorrelation\t    sphinx-autogen\r\n",
      "computeMatrixOperations   plotCoverage\t\t    sphinx-build\r\n",
      "correctGCBias\t\t  plotEnrichment\t    sphinx-quickstart\r\n",
      "deeptools\t\t  plotFingerprint\t    tensorboard\r\n",
      "estimateReadFiltering\t  plotHeatmap\t\t    tflite_convert\r\n",
      "estimateScaleFactor\t  plotPCA\t\t    tf_upgrade_v2\r\n",
      "estimator_ckpt_converter  plotProfile\t\t    toco\r\n",
      "f2py\t\t\t  pybabel\t\t    toco_from_protos\r\n",
      "f2py3\t\t\t  pyrsa-decrypt\t\t    tqdm\r\n",
      "f2py3.8\t\t\t  pyrsa-encrypt\t\t    wheel\r\n"
     ]
    }
   ],
   "source": [
    "#!ls /usr/local/bin/\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[1;30m[WARNING]\u001b[0m         multiqc : \u001b[33mMultiQC Version v1.11 now available!\u001b[0m\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : This is MultiQC v1.10.1\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : Template    : default\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : Searching   : /DATA/share/pipelines/lgleon/test/testCode_steps_Nousefuldata/testdata/fastq\n",
      "\u001b[2KSearching   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[35m100%\u001b[0m \u001b[32m20/20\u001b[0m  20\u001b[0m  \n",
      "\u001b[?25h\u001b[1;30m[WARNING]\u001b[0m         multiqc : \u001b[33mNo analysis results found. Cleaning up..\u001b[0m\n",
      "\u001b[1;30m[INFO   ]\u001b[0m         multiqc : MultiQC complete\n"
     ]
    }
   ],
   "source": [
    "!multiqc /DATA/share/pipelines/lgleon/test/testCode_steps_Nousefuldata/testdata/fastq/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/bin/bash: cutapt: command not found\r\n"
     ]
    }
   ],
   "source": [
    "!cutapt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.\r\n",
      "To initialize your shell, run\r\n",
      "\r\n",
      "    $ conda init <SHELL_NAME>\r\n",
      "\r\n",
      "Currently supported shells are:\r\n",
      "  - bash\r\n",
      "  - fish\r\n",
      "  - tcsh\r\n",
      "  - xonsh\r\n",
      "  - zsh\r\n",
      "  - powershell\r\n",
      "\r\n",
      "See 'conda init --help' for more information and options.\r\n",
      "\r\n",
      "IMPORTANT: You may need to close and restart your shell after running 'conda init'.\r\n",
      "\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!conda activate base"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python [conda env:anaconda3]",
   "language": "python",
   "name": "conda-env-anaconda3-py"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
