{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# This is a Notebook to run SCNA analysis in the server\n",
    "\n",
    "### Input is fastq files for this anaysis is low-coverage WGS data\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "What are you running:\n",
    "\n",
    "- QC check using fastqc\n",
    "- Alignment (mapping): bwa-aln + bwa samse\n",
    "- Indexing\n",
    "- multiqc check, optional\n",
    "- QDNAseq (new version using hg38)\n",
    "\n",
    "I am using it to have the steps for the SCNA analaysis in one notbeook as pipeline, and it can make it run form here but remember that it is running in th terminal\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/DATA/home/l.gonzalez/Notebooks\r\n"
     ]
    }
   ],
   "source": [
    "#in case you forgot where are you\n",
    "!pwd\n",
    "#!ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os, sys\n",
    "import glob\n",
    "import logging"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Before starting the analysis we will check the QC of the fastq files usign fastqc\n",
    "To do that, please go to the cell below and change the path to the folder where the fastq files are, then run it"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!fastqc /DATA/share/pipelines/lgleon/test/testCode_steps_Nousefuldata/testdata/fastq/*fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### As an optional step we can run multiqc after fastqc to create a unified html files with all the fastqc to see then in one go"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run bwa for the mapping single end fastq files\n",
    "There is no need for trimming\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#Script:\n",
    "#BWA ALN\n",
    "\n",
    "#Ref\n",
    "#/DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa\n",
    "\n",
    "#data\n",
    "prj=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/B16PON/fastq\n",
    "\n",
    "\n",
    "for file in ${prj}/*.f*q.gz\n",
    "do\n",
    "\n",
    "   inputfqgz=$file\n",
    "   #outputbam=${file/%.fastq.gz/.bam}  #  % means that suffixes are matched, # would mean prefixes\"\n",
    "   filenamestem=${file/%.f*q.gz/}\n",
    "   \n",
    "   echo \"!!!!!!!!!!!!!! Expected final output {$filenamestem.markdup.bam}\"\n",
    "\n",
    "   bwa aln -t 32 -n 2 \\\n",
    "    /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa  \\\n",
    "    $inputfqgz | bwa samse -r '@RG\\tID:inputfqgz\\tSM:inputfqgz' \\\n",
    "    /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa \\\n",
    "    - $inputfqgz | samtools view -Su - | samtools sort - -o $filenamestem.sorted.bam > $filenamestem.sorted.bam\n",
    "\n",
    "   picard MarkDuplicates -I $filenamestem.sorted.bam -O $filenamestem.markdup.bam -M $filenamestem.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX true -VALIDATION_STRINGENCY LENIENT\n",
    "\n",
    "done\n",
    "\n",
    "                        \n",
    "\n",
    "mkdir fastQC\n",
    "#mv *f*q.gz fastQC\n",
    "\n",
    "mkdir sorted_bam\n",
    "mv *sorted.bam sorted_bam\n",
    "#rm *sorted.bam\n",
    "\n",
    "mkdir markdup_bam\n",
    "mv *markdup.bam markdup_bam  \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_10_Tissue_pt_264_2_S40_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 44.87 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.05 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.04 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.15 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.37 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.85 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.06 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.37 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.80 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.52 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.16 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.14 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.89 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.24 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.23 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 4.95 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.01 sec\n",
      "[bwa_aln_core] 7890171 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_10_Tissue_pt_264_2_S40_R1_001.fastq.gz\n",
      "[main] Real time: 233.545 sec; CPU: 1379.217 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.55 sec\n",
      "[bwa_aln_core] print alignments... 0.03 sec\n",
      "[bwa_aln_core] 7890171 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_10_Tissue_pt_264_2_S40_R1_001.fastq.gz\n",
      "[main] Real time: 237.167 sec; CPU: 161.588 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 17:04:47\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_10_Tissue_pt_264_2_S40_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_10_Tissue_pt_264_2_S40_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_10_Tissue_pt_264_2_S40_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "17:04:48.131 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 17:04:48 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_10_Tissue_pt_264_2_S40_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_10_Tissue_pt_264_2_S40_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_10_Tissue_pt_264_2_S40_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 17:04:48 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 17:04:48\tMarkDuplicates\tStart of doWork freeMemory: 502528144; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:04:48\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 17:04:48\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 17:04:55\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:07s.  Time for last 1,000,000:    7s.  Last read position: 11:50,668,712\n",
      "INFO\t2021-05-07 17:04:55\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:05:00\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    5s.  Last read position: 15:26,673,451\n",
      "INFO\t2021-05-07 17:05:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:05:05\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    5s.  Last read position: 2:28,544,639\n",
      "INFO\t2021-05-07 17:05:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:05:17\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 22:41,667,904\n",
      "INFO\t2021-05-07 17:05:17\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:05:22\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:34s.  Time for last 1,000,000:    5s.  Last read position: 5:48,135,400\n",
      "INFO\t2021-05-07 17:05:22\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:05:30\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    8s.  Last read position: 7:97,404,070\n",
      "INFO\t2021-05-07 17:05:30\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 6791277, Read name D00645:375:CDYJNANXX:4:2310:19595:10721, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6791278, Read name D00645:375:CDYJNANXX:4:2214:18716:26866, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6791279, Read name D00645:375:CDYJNANXX:4:1104:15990:5867, MAPQ should be 0 for unmapped read.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 17:05:36\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:48s.  Time for last 1,000,000:    5s.  Last read position: X:110,025,519\n",
      "INFO\t2021-05-07 17:05:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 7098523, Read name D00645:375:CDYJNANXX:4:1108:7809:17181, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7145316, Read name D00645:375:CDYJNANXX:4:2108:6695:73950, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:05:37\tMarkDuplicates\tRead 7148943 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 17:05:37\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 731951176; totalMemory: 757596160; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:05:37\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 17:05:37\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:05:37\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:05:37\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 17:05:39\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 17:05:39\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 516510912; totalMemory: 767557632; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:05:39\tMarkDuplicates\tMarking 82894 records as duplicates.\n",
      "INFO\t2021-05-07 17:05:39\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 17:05:39\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 6791277, Read name D00645:375:CDYJNANXX:4:2310:19595:10721, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6791278, Read name D00645:375:CDYJNANXX:4:2214:18716:26866, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6791279, Read name D00645:375:CDYJNANXX:4:1104:15990:5867, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7098523, Read name D00645:375:CDYJNANXX:4:1108:7809:17181, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7145316, Read name D00645:375:CDYJNANXX:4:2108:6695:73950, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:06:28\tMarkDuplicates\tBefore output close freeMemory: 768226504; totalMemory: 781713408; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:06:28\tMarkDuplicates\tAfter output close freeMemory: 768750792; totalMemory: 782237696; maxMemory: 954728448\n",
      "[Fri May 07 17:06:28 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.68 minutes.\n",
      "Runtime.totalMemory()=782237696\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_2_Tissue_pt_83_2_S32_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 40.85 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.28 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.50 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.78 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 42.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.36 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 39.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.27 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 38.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.42 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 38.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.99 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.12 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 38.55 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 37.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 38.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.34 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.11 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 28.05 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5151795 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_2_Tissue_pt_83_2_S32_R1_001.fastq.gz\n",
      "[main] Real time: 150.208 sec; CPU: 827.259 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.20 sec\n",
      "[bwa_aln_core] 5151795 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_2_Tissue_pt_83_2_S32_R1_001.fastq.gz\n",
      "[main] Real time: 154.973 sec; CPU: 107.901 sec\n",
      "[bam_sort_core] merging from 1 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 17:09:21\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_2_Tissue_pt_83_2_S32_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_2_Tissue_pt_83_2_S32_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_2_Tissue_pt_83_2_S32_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "17:09:21.824 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 17:09:21 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_2_Tissue_pt_83_2_S32_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_2_Tissue_pt_83_2_S32_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_2_Tissue_pt_83_2_S32_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 17:09:21 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 17:09:21\tMarkDuplicates\tStart of doWork freeMemory: 502528200; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:09:21\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 17:09:21\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 17:09:28\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 12:91,027,812\n",
      "INFO\t2021-05-07 17:09:28\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:09:32\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    4s.  Last read position: 2:31,182,653\n",
      "INFO\t2021-05-07 17:09:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:09:37\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    4s.  Last read position: 4:36,176,370\n",
      "INFO\t2021-05-07 17:09:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:09:49\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:27s.  Time for last 1,000,000:   12s.  Last read position: 7:153,832,177\n",
      "INFO\t2021-05-07 17:09:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 4744371, Read name D00645:375:CDYJNANXX:4:1110:3499:91095, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4764498, Read name D00645:375:CDYJNANXX:4:1316:3033:22304, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4766554, Read name D00645:375:CDYJNANXX:4:1102:7170:19855, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:09:53\tMarkDuplicates\tRead 4767868 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 17:09:54\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 801639488; totalMemory: 827326464; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:09:54\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 17:09:54\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:09:54\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:09:54\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 17:09:55\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 17:09:55\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 575852992; totalMemory: 826802176; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:09:55\tMarkDuplicates\tMarking 64090 records as duplicates.\n",
      "INFO\t2021-05-07 17:09:55\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 17:09:55\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 4744371, Read name D00645:375:CDYJNANXX:4:1110:3499:91095, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4764498, Read name D00645:375:CDYJNANXX:4:1316:3033:22304, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4766554, Read name D00645:375:CDYJNANXX:4:1102:7170:19855, MAPQ should be 0 for unmapped read.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 17:10:28\tMarkDuplicates\tBefore output close freeMemory: 826479568; totalMemory: 839909376; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:10:28\tMarkDuplicates\tAfter output close freeMemory: 827003856; totalMemory: 840433664; maxMemory: 954728448\n",
      "[Fri May 07 17:10:28 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.11 minutes.\n",
      "Runtime.totalMemory()=840433664\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_5_Tissue_pt_120_2_S35_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 40.58 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 39.71 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.03 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 42.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.40 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 36.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.57 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.09 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.17 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.56 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 36.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.71 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.39 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.16 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.99 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.28 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 36.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 36.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.00 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.14 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 37.07 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.16 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 13.95 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.03 sec\n",
      "[bwa_aln_core] 5601490 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_5_Tissue_pt_120_2_S35_R1_001.fastq.gz[bwa_aln_core] convert to sequence coordinate... \n",
      "[main] Real time: 162.845 sec; CPU: 877.089 sec\n",
      "3.20 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.53 sec\n",
      "[bwa_aln_core] print alignments... 0.11 sec\n",
      "[bwa_aln_core] 5601490 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_5_Tissue_pt_120_2_S35_R1_001.fastq.gz\n",
      "[main] Real time: 166.710 sec; CPU: 117.435 sec\n",
      "[bam_sort_core] merging from 1 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 17:13:34\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_5_Tissue_pt_120_2_S35_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_5_Tissue_pt_120_2_S35_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_5_Tissue_pt_120_2_S35_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "17:13:34.350 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 17:13:34 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_5_Tissue_pt_120_2_S35_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_5_Tissue_pt_120_2_S35_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_5_Tissue_pt_120_2_S35_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 17:13:34 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 17:13:34\tMarkDuplicates\tStart of doWork freeMemory: 502528176; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:13:34\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 17:13:34\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 17:13:41\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 12:82,708,765\n",
      "INFO\t2021-05-07 17:13:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:13:46\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    4s.  Last read position: 18:74,792,894\n",
      "INFO\t2021-05-07 17:13:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:13:51\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    5s.  Last read position: 3:96,958,089\n",
      "INFO\t2021-05-07 17:13:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:14:03\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:29s.  Time for last 1,000,000:   12s.  Last read position: 6:138,006,874\n",
      "INFO\t2021-05-07 17:14:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:14:09\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:34s.  Time for last 1,000,000:    5s.  Last read position: X:123,684,590\n",
      "INFO\t2021-05-07 17:14:09\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 5109145, Read name D00645:375:CDYJNANXX:4:2311:17360:71617, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5111842, Read name D00645:375:CDYJNANXX:4:2209:14504:95452, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:14:09\tMarkDuplicates\tRead 5112071 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 17:14:10\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 805339616; totalMemory: 830472192; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:14:10\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 17:14:10\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:14:10\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:14:10\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 17:14:11\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 17:14:11\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 578051824; totalMemory: 828899328; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:14:11\tMarkDuplicates\tMarking 55368 records as duplicates.\n",
      "INFO\t2021-05-07 17:14:11\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 17:14:11\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 5109145, Read name D00645:375:CDYJNANXX:4:2311:17360:71617, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5111842, Read name D00645:375:CDYJNANXX:4:2209:14504:95452, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:14:46\tMarkDuplicates\tBefore output close freeMemory: 828553080; totalMemory: 842006528; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:14:46\tMarkDuplicates\tAfter output close freeMemory: 828553080; totalMemory: 842006528; maxMemory: 954728448\n",
      "[Fri May 07 17:14:46 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.21 minutes.\n",
      "Runtime.totalMemory()=842006528\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_6_Tissue_pt_137_2_S36_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 49.10 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.78 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.41 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 6.27 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.69 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.15 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.12 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.72 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.80 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.83 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.27 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.50 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6807610 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_6_Tissue_pt_137_2_S36_R1_001.fastq.gz\n",
      "[main] Real time: 202.225 sec; CPU: 1376.505 sec\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6807610 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_6_Tissue_pt_137_2_S36_R1_001.fastq.gz\n",
      "[main] Real time: 206.995 sec; CPU: 133.809 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 17:18:36\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_6_Tissue_pt_137_2_S36_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_6_Tissue_pt_137_2_S36_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_6_Tissue_pt_137_2_S36_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "17:18:36.609 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 17:18:36 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_6_Tissue_pt_137_2_S36_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_6_Tissue_pt_137_2_S36_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_6_Tissue_pt_137_2_S36_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 17:18:36 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 17:18:36\tMarkDuplicates\tStart of doWork freeMemory: 502528176; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:18:36\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 17:18:36\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 17:18:43\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:07s.  Time for last 1,000,000:    7s.  Last read position: 12:16,885,974\n",
      "INFO\t2021-05-07 17:18:43\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:18:49\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:13s.  Time for last 1,000,000:    6s.  Last read position: 16:62,874,444\n",
      "INFO\t2021-05-07 17:18:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:18:55\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:18s.  Time for last 1,000,000:    5s.  Last read position: 20:8,521,349\n",
      "INFO\t2021-05-07 17:18:55\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:19:06\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:29s.  Time for last 1,000,000:   11s.  Last read position: 5:11,150,477\n",
      "INFO\t2021-05-07 17:19:06\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:19:11\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:34s.  Time for last 1,000,000:    4s.  Last read position: 7:158,143,297\n",
      "INFO\t2021-05-07 17:19:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 5573303, Read name D00645:375:CDYJNANXX:4:1102:8881:82769, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5573304, Read name D00645:375:CDYJNANXX:4:1211:19528:69076, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5868400, Read name D00645:375:CDYJNANXX:4:1213:4009:57900, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5919235, Read name D00645:375:CDYJNANXX:4:1114:19827:46631, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5919236, Read name D00645:375:CDYJNANXX:4:1113:9518:27381, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5919494, Read name D00645:375:CDYJNANXX:4:1206:1403:46645, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5919495, Read name D00645:375:CDYJNANXX:4:1316:13489:38424, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:19:19\tMarkDuplicates\tRead 5922321 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 17:19:23\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 883187136; totalMemory: 905445376; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:19:23\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 17:19:24\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:19:24\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:19:24\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 17:19:25\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 17:19:25\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 658189624; totalMemory: 905445376; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:19:25\tMarkDuplicates\tMarking 87709 records as duplicates.\n",
      "INFO\t2021-05-07 17:19:25\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 17:19:25\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 5573303, Read name D00645:375:CDYJNANXX:4:1102:8881:82769, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5573304, Read name D00645:375:CDYJNANXX:4:1211:19528:69076, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5868400, Read name D00645:375:CDYJNANXX:4:1213:4009:57900, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5919235, Read name D00645:375:CDYJNANXX:4:1114:19827:46631, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5919236, Read name D00645:375:CDYJNANXX:4:1113:9518:27381, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5919494, Read name D00645:375:CDYJNANXX:4:1206:1403:46645, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5919495, Read name D00645:375:CDYJNANXX:4:1316:13489:38424, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:20:07\tMarkDuplicates\tBefore output close freeMemory: 844894512; totalMemory: 855113728; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:20:07\tMarkDuplicates\tAfter output close freeMemory: 844894512; totalMemory: 855113728; maxMemory: 954728448\n",
      "[Fri May 07 17:20:07 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.52 minutes.\n",
      "Runtime.totalMemory()=855113728\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_9_Tissue_pt_216_2_S39_R1_001.markdup.bam}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 42.93 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 42.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.39 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.60 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.04 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 42.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.04 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 40.16 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6266578 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_9_Tissue_pt_216_2_S39_R1_001.fastq.gz\n",
      "[main] Real time: 183.016 sec; CPU: 1070.949 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6266578 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_9_Tissue_pt_216_2_S39_R1_001.fastq.gz\n",
      "[main] Real time: 187.934 sec; CPU: 126.745 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 17:23:37\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_9_Tissue_pt_216_2_S39_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_9_Tissue_pt_216_2_S39_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_9_Tissue_pt_216_2_S39_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "17:23:37.607 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 17:23:37 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_9_Tissue_pt_216_2_S39_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_9_Tissue_pt_216_2_S39_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5564_9_Tissue_pt_216_2_S39_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 17:23:37 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 17:23:37\tMarkDuplicates\tStart of doWork freeMemory: 502528176; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:23:37\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 17:23:37\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 17:23:44\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 11:127,201,301\n",
      "INFO\t2021-05-07 17:23:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:23:49\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    5s.  Last read position: 16:84,446,449\n",
      "INFO\t2021-05-07 17:23:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:23:55\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    5s.  Last read position: 20:35,693,850\n",
      "INFO\t2021-05-07 17:23:55\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:24:06\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 5:30,004,375\n",
      "INFO\t2021-05-07 17:24:06\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:24:11\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    4s.  Last read position: 8:18,294,721\n",
      "INFO\t2021-05-07 17:24:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 5496323, Read name D00645:375:CDYJNANXX:4:2112:11662:55993, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5496324, Read name D00645:375:CDYJNANXX:4:1203:9992:13712, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:24:15\tMarkDuplicates\tRead 5838889 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 17:24:16\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 805236664; totalMemory: 830472192; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:24:16\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 17:24:16\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:24:16\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:24:16\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 17:24:17\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 17:24:18\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 578008352; totalMemory: 828899328; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:24:18\tMarkDuplicates\tMarking 61779 records as duplicates.\n",
      "INFO\t2021-05-07 17:24:18\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 17:24:18\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 5496323, Read name D00645:375:CDYJNANXX:4:2112:11662:55993, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5496324, Read name D00645:375:CDYJNANXX:4:1203:9992:13712, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:24:57\tMarkDuplicates\tBefore output close freeMemory: 829621048; totalMemory: 843055104; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:24:58\tMarkDuplicates\tAfter output close freeMemory: 829621048; totalMemory: 843055104; maxMemory: 954728448\n",
      "[Fri May 07 17:24:58 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.34 minutes.\n",
      "Runtime.totalMemory()=843055104\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_1_74_CTGATCGT_S2_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 54.12 sec\n",
      "[bwa_aln_core] write to the disk... 0.08 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 55.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.15 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.71 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.12 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.09 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.21 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.14 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.91 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.50 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.16 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.91 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.72 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.02 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.94 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.88 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.80 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.28 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.21 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.07 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.66 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.63 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.17 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 20.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 7707705 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_1_74_CTGATCGT_S2_R1_001.fastq.gz[bwa_aln_core] convert to sequence coordinate... \n",
      "[main] Real time: 233.976 sec; CPU: 1614.566 sec\n",
      "3.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.53 sec\n",
      "[bwa_aln_core] print alignments... 0.11 sec\n",
      "[bwa_aln_core] 7707705 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_1_74_CTGATCGT_S2_R1_001.fastq.gz\n",
      "[main] Real time: 237.805 sec; CPU: 153.461 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 17:29:20\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_1_74_CTGATCGT_S2_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_1_74_CTGATCGT_S2_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_1_74_CTGATCGT_S2_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "17:29:21.011 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 17:29:21 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_1_74_CTGATCGT_S2_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_1_74_CTGATCGT_S2_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_1_74_CTGATCGT_S2_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 17:29:21 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 17:29:21\tMarkDuplicates\tStart of doWork freeMemory: 502528304; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:29:21\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 17:29:21\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 17:29:28\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:07s.  Time for last 1,000,000:    7s.  Last read position: 12:14,958,770\n",
      "INFO\t2021-05-07 17:29:28\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:29:33\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    5s.  Last read position: 16:53,767,903\n",
      "INFO\t2021-05-07 17:29:33\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:29:38\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    4s.  Last read position: 2:225,427,676\n",
      "INFO\t2021-05-07 17:29:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:29:49\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 4:139,504,918\n",
      "INFO\t2021-05-07 17:29:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:29:54\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    4s.  Last read position: 7:82,125,614\n",
      "INFO\t2021-05-07 17:29:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 5654202, Read name D00645:389:CE46UANXX:1:2103:16794:92381, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5654203, Read name D00645:389:CE46UANXX:1:1302:9846:82073, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:30:02\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    8s.  Last read position: GL000219.1:9,444\n",
      "INFO\t2021-05-07 17:30:02\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 6012528, Read name D00645:389:CE46UANXX:1:1303:18626:95178, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:30:02\tMarkDuplicates\tRead 6032939 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 17:30:03\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 750857648; totalMemory: 776470528; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:30:03\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 17:30:03\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:30:03\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:30:03\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 17:30:05\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 17:30:05\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 519180504; totalMemory: 770179072; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:30:05\tMarkDuplicates\tMarking 74607 records as duplicates.\n",
      "INFO\t2021-05-07 17:30:05\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 17:30:05\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 5654202, Read name D00645:389:CE46UANXX:1:2103:16794:92381, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5654203, Read name D00645:389:CE46UANXX:1:1302:9846:82073, MAPQ should be 0 for unmapped read.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Ignoring SAM validation error: ERROR: Record 6012528, Read name D00645:389:CE46UANXX:1:1303:18626:95178, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:30:50\tMarkDuplicates\tBefore output close freeMemory: 787604816; totalMemory: 801112064; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:30:50\tMarkDuplicates\tAfter output close freeMemory: 787604816; totalMemory: 801112064; maxMemory: 954728448\n",
      "[Fri May 07 17:30:51 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.50 minutes.\n",
      "Runtime.totalMemory()=801112064\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_19_88_CTTAGGAC_S19_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 53.85 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.68 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.02 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.89 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.68 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.20 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.68 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.36 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.68 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.04 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.70 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.19 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.05 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.55 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.56 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.43 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.50 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.71 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 28.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.71 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 9833881 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_19_88_CTTAGGAC_S19_R1_001.fastq.gz\n",
      "[main] Real time: 291.302 sec; CPU: 2086.767 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.28 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.54 sec\n",
      "[bwa_aln_core] print alignments... 0.15 sec\n",
      "[bwa_aln_core] 9833881 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_19_88_CTTAGGAC_S19_R1_001.fastq.gz\n",
      "[main] Real time: 295.280 sec; CPU: 187.531 sec\n",
      "[bam_sort_core] merging from 3 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 17:36:19\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_19_88_CTTAGGAC_S19_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_19_88_CTTAGGAC_S19_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_19_88_CTTAGGAC_S19_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "17:36:19.437 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 17:36:19 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_19_88_CTTAGGAC_S19_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_19_88_CTTAGGAC_S19_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_19_88_CTTAGGAC_S19_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 17:36:19 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 17:36:19\tMarkDuplicates\tStart of doWork freeMemory: 502528240; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:36:19\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 17:36:19\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 17:36:26\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 10:98,597,553\n",
      "INFO\t2021-05-07 17:36:26\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:36:31\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    5s.  Last read position: 13:52,917,687\n",
      "INFO\t2021-05-07 17:36:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:36:36\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    4s.  Last read position: 16:78,652,789\n",
      "INFO\t2021-05-07 17:36:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:36:47\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:27s.  Time for last 1,000,000:   11s.  Last read position: 2:129,740,807\n",
      "INFO\t2021-05-07 17:36:47\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:36:52\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:32s.  Time for last 1,000,000:    5s.  Last read position: 3:80,761,177\n",
      "INFO\t2021-05-07 17:36:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:37:00\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    8s.  Last read position: 5:55,611,863\n",
      "INFO\t2021-05-07 17:37:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:37:11\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:52s.  Time for last 1,000,000:   10s.  Last read position: 7:64,691,382\n",
      "INFO\t2021-05-07 17:37:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:37:16\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:56s.  Time for last 1,000,000:    4s.  Last read position: 9:97,583,145\n",
      "INFO\t2021-05-07 17:37:16\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 8418179, Read name D00645:389:CE46UANXX:1:1306:17745:67904, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8484663, Read name D00645:389:CE46UANXX:1:2315:1925:53109, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8484664, Read name D00645:389:CE46UANXX:1:2208:5740:97014, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8484751, Read name D00645:389:CE46UANXX:1:1205:14840:55783, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8487611, Read name D00645:389:CE46UANXX:1:1210:14473:10527, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:37:19\tMarkDuplicates\tRead 8488390 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 17:37:19\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 890986336; totalMemory: 916979712; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:37:19\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 17:37:20\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:37:20\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:37:20\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 17:37:21\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 17:37:21\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 665284432; totalMemory: 916455424; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:37:21\tMarkDuplicates\tMarking 88198 records as duplicates.\n",
      "INFO\t2021-05-07 17:37:21\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 17:37:21\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 8418179, Read name D00645:389:CE46UANXX:1:1306:17745:67904, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8484663, Read name D00645:389:CE46UANXX:1:2315:1925:53109, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8484664, Read name D00645:389:CE46UANXX:1:2208:5740:97014, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8484751, Read name D00645:389:CE46UANXX:1:1205:14840:55783, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8487611, Read name D00645:389:CE46UANXX:1:1210:14473:10527, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:38:21\tMarkDuplicates\tBefore output close freeMemory: 918665056; totalMemory: 932184064; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:38:21\tMarkDuplicates\tAfter output close freeMemory: 918665056; totalMemory: 932184064; maxMemory: 954728448\n",
      "[Fri May 07 17:38:21 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.04 minutes.\n",
      "Runtime.totalMemory()=932184064\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_5_152_CTTGTCGA_S9_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 55.92 sec\n",
      "[bwa_aln_core] write to the disk... 0.12 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 81.89 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.12 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.71 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.69 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.72 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.71 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.72 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.72 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.71 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.38 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.67 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.23 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.68 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.63 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.56 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.68 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.00 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.66 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.64 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.21 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.63 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.65 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.65 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.63 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.61 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.32 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.71 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.64 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.21 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.58 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.05 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.59 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.72 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.61 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.87 sec\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.68 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.64 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.65 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 1.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.00 sec\n",
      "[bwa_aln_core] 9180607 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_5_152_CTTGTCGA_S9_R1_001.fastq.gz\n",
      "[main] Real time: 278.451 sec; CPU: 2100.213 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 2.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.47 sec\n",
      "[bwa_aln_core] print alignments... 0.01 sec\n",
      "[bwa_aln_core] 9180607 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_5_152_CTTGTCGA_S9_R1_001.fastq.gz\n",
      "[main] Real time: 281.637 sec; CPU: 172.861 sec\n",
      "[bam_sort_core] merging from 3 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 17:43:32\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_5_152_CTTGTCGA_S9_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_5_152_CTTGTCGA_S9_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_5_152_CTTGTCGA_S9_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "17:43:32.998 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 17:43:33 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_5_152_CTTGTCGA_S9_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_5_152_CTTGTCGA_S9_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5710_5_152_CTTGTCGA_S9_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 17:43:33 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 17:43:33\tMarkDuplicates\tStart of doWork freeMemory: 502528272; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:43:33\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 17:43:33\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 17:43:37\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 11:12,459,507\n",
      "INFO\t2021-05-07 17:43:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:43:42\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    4s.  Last read position: 14:69,727,836\n",
      "INFO\t2021-05-07 17:43:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:43:47\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    4s.  Last read position: 19:6,336,144\n",
      "INFO\t2021-05-07 17:43:47\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:43:58\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:25s.  Time for last 1,000,000:   11s.  Last read position: 20:61,373,362\n",
      "INFO\t2021-05-07 17:43:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:44:04\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    5s.  Last read position: 4:120,047,819\n",
      "INFO\t2021-05-07 17:44:04\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:44:09\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:36s.  Time for last 1,000,000:    5s.  Last read position: 7:16,913,287\n",
      "INFO\t2021-05-07 17:44:09\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:44:19\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:45s.  Time for last 1,000,000:    9s.  Last read position: 9:92,094,452\n",
      "INFO\t2021-05-07 17:44:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 7144279, Read name D00645:389:CE46UANXX:1:1306:15271:85639, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7144280, Read name D00645:389:CE46UANXX:1:2204:7547:53763, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7144281, Read name D00645:389:CE46UANXX:1:1106:10250:31132, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7166013, Read name D00645:389:CE46UANXX:1:2316:3949:77423, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7166014, Read name D00645:389:CE46UANXX:1:1316:13475:22559, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7641249, Read name D00645:389:CE46UANXX:1:2111:11183:43178, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:44:22\tMarkDuplicates\tRead 7645460 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 17:44:23\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 604447976; totalMemory: 629145600; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:44:23\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 17:44:23\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:44:23\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:44:23\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 17:44:25\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 17:44:25\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 438816272; totalMemory: 688390144; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:44:25\tMarkDuplicates\tMarking 106729 records as duplicates.\n",
      "INFO\t2021-05-07 17:44:25\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 17:44:25\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 7144279, Read name D00645:389:CE46UANXX:1:1306:15271:85639, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7144280, Read name D00645:389:CE46UANXX:1:2204:7547:53763, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7144281, Read name D00645:389:CE46UANXX:1:1106:10250:31132, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7166013, Read name D00645:389:CE46UANXX:1:2316:3949:77423, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7166014, Read name D00645:389:CE46UANXX:1:1316:13475:22559, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7641249, Read name D00645:389:CE46UANXX:1:2111:11183:43178, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:45:21\tMarkDuplicates\tBefore output close freeMemory: 730095760; totalMemory: 743440384; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:45:21\tMarkDuplicates\tAfter output close freeMemory: 730620048; totalMemory: 743964672; maxMemory: 954728448\n",
      "[Fri May 07 17:45:21 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.82 minutes.\n",
      "Runtime.totalMemory()=743964672\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_10_110_TAACCGGT_S51_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 64.34 sec\n",
      "[bwa_aln_core] write to the disk... 0.04 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.60 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.19 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.06 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.36 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 73.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.72 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.14 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.26 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.94 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.19 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.07 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.80 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.71 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.14 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.21 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.38 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.23 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.71 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.83 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.72 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.05 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.61 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.56 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 13369344 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.89 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 13369344 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 13631488 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.25 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 13631488 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 13861212 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_10_110_TAACCGGT_S51_R1_001.fastq.gz\n",
      "[main] Real time: 443.304 sec; CPU: 3471.537 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 13861212 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_10_110_TAACCGGT_S51_R1_001.fastq.gz\n",
      "[main] Real time: 448.021 sec; CPU: 275.916 sec\n",
      "[bam_sort_core] merging from 4 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 17:53:33\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_10_110_TAACCGGT_S51_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_10_110_TAACCGGT_S51_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_10_110_TAACCGGT_S51_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "17:53:34.205 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 17:53:34 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_10_110_TAACCGGT_S51_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_10_110_TAACCGGT_S51_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_10_110_TAACCGGT_S51_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 17:53:34 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 17:53:34\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:53:34\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 17:53:34\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 17:53:40\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 10:36,953,040\n",
      "INFO\t2021-05-07 17:53:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:53:46\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    5s.  Last read position: 12:40,515,669\n",
      "INFO\t2021-05-07 17:53:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:53:50\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    4s.  Last read position: 14:99,580,990\n",
      "INFO\t2021-05-07 17:53:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:54:02\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 17:82,088,432\n",
      "INFO\t2021-05-07 17:54:02\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:54:08\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    5s.  Last read position: 2:131,128,547\n",
      "INFO\t2021-05-07 17:54:08\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:54:16\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    8s.  Last read position: 22:36,430,534\n",
      "INFO\t2021-05-07 17:54:16\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:54:27\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:53s.  Time for last 1,000,000:   10s.  Last read position: 4:78,252,464\n",
      "INFO\t2021-05-07 17:54:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:54:31\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:57s.  Time for last 1,000,000:    4s.  Last read position: 6:11,861,721\n",
      "INFO\t2021-05-07 17:54:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:54:36\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:01:02s.  Time for last 1,000,000:    4s.  Last read position: 7:101,238,704\n",
      "INFO\t2021-05-07 17:54:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 17:54:41\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:01:07s.  Time for last 1,000,000:    5s.  Last read position: 9:62,478,603\n",
      "INFO\t2021-05-07 17:54:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 10291341, Read name D00645:415:CE7Y1ANXX:5:1210:19356:73737, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10301313, Read name D00645:415:CE7Y1ANXX:5:2115:1250:75579, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10301314, Read name D00645:415:CE7Y1ANXX:5:1316:1316:36520, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10301315, Read name D00645:415:CE7Y1ANXX:5:1312:14733:5947, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10301316, Read name D00645:415:CE7Y1ANXX:5:1301:10705:90507, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10846777, Read name D00645:415:CE7Y1ANXX:5:1114:7554:47264, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10930401, Read name D00645:415:CE7Y1ANXX:5:2201:12677:27521, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10930696, Read name D00645:415:CE7Y1ANXX:5:1208:14609:64362, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10932701, Read name D00645:415:CE7Y1ANXX:5:2205:21116:17834, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:54:47\tMarkDuplicates\tRead 10934228 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 17:54:48\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 664000912; totalMemory: 689963008; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:54:48\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 17:54:48\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:54:48\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 17:54:48\tSortingCollection\tCreating merging iterator from 4 files\n",
      "INFO\t2021-05-07 17:54:50\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 17:54:50\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 454367664; totalMemory: 705167360; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:54:50\tMarkDuplicates\tMarking 124964 records as duplicates.\n",
      "INFO\t2021-05-07 17:54:50\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 17:54:50\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2021-05-07 17:55:53\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:02s.  Time for last 10,000,000:   62s.  Last read position: 9:62,478,603\n",
      "Ignoring SAM validation error: ERROR: Record 10291341, Read name D00645:415:CE7Y1ANXX:5:1210:19356:73737, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10301313, Read name D00645:415:CE7Y1ANXX:5:2115:1250:75579, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10301314, Read name D00645:415:CE7Y1ANXX:5:1316:1316:36520, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10301315, Read name D00645:415:CE7Y1ANXX:5:1312:14733:5947, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10301316, Read name D00645:415:CE7Y1ANXX:5:1301:10705:90507, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10846777, Read name D00645:415:CE7Y1ANXX:5:1114:7554:47264, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10930401, Read name D00645:415:CE7Y1ANXX:5:2201:12677:27521, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10930696, Read name D00645:415:CE7Y1ANXX:5:1208:14609:64362, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10932701, Read name D00645:415:CE7Y1ANXX:5:2205:21116:17834, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 17:56:12\tMarkDuplicates\tBefore output close freeMemory: 724181752; totalMemory: 737673216; maxMemory: 954728448\n",
      "INFO\t2021-05-07 17:56:12\tMarkDuplicates\tAfter output close freeMemory: 724181752; totalMemory: 737673216; maxMemory: 954728448\n",
      "[Fri May 07 17:56:12 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.64 minutes.\n",
      "Runtime.totalMemory()=737673216\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_11_111_CTACTTGG_S41_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 61.64 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.41 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.13 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 78.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.63 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.55 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.61 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.09 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.58 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.52 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.55 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.56 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.57 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.07 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.58 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.27 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.15 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.56 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.36 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.58 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.58 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.88 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.77 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.52 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.63 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.61 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.56 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 56.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.73 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.85 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.14 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.28 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.73 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 74.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.19 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.38 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.28 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.38 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.15 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.38 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.00 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.68 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.17 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.09 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.43 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.43 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.32 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.08 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.43 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 13369344 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.50 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 13369344 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 13631488 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.36 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.57 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 13631488 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 13893632 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 20.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.19 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 13893632 sequences have been processed.\n",
      "0.03 sec\n",
      "[bwa_aln_core] 13973326 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_11_111_CTACTTGG_S41_R1_001.fastq.gz[bwa_aln_core] convert to sequence coordinate... \n",
      "[main] Real time: 449.699 sec; CPU: 3494.680 sec\n",
      "3.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.09 sec\n",
      "[bwa_aln_core] 13973326 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_11_111_CTACTTGG_S41_R1_001.fastq.gz\n",
      "[main] Real time: 453.971 sec; CPU: 280.884 sec\n",
      "[bam_sort_core] merging from 4 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 18:04:31\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_11_111_CTACTTGG_S41_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_11_111_CTACTTGG_S41_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_11_111_CTACTTGG_S41_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "18:04:32.222 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 18:04:32 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_11_111_CTACTTGG_S41_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_11_111_CTACTTGG_S41_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_11_111_CTACTTGG_S41_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 18:04:32 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 18:04:32\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:04:32\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 18:04:32\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 18:04:38\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 10:41,474,268\n",
      "INFO\t2021-05-07 18:04:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:04:44\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    5s.  Last read position: 12:50,674,976\n",
      "INFO\t2021-05-07 18:04:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:04:49\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    5s.  Last read position: 14:42,830,716\n",
      "INFO\t2021-05-07 18:04:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:05:01\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 17:19,015,285\n",
      "INFO\t2021-05-07 18:05:01\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:05:06\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    5s.  Last read position: 2:68,244,200\n",
      "INFO\t2021-05-07 18:05:06\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:05:15\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    8s.  Last read position: 21:10,354,283\n",
      "INFO\t2021-05-07 18:05:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:05:20\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:48s.  Time for last 1,000,000:    5s.  Last read position: 3:192,154,032\n",
      "INFO\t2021-05-07 18:05:20\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:05:25\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:53s.  Time for last 1,000,000:    5s.  Last read position: 5:104,446,396\n",
      "INFO\t2021-05-07 18:05:25\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:05:31\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:58s.  Time for last 1,000,000:    5s.  Last read position: 7:9,060,628\n",
      "INFO\t2021-05-07 18:05:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:05:39\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:01:06s.  Time for last 1,000,000:    7s.  Last read position: 8:83,820,323\n",
      "INFO\t2021-05-07 18:05:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 10820299, Read name D00645:415:CE7Y1ANXX:5:1303:1772:59031, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10820300, Read name D00645:415:CE7Y1ANXX:5:2216:9994:10407, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:05:44\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:01:12s.  Time for last 1,000,000:    5s.  Last read position: X:65,990,457\n",
      "INFO\t2021-05-07 18:05:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Ignoring SAM validation error: ERROR: Record 11290458, Read name D00645:415:CE7Y1ANXX:5:2212:13853:75099, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 11374303, Read name D00645:415:CE7Y1ANXX:5:2103:14185:48754, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:05:46\tMarkDuplicates\tRead 11381876 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 18:05:46\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 756865632; totalMemory: 783286272; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:05:46\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 18:05:46\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:05:46\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:05:46\tSortingCollection\tCreating merging iterator from 4 files\n",
      "INFO\t2021-05-07 18:05:49\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 18:05:49\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 539361952; totalMemory: 790626304; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:05:49\tMarkDuplicates\tMarking 119493 records as duplicates.\n",
      "INFO\t2021-05-07 18:05:49\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 18:05:49\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2021-05-07 18:06:52\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:02s.  Time for last 10,000,000:   62s.  Last read position: 8:83,820,323\n",
      "Ignoring SAM validation error: ERROR: Record 10820299, Read name D00645:415:CE7Y1ANXX:5:1303:1772:59031, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10820300, Read name D00645:415:CE7Y1ANXX:5:2216:9994:10407, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 11290458, Read name D00645:415:CE7Y1ANXX:5:2212:13853:75099, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 11374303, Read name D00645:415:CE7Y1ANXX:5:2103:14185:48754, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:07:12\tMarkDuplicates\tBefore output close freeMemory: 794460672; totalMemory: 807927808; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:07:12\tMarkDuplicates\tAfter output close freeMemory: 794460672; totalMemory: 807927808; maxMemory: 954728448\n",
      "[Fri May 07 18:07:12 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.68 minutes.\n",
      "Runtime.totalMemory()=807927808\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_1_1_ATGCACGA_S42_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 55.54 sec\n",
      "[bwa_aln_core] write to the disk... 0.08 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.32 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.70 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.43 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.49 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.41 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.21 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.89 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.78 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.72 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.15 sec\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.05 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.52 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.36 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.33 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.31 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.29 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.19 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.55 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.77 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.00 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.77 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.66 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.07 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.12 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 12.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.65 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.03 sec\n",
      "[bwa_aln_core] 8964092 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_1_1_ATGCACGA_S42_R1_001.fastq.gz\n",
      "[main] Real time: 280.363 sec; CPU: 2056.038 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.20 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.48 sec\n",
      "[bwa_aln_core] print alignments... 0.05 sec\n",
      "[bwa_aln_core] 8964092 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_1_1_ATGCACGA_S42_R1_001.fastq.gz\n",
      "[main] Real time: 284.113 sec; CPU: 181.935 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 18:12:25\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_1_1_ATGCACGA_S42_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_1_1_ATGCACGA_S42_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_1_1_ATGCACGA_S42_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "18:12:25.810 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 18:12:25 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_1_1_ATGCACGA_S42_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_1_1_ATGCACGA_S42_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_1_1_ATGCACGA_S42_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 18:12:25 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 18:12:25\tMarkDuplicates\tStart of doWork freeMemory: 502528304; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:12:25\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 18:12:25\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 18:12:32\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 11:43,059,472\n",
      "INFO\t2021-05-07 18:12:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:12:37\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    4s.  Last read position: 14:94,656,732\n",
      "INFO\t2021-05-07 18:12:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:12:42\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    4s.  Last read position: 2:366,074\n",
      "INFO\t2021-05-07 18:12:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:12:54\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 3:12,238,457\n",
      "INFO\t2021-05-07 18:12:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:12:59\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    4s.  Last read position: 5:52,423,332\n",
      "INFO\t2021-05-07 18:12:59\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:13:08\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    9s.  Last read position: 7:107,143,881\n",
      "INFO\t2021-05-07 18:13:08\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:13:14\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:48s.  Time for last 1,000,000:    5s.  Last read position: X:148,957,846\n",
      "INFO\t2021-05-07 18:13:14\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 7061856, Read name D00645:415:CE7Y1ANXX:5:1309:9890:78459, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7085213, Read name D00645:415:CE7Y1ANXX:5:1205:16612:87689, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7085334, Read name D00645:415:CE7Y1ANXX:5:1312:1276:37949, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:13:14\tMarkDuplicates\tRead 7085814 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 18:13:14\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 731264984; totalMemory: 757596160; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:13:14\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 18:13:14\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:13:14\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:13:14\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 18:13:16\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 18:13:16\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 500592888; totalMemory: 751828992; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:13:16\tMarkDuplicates\tMarking 75738 records as duplicates.\n",
      "INFO\t2021-05-07 18:13:16\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 18:13:16\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 7061856, Read name D00645:415:CE7Y1ANXX:5:1309:9890:78459, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7085213, Read name D00645:415:CE7Y1ANXX:5:1205:16612:87689, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7085334, Read name D00645:415:CE7Y1ANXX:5:1312:1276:37949, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:14:09\tMarkDuplicates\tBefore output close freeMemory: 769664928; totalMemory: 783286272; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:14:09\tMarkDuplicates\tAfter output close freeMemory: 769664928; totalMemory: 783286272; maxMemory: 954728448\n",
      "[Fri May 07 18:14:09 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.73 minutes.\n",
      "Runtime.totalMemory()=783286272\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_12_132_CAACGGAT_S43_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 57.86 sec\n",
      "[bwa_aln_core] write to the disk... 0.08 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.45 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.69 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.75 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.32 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.06 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.65 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.42 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.25 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.24 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.19 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.32 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.52 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.10 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.07 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.64 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.19 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.20 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.67 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.21 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 53.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.67 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.06 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.43 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 16.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.62 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.02 sec\n",
      "[bwa_aln_core] 9776933 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa[bwa_aln_core] convert to sequence coordinate...  /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_12_132_CAACGGAT_S43_R1_001.fastq.gz\n",
      "[main] Real time: 308.074 sec; CPU: 2251.699 sec\n",
      "3.21 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.13 sec\n",
      "[bwa_aln_core] 9776933 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_12_132_CAACGGAT_S43_R1_001.fastq.gz\n",
      "[main] Real time: 312.081 sec; CPU: 196.842 sec\n",
      "[bam_sort_core] merging from 3 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 18:19:53\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_12_132_CAACGGAT_S43_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_12_132_CAACGGAT_S43_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_12_132_CAACGGAT_S43_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "18:19:53.349 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 18:19:53 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_12_132_CAACGGAT_S43_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_12_132_CAACGGAT_S43_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_12_132_CAACGGAT_S43_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 18:19:53 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 18:19:53\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:19:53\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 18:19:53\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 18:20:00\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 10:100,807,061\n",
      "INFO\t2021-05-07 18:20:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:20:05\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    5s.  Last read position: 13:79,593,148\n",
      "INFO\t2021-05-07 18:20:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:20:11\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    5s.  Last read position: 17:66,424,658\n",
      "INFO\t2021-05-07 18:20:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:20:22\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:29s.  Time for last 1,000,000:   11s.  Last read position: 2:241,922,628\n",
      "INFO\t2021-05-07 18:20:22\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:20:27\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:34s.  Time for last 1,000,000:    5s.  Last read position: 4:59,254,533\n",
      "INFO\t2021-05-07 18:20:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:20:35\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    7s.  Last read position: 6:107,703,120\n",
      "INFO\t2021-05-07 18:20:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 18:20:46\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:52s.  Time for last 1,000,000:   11s.  Last read position: 8:118,166,789\n",
      "INFO\t2021-05-07 18:20:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 7415094, Read name D00645:415:CE7Y1ANXX:5:1102:5404:7969, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7415095, Read name D00645:415:CE7Y1ANXX:5:2210:8778:48668, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7427491, Read name D00645:415:CE7Y1ANXX:5:1101:11286:62001, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7705816, Read name D00645:415:CE7Y1ANXX:5:2108:1508:34725, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7763926, Read name D00645:415:CE7Y1ANXX:5:2116:6451:14668, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7765675, Read name D00645:415:CE7Y1ANXX:5:2203:20237:51650, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7766007, Read name D00645:415:CE7Y1ANXX:5:1114:7124:27520, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:20:50\tMarkDuplicates\tRead 7766644 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 18:20:50\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 867080296; totalMemory: 892862464; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:20:50\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 18:20:51\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:20:51\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:20:51\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 18:20:52\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 18:20:52\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 634413200; totalMemory: 885522432; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:20:52\tMarkDuplicates\tMarking 95163 records as duplicates.\n",
      "INFO\t2021-05-07 18:20:52\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 18:20:52\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 7415094, Read name D00645:415:CE7Y1ANXX:5:1102:5404:7969, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7415095, Read name D00645:415:CE7Y1ANXX:5:2210:8778:48668, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7427491, Read name D00645:415:CE7Y1ANXX:5:1101:11286:62001, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7705816, Read name D00645:415:CE7Y1ANXX:5:2108:1508:34725, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7763926, Read name D00645:415:CE7Y1ANXX:5:2116:6451:14668, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7765675, Read name D00645:415:CE7Y1ANXX:5:2203:20237:51650, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7766007, Read name D00645:415:CE7Y1ANXX:5:1114:7124:27520, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:21:50\tMarkDuplicates\tBefore output close freeMemory: 903931344; totalMemory: 917504000; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:21:50\tMarkDuplicates\tAfter output close freeMemory: 903931344; totalMemory: 917504000; maxMemory: 954728448\n",
      "[Fri May 07 18:21:50 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.96 minutes.\n",
      "Runtime.totalMemory()=917504000\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_13_142_CATGGCTA_S46_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 52.37 sec\n",
      "[bwa_aln_core] write to the disk... 0.07 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.60 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.83 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.16 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.11 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.11 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.91 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.55 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.10 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.52 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.68 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.44 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.02 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.46 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.12 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.95 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.43 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.88 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.44 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.42 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.42 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.43 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.69 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.11 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.25 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.72 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.24 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 9627339 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2[bwa_aln_core] convert to sequence coordinate...  /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_13_142_CATGGCTA_S46_R1_001.fastq.gz\n",
      "[main] Real time: 296.941 sec; CPU: 2017.290 sec\n",
      "4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.21 sec\n",
      "[bwa_aln_core] 9627339 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_13_142_CATGGCTA_S46_R1_001.fastq.gz\n",
      "[main] Real time: 301.768 sec; CPU: 194.288 sec\n",
      "[bam_sort_core] merging from 3 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 18:27:23\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_13_142_CATGGCTA_S46_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_13_142_CATGGCTA_S46_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_13_142_CATGGCTA_S46_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "18:27:23.808 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 18:27:23 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_13_142_CATGGCTA_S46_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_13_142_CATGGCTA_S46_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_13_142_CATGGCTA_S46_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 18:27:23 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 18:27:23\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:27:23\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 18:27:23\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 18:27:29\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 10:117,499,782\n",
      "INFO\t2021-05-07 18:27:29\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:27:34\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: 13:91,165,824\n",
      "INFO\t2021-05-07 18:27:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:27:40\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    5s.  Last read position: 17:65,378,582\n",
      "INFO\t2021-05-07 18:27:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:27:51\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:27s.  Time for last 1,000,000:   11s.  Last read position: 2:218,089,310\n",
      "INFO\t2021-05-07 18:27:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:27:56\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:32s.  Time for last 1,000,000:    5s.  Last read position: 3:184,756,349\n",
      "INFO\t2021-05-07 18:27:56\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:28:05\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    9s.  Last read position: 5:178,668,620\n",
      "INFO\t2021-05-07 18:28:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:28:16\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:52s.  Time for last 1,000,000:   10s.  Last read position: 8:37,864,537\n",
      "INFO\t2021-05-07 18:28:16\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 7675388, Read name D00645:415:CE7Y1ANXX:5:2305:8889:32408, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7682955, Read name D00645:415:CE7Y1ANXX:5:2314:8577:64394, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:28:21\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:57s.  Time for last 1,000,000:    5s.  Last read position: GL000220.1:141,586\n",
      "INFO\t2021-05-07 18:28:21\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:28:22\tMarkDuplicates\tRead 8034010 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 18:28:22\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 869676752; totalMemory: 896532480; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:28:22\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 18:28:22\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:28:22\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:28:22\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 18:28:24\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 18:28:24\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 636847128; totalMemory: 888143872; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:28:24\tMarkDuplicates\tMarking 71565 records as duplicates.\n",
      "INFO\t2021-05-07 18:28:24\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 18:28:24\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 7675388, Read name D00645:415:CE7Y1ANXX:5:2305:8889:32408, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7682955, Read name D00645:415:CE7Y1ANXX:5:2314:8577:64394, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:29:22\tMarkDuplicates\tBefore output close freeMemory: 906550776; totalMemory: 920125440; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:29:22\tMarkDuplicates\tAfter output close freeMemory: 906550776; totalMemory: 920125440; maxMemory: 954728448\n",
      "[Fri May 07 18:29:22 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.99 minutes.\n",
      "Runtime.totalMemory()=920125440\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_14_162_AGCGTGTT_S55_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 55.71 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.58 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.74 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.60 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.60 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.38 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.19 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.36 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.05 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.29 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.02 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.20 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.91 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.06 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.16 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.03 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.95 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.39 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.45 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.16 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.00 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.08 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.83 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.28 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.32 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 13369344 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 13369344 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 13631488 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 13631488 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 13893632 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 13893632 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 14155776 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.17 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 14155776 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 14417920 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.85 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 14417920 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 14680064 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.04 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 14680064 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 14942208 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 14942208 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 15181357 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln[bwa_aln_core] convert to sequence coordinate...  -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_14_162_AGCGTGTT_S55_R1_001.fastq.gz\n",
      "[main] Real time: 467.717 sec; CPU: 3231.156 sec\n",
      "4.20 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 15181357 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_14_162_AGCGTGTT_S55_R1_001.fastq.gz\n",
      "[main] Real time: 478.982 sec; CPU: 306.136 sec\n",
      "[bam_sort_core] merging from 5 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 18:38:11\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_14_162_AGCGTGTT_S55_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_14_162_AGCGTGTT_S55_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_14_162_AGCGTGTT_S55_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "18:38:11.530 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 18:38:11 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_14_162_AGCGTGTT_S55_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_14_162_AGCGTGTT_S55_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_14_162_AGCGTGTT_S55_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 18:38:11 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 18:38:11\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:38:11\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 18:38:11\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 18:38:17\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 1:225,521,805\n",
      "INFO\t2021-05-07 18:38:17\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 18:38:22\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: 11:72,808,846\n",
      "INFO\t2021-05-07 18:38:22\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:38:26\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    4s.  Last read position: 13:43,362,932\n",
      "INFO\t2021-05-07 18:38:26\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:38:38\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:26s.  Time for last 1,000,000:   11s.  Last read position: 16:9,428,864\n",
      "INFO\t2021-05-07 18:38:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:38:43\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    4s.  Last read position: 18:76,166,582\n",
      "INFO\t2021-05-07 18:38:43\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:38:51\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:40s.  Time for last 1,000,000:    8s.  Last read position: 2:160,166,718\n",
      "INFO\t2021-05-07 18:38:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:38:58\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:46s.  Time for last 1,000,000:    6s.  Last read position: 22:27,982,795\n",
      "INFO\t2021-05-07 18:38:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:39:03\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:51s.  Time for last 1,000,000:    5s.  Last read position: 4:18,837,703\n",
      "INFO\t2021-05-07 18:39:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:39:08\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:56s.  Time for last 1,000,000:    5s.  Last read position: 5:90,009,449\n",
      "INFO\t2021-05-07 18:39:08\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:39:16\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:01:04s.  Time for last 1,000,000:    7s.  Last read position: 6:146,065,817\n",
      "INFO\t2021-05-07 18:39:16\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:39:22\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:01:10s.  Time for last 1,000,000:    5s.  Last read position: 8:54,646,273\n",
      "INFO\t2021-05-07 18:39:22\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:39:27\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:01:15s.  Time for last 1,000,000:    5s.  Last read position: X:26,456,253\n",
      "INFO\t2021-05-07 18:39:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 12424525, Read name D00645:415:CE7Y1ANXX:5:2213:17224:96002, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12453606, Read name D00645:415:CE7Y1ANXX:5:2212:12792:75889, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12496251, Read name D00645:415:CE7Y1ANXX:5:1211:17954:37945, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12507585, Read name D00645:415:CE7Y1ANXX:5:1301:8858:8990, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12511075, Read name D00645:415:CE7Y1ANXX:5:1111:11478:13714, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:39:30\tMarkDuplicates\tRead 12512092 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 18:39:31\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 781317760; totalMemory: 807403520; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:39:31\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 18:39:31\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:39:31\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:39:31\tSortingCollection\tCreating merging iterator from 4 files\n",
      "INFO\t2021-05-07 18:39:33\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 18:39:33\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 563458560; totalMemory: 814743552; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:39:33\tMarkDuplicates\tMarking 125314 records as duplicates.\n",
      "INFO\t2021-05-07 18:39:33\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 18:39:33\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2021-05-07 18:40:36\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:02s.  Time for last 10,000,000:   62s.  Last read position: 6:146,065,817\n",
      "Ignoring SAM validation error: ERROR: Record 12424525, Read name D00645:415:CE7Y1ANXX:5:2213:17224:96002, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12453606, Read name D00645:415:CE7Y1ANXX:5:2212:12792:75889, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12496251, Read name D00645:415:CE7Y1ANXX:5:1211:17954:37945, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12507585, Read name D00645:415:CE7Y1ANXX:5:1301:8858:8990, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12511075, Read name D00645:415:CE7Y1ANXX:5:1111:11478:13714, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:41:04\tMarkDuplicates\tBefore output close freeMemory: 817538864; totalMemory: 830996480; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:41:04\tMarkDuplicates\tAfter output close freeMemory: 817538864; totalMemory: 830996480; maxMemory: 954728448\n",
      "[Fri May 07 18:41:04 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.88 minutes.\n",
      "Runtime.totalMemory()=830996480\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_15_181_TTGGACGT_S50_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 48.40 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.07 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.05 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.77 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.32 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.89 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.23 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.89 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 21.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.03 sec\n",
      "[bwa_aln_core] 6921964 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_15_181_TTGGACGT_S50_R1_001.fastq.gz\n",
      "[main] Real time: 209.275 sec; CPU: 1342.180 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.23 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.12 sec\n",
      "[bwa_aln_core] 6921964 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_15_181_TTGGACGT_S50_R1_001.fastq.gz\n",
      "[main] Real time: 213.204 sec; CPU: 139.559 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 18:45:00\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_15_181_TTGGACGT_S50_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_15_181_TTGGACGT_S50_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_15_181_TTGGACGT_S50_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "18:45:00.865 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 18:45:00 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_15_181_TTGGACGT_S50_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_15_181_TTGGACGT_S50_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_15_181_TTGGACGT_S50_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 18:45:00 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 18:45:00\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:45:00\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 18:45:00\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 18:45:07\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 11:127,974,949\n",
      "INFO\t2021-05-07 18:45:07\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:45:12\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    4s.  Last read position: 16:49,777,122\n",
      "INFO\t2021-05-07 18:45:12\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:45:17\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    4s.  Last read position: 2:236,460,069\n",
      "INFO\t2021-05-07 18:45:17\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:45:28\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:27s.  Time for last 1,000,000:   10s.  Last read position: 4:139,627,531\n",
      "INFO\t2021-05-07 18:45:28\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:45:32\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    4s.  Last read position: 7:66,844,065\n",
      "INFO\t2021-05-07 18:45:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 5750759, Read name D00645:415:CE7Y1ANXX:5:1214:17003:71085, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5750760, Read name D00645:415:CE7Y1ANXX:5:2103:1927:70963, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5750761, Read name D00645:415:CE7Y1ANXX:5:1203:3400:87289, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5764651, Read name D00645:415:CE7Y1ANXX:5:2309:10588:39116, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5979158, Read name D00645:415:CE7Y1ANXX:5:1205:13513:30187, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:45:41\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:40s.  Time for last 1,000,000:    8s.  Last read position: KI270749.1:96,756\n",
      "INFO\t2021-05-07 18:45:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 6013787, Read name D00645:415:CE7Y1ANXX:5:1211:9269:4177, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6019850, Read name D00645:415:CE7Y1ANXX:5:2211:17213:9949, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6019851, Read name D00645:415:CE7Y1ANXX:5:1204:11973:89540, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:45:41\tMarkDuplicates\tRead 6020215 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 18:45:42\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 747128240; totalMemory: 772800512; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:45:42\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 18:45:42\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:45:42\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:45:42\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 18:45:43\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 18:45:43\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 530720392; totalMemory: 781713408; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:45:43\tMarkDuplicates\tMarking 66472 records as duplicates.\n",
      "INFO\t2021-05-07 18:45:43\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 18:45:43\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 5750759, Read name D00645:415:CE7Y1ANXX:5:1214:17003:71085, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5750760, Read name D00645:415:CE7Y1ANXX:5:2103:1927:70963, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5750761, Read name D00645:415:CE7Y1ANXX:5:1203:3400:87289, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5764651, Read name D00645:415:CE7Y1ANXX:5:2309:10588:39116, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5979158, Read name D00645:415:CE7Y1ANXX:5:1205:13513:30187, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6013787, Read name D00645:415:CE7Y1ANXX:5:1211:9269:4177, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6019850, Read name D00645:415:CE7Y1ANXX:5:2211:17213:9949, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6019851, Read name D00645:415:CE7Y1ANXX:5:1204:11973:89540, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:46:25\tMarkDuplicates\tBefore output close freeMemory: 782321648; totalMemory: 795869184; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:46:25\tMarkDuplicates\tAfter output close freeMemory: 782321648; totalMemory: 795869184; maxMemory: 954728448\n",
      "[Fri May 07 18:46:25 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.41 minutes.\n",
      "Runtime.totalMemory()=795869184\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_16_186_TTGCAGAC_S56_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 49.89 sec\n",
      "[bwa_aln_core] write to the disk... 0.06 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.94 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 51.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.41 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.03 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.39 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.30 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.25 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.91 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.23 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.91 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.31 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.19 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.85 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.71 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.38 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.60 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.24 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.37 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.27 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.33 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.28 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.31 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.06 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7059524 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_16_186_TTGCAGAC_S56_R1_001.fastq.gz\n",
      "[main] Real time: 214.156 sec; CPU: 1399.791 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 7059524 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_16_186_TTGCAGAC_S56_R1_001.fastq.gz\n",
      "[main] Real time: 218.802 sec; CPU: 144.665 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 18:50:27\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_16_186_TTGCAGAC_S56_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_16_186_TTGCAGAC_S56_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_16_186_TTGCAGAC_S56_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "18:50:27.686 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 18:50:27 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_16_186_TTGCAGAC_S56_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_16_186_TTGCAGAC_S56_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_16_186_TTGCAGAC_S56_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 18:50:27 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 18:50:27\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:50:27\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 18:50:27\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 18:50:35\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:07s.  Time for last 1,000,000:    7s.  Last read position: 11:119,771,751\n",
      "INFO\t2021-05-07 18:50:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:50:40\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    5s.  Last read position: 16:81,793,503\n",
      "INFO\t2021-05-07 18:50:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:50:46\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:18s.  Time for last 1,000,000:    5s.  Last read position: 20:19,553,127\n",
      "INFO\t2021-05-07 18:50:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:50:58\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:30s.  Time for last 1,000,000:   11s.  Last read position: 4:171,599,395\n",
      "INFO\t2021-05-07 18:50:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:51:04\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:36s.  Time for last 1,000,000:    5s.  Last read position: 7:106,603,486\n",
      "INFO\t2021-05-07 18:51:04\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 5659537, Read name D00645:415:CE7Y1ANXX:5:1302:10982:68626, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5659538, Read name D00645:415:CE7Y1ANXX:5:1305:12118:21759, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:51:11\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:44s.  Time for last 1,000,000:    7s.  Last read position: KI270438.1:6,850\n",
      "INFO\t2021-05-07 18:51:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:51:11\tMarkDuplicates\tRead 6019865 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 18:51:12\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 732532096; totalMemory: 758120448; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:51:12\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 18:51:12\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:51:12\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:51:12\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 18:51:14\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 18:51:14\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 516051120; totalMemory: 767033344; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:51:14\tMarkDuplicates\tMarking 62164 records as duplicates.\n",
      "INFO\t2021-05-07 18:51:14\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 18:51:14\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 5659537, Read name D00645:415:CE7Y1ANXX:5:1302:10982:68626, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5659538, Read name D00645:415:CE7Y1ANXX:5:1305:12118:21759, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:51:56\tMarkDuplicates\tBefore output close freeMemory: 767728520; totalMemory: 781189120; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:51:56\tMarkDuplicates\tAfter output close freeMemory: 767728520; totalMemory: 781189120; maxMemory: 954728448\n",
      "[Fri May 07 18:51:56 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.48 minutes.\n",
      "Runtime.totalMemory()=781189120\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_17_203_TACGCCTT_S57_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 50.08 sec\n",
      "[bwa_aln_core] write to the disk... 0.06 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.27 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.31 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.05 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.69 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.71 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.89 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.23 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.17 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.57 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.39 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.23 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.28 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.10 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.83 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.20 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.61 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.52 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.79 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.77 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.59 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.75 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.55 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.75 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.23 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.23 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.34 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.07 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.04 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.46 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 26.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 8267462 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_17_203_TACGCCTT_S57_R1_001.fastq.gz\n",
      "[main] Real time: 257.920 sec; CPU: 1614.453 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.41 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.55 sec\n",
      "[bwa_aln_core] print alignments... 0.16 sec\n",
      "[bwa_aln_core] 8267462 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_17_203_TACGCCTT_S57_R1_001.fastq.gz\n",
      "[main] Real time: 262.042 sec; CPU: 176.949 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 18:56:45\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_17_203_TACGCCTT_S57_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_17_203_TACGCCTT_S57_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_17_203_TACGCCTT_S57_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "18:56:45.700 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 18:56:45 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_17_203_TACGCCTT_S57_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_17_203_TACGCCTT_S57_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_17_203_TACGCCTT_S57_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 18:56:45 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 18:56:45\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:56:45\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 18:56:45\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 18:56:52\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 11:21,307,956\n",
      "INFO\t2021-05-07 18:56:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:56:57\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    5s.  Last read position: 14:98,231,082\n",
      "INFO\t2021-05-07 18:56:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:57:03\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    5s.  Last read position: 19:54,772,491\n",
      "INFO\t2021-05-07 18:57:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:57:14\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 3:19,075,705\n",
      "INFO\t2021-05-07 18:57:14\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:57:19\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    5s.  Last read position: 5:62,307,793\n",
      "INFO\t2021-05-07 18:57:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 18:57:27\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    8s.  Last read position: 7:108,922,366\n",
      "INFO\t2021-05-07 18:57:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 6776947, Read name D00645:415:CE7Y1ANXX:5:1207:1827:83455, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6784687, Read name D00645:415:CE7Y1ANXX:5:2113:18869:35704, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6784688, Read name D00645:415:CE7Y1ANXX:5:1111:1232:39607, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:57:38\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:52s.  Time for last 1,000,000:   10s.  Last read position: X:99,245,049\n",
      "INFO\t2021-05-07 18:57:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Ignoring SAM validation error: ERROR: Record 7140760, Read name D00645:415:CE7Y1ANXX:5:2207:10018:92352, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7140761, Read name D00645:415:CE7Y1ANXX:5:2305:13407:81308, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7143171, Read name D00645:415:CE7Y1ANXX:5:2216:15027:3153, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:57:39\tMarkDuplicates\tRead 7198227 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 18:57:39\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 871184296; totalMemory: 896532480; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:57:39\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 18:57:39\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:57:39\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 18:57:39\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 18:57:41\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 18:57:41\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 638170656; totalMemory: 889192448; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:57:41\tMarkDuplicates\tMarking 93840 records as duplicates.\n",
      "INFO\t2021-05-07 18:57:41\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 18:57:41\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 6776947, Read name D00645:415:CE7Y1ANXX:5:1207:1827:83455, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6784687, Read name D00645:415:CE7Y1ANXX:5:2113:18869:35704, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6784688, Read name D00645:415:CE7Y1ANXX:5:1111:1232:39607, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7140760, Read name D00645:415:CE7Y1ANXX:5:2207:10018:92352, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7140761, Read name D00645:415:CE7Y1ANXX:5:2305:13407:81308, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7143171, Read name D00645:415:CE7Y1ANXX:5:2216:15027:3153, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 18:58:32\tMarkDuplicates\tBefore output close freeMemory: 907063888; totalMemory: 920649728; maxMemory: 954728448\n",
      "INFO\t2021-05-07 18:58:32\tMarkDuplicates\tAfter output close freeMemory: 907063888; totalMemory: 920649728; maxMemory: 954728448\n",
      "[Fri May 07 18:58:32 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.78 minutes.\n",
      "Runtime.totalMemory()=920649728\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_18_221_AACCGTTC_S58_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 51.06 sec\n",
      "[bwa_aln_core] write to the disk... 0.06 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.83 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.73 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.58 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.23 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.52 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.03 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.25 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.38 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.27 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.54 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.38 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.43 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.71 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.78 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.09 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.20 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.41 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.72 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.77 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.55 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.14 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.12 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.23 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.17 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.21 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.29 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.64 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.99 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.19 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.57 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.61 sec\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.50 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.36 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.82 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.88 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.36 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.71 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.04 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.37 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.73 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.74 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.07 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.48 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.59 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 13369344 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.70 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.36 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 13369344 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 13631488 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.60 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.53 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 13631488 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 13893632 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 13893632 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 14143465 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_18_221_AACCGTTC_S58_R1_001.fastq.gz\n",
      "[main] Real time: 442.708 sec; CPU: 2874.194 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 4.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 14143465 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_18_221_AACCGTTC_S58_R1_001.fastq.gz\n",
      "[main] Real time: 448.566 sec; CPU: 296.849 sec\n",
      "[bam_sort_core] merging from 4 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 19:06:46\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_18_221_AACCGTTC_S58_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_18_221_AACCGTTC_S58_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_18_221_AACCGTTC_S58_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "19:06:46.463 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 19:06:46 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_18_221_AACCGTTC_S58_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_18_221_AACCGTTC_S58_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_18_221_AACCGTTC_S58_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 19:06:46 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 19:06:46\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:06:46\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 19:06:46\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 19:06:51\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 10:12,649,031\n",
      "INFO\t2021-05-07 19:06:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:06:57\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: 11:128,729,428\n",
      "INFO\t2021-05-07 19:06:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:07:01\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    4s.  Last read position: 14:34,876,956\n",
      "INFO\t2021-05-07 19:07:01\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:07:13\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:26s.  Time for last 1,000,000:   11s.  Last read position: 16:83,285,453\n",
      "INFO\t2021-05-07 19:07:13\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:07:19\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:32s.  Time for last 1,000,000:    5s.  Last read position: 2:20,411,112\n",
      "INFO\t2021-05-07 19:07:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:07:27\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    8s.  Last read position: 20:23,175,507\n",
      "INFO\t2021-05-07 19:07:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:07:37\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:51s.  Time for last 1,000,000:   10s.  Last read position: 3:101,921,057\n",
      "INFO\t2021-05-07 19:07:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:07:42\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:56s.  Time for last 1,000,000:    4s.  Last read position: 4:174,317,010\n",
      "INFO\t2021-05-07 19:07:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:07:47\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:01:01s.  Time for last 1,000,000:    5s.  Last read position: 6:50,936,521\n",
      "INFO\t2021-05-07 19:07:47\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:07:53\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:01:06s.  Time for last 1,000,000:    5s.  Last read position: 7:119,066,537\n",
      "INFO\t2021-05-07 19:07:53\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:08:00\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:01:13s.  Time for last 1,000,000:    6s.  Last read position: 9:61,529,497\n",
      "INFO\t2021-05-07 19:08:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 11326810, Read name D00645:415:CE7Y1ANXX:5:1106:8121:83984, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 11326811, Read name D00645:415:CE7Y1ANXX:5:2103:12391:95644, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 11335737, Read name D00645:415:CE7Y1ANXX:5:2106:15472:87436, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:08:05\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:01:19s.  Time for last 1,000,000:    5s.  Last read position: KI270706.1:33,217\n",
      "INFO\t2021-05-07 19:08:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 12057321, Read name D00645:415:CE7Y1ANXX:5:1316:17536:32389, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12057522, Read name D00645:415:CE7Y1ANXX:5:1106:9483:79034, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12058247, Read name D00645:415:CE7Y1ANXX:5:2211:18624:88932, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12060984, Read name D00645:415:CE7Y1ANXX:5:2214:11480:73507, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12061610, Read name D00645:415:CE7Y1ANXX:5:2208:10473:22825, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:08:05\tMarkDuplicates\tRead 12061996 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 19:08:06\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 701500584; totalMemory: 727711744; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:08:06\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 19:08:06\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:08:06\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:08:06\tSortingCollection\tCreating merging iterator from 4 files\n",
      "INFO\t2021-05-07 19:08:08\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 19:08:08\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 483249376; totalMemory: 734527488; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:08:08\tMarkDuplicates\tMarking 93509 records as duplicates.\n",
      "INFO\t2021-05-07 19:08:08\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 19:08:08\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2021-05-07 19:09:10\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:01s.  Time for last 10,000,000:   61s.  Last read position: 7:119,066,537\n",
      "Ignoring SAM validation error: ERROR: Record 11326810, Read name D00645:415:CE7Y1ANXX:5:1106:8121:83984, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 11326811, Read name D00645:415:CE7Y1ANXX:5:2103:12391:95644, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 11335737, Read name D00645:415:CE7Y1ANXX:5:2106:15472:87436, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12057321, Read name D00645:415:CE7Y1ANXX:5:1316:17536:32389, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12057522, Read name D00645:415:CE7Y1ANXX:5:1106:9483:79034, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12058247, Read name D00645:415:CE7Y1ANXX:5:2211:18624:88932, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12060984, Read name D00645:415:CE7Y1ANXX:5:2214:11480:73507, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 12061610, Read name D00645:415:CE7Y1ANXX:5:2208:10473:22825, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:09:32\tMarkDuplicates\tBefore output close freeMemory: 737743072; totalMemory: 751304704; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:09:32\tMarkDuplicates\tAfter output close freeMemory: 737743072; totalMemory: 751304704; maxMemory: 954728448\n",
      "[Fri May 07 19:09:32 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.77 minutes.\n",
      "Runtime.totalMemory()=751304704\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_19_229_CCTCAGTT_S59_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 46.73 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.85 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.56 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.73 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.08 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.22 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.71 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.36 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.69 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.21 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.40 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.42 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.63 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.20 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.57 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.77 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.80 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.20 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.69 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.74 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.37 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.14 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.39 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.05 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.33 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.32 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 45.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4950144 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_19_229_CCTCAGTT_S59_R1_001.fastq.gz\n",
      "[main] Real time: 156.192 sec; CPU: 986.960 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 4.28 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 4950144 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_19_229_CCTCAGTT_S59_R1_001.fastq.gz\n",
      "[main] Real time: 161.390 sec; CPU: 109.172 sec\n",
      "[bam_sort_core] merging from 1 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 19:12:30\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_19_229_CCTCAGTT_S59_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_19_229_CCTCAGTT_S59_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_19_229_CCTCAGTT_S59_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "19:12:31.124 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 19:12:31 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_19_229_CCTCAGTT_S59_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_19_229_CCTCAGTT_S59_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_19_229_CCTCAGTT_S59_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 19:12:31 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 19:12:31\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:12:31\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 19:12:31\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 19:12:37\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 13:33,182,365\n",
      "INFO\t2021-05-07 19:12:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:12:43\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    5s.  Last read position: 2:116,036,386\n",
      "INFO\t2021-05-07 19:12:43\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:12:48\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    5s.  Last read position: 5:19,799,780\n",
      "INFO\t2021-05-07 19:12:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:12:59\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 9:107,924,250\n",
      "INFO\t2021-05-07 19:12:59\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 4049642, Read name D00645:415:CE7Y1ANXX:5:2302:5908:100736, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4049643, Read name D00645:415:CE7Y1ANXX:5:2309:11573:22967, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4049644, Read name D00645:415:CE7Y1ANXX:5:1201:17947:36884, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4057841, Read name D00645:415:CE7Y1ANXX:5:2206:1966:56490, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4290952, Read name D00645:415:CE7Y1ANXX:5:1210:18482:55091, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4292914, Read name D00645:415:CE7Y1ANXX:5:1207:20869:72062, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:13:01\tMarkDuplicates\tRead 4292919 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 19:13:02\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 637620304; totalMemory: 661651456; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:13:02\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 19:13:02\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:13:02\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:13:02\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 19:13:03\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 19:13:04\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 411574792; totalMemory: 660602880; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:13:04\tMarkDuplicates\tMarking 33448 records as duplicates.\n",
      "INFO\t2021-05-07 19:13:04\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 19:13:04\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 4049642, Read name D00645:415:CE7Y1ANXX:5:2302:5908:100736, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4049643, Read name D00645:415:CE7Y1ANXX:5:2309:11573:22967, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4049644, Read name D00645:415:CE7Y1ANXX:5:1201:17947:36884, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4057841, Read name D00645:415:CE7Y1ANXX:5:2206:1966:56490, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4290952, Read name D00645:415:CE7Y1ANXX:5:1210:18482:55091, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4292914, Read name D00645:415:CE7Y1ANXX:5:1207:20869:72062, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:13:33\tMarkDuplicates\tBefore output close freeMemory: 665918112; totalMemory: 678428672; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:13:33\tMarkDuplicates\tAfter output close freeMemory: 671160992; totalMemory: 683671552; maxMemory: 954728448\n",
      "[Fri May 07 19:13:33 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.05 minutes.\n",
      "Runtime.totalMemory()=683671552\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_20_272_TGGCTATC_S60_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 45.25 sec\n",
      "[bwa_aln_core] write to the disk... 0.06 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.02 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 5.42 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.73 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 44.88 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 42.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.38 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.34 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 42.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.44 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.21 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.21 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.77 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 42.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.33 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.06 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 41.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 47.95 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.94 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.09 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.78 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.52 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.67 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.35 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.50 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.14 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.19 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.26 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 43.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.51 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 18.88 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.24 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.74 sec\n",
      "[bwa_aln_core] print alignments... 0.37 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.02 sec\n",
      "[bwa_aln_core] 7699415 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_20_272_TGGCTATC_S60_R1_001.fastq.gz\n",
      "[main] Real time: 235.696 sec; CPU: 1376.237 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.11 sec\n",
      "[bwa_aln_core] 7699415 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_20_272_TGGCTATC_S60_R1_001.fastq.gz\n",
      "[main] Real time: 240.414 sec; CPU: 163.724 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 19:17:59\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_20_272_TGGCTATC_S60_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_20_272_TGGCTATC_S60_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_20_272_TGGCTATC_S60_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "19:17:59.849 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 19:17:59 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_20_272_TGGCTATC_S60_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_20_272_TGGCTATC_S60_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_20_272_TGGCTATC_S60_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 19:17:59 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 19:17:59\tMarkDuplicates\tStart of doWork freeMemory: 502528224; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:17:59\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 19:17:59\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 19:18:05\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 11:66,837,600\n",
      "INFO\t2021-05-07 19:18:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:18:11\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    5s.  Last read position: 15:42,569,871\n",
      "INFO\t2021-05-07 19:18:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:18:16\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    5s.  Last read position: 2:80,065,387\n",
      "INFO\t2021-05-07 19:18:16\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:18:27\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:27s.  Time for last 1,000,000:   10s.  Last read position: 3:127,410,885\n",
      "INFO\t2021-05-07 19:18:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:18:32\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:32s.  Time for last 1,000,000:    5s.  Last read position: 6:20,655,404\n",
      "INFO\t2021-05-07 19:18:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:18:40\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:40s.  Time for last 1,000,000:    7s.  Last read position: 8:106,611,063\n",
      "INFO\t2021-05-07 19:18:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 6424488, Read name D00645:415:CE7Y1ANXX:5:2109:4953:2523, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6424489, Read name D00645:415:CE7Y1ANXX:5:2315:19557:10207, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6424490, Read name D00645:415:CE7Y1ANXX:5:1215:11793:44066, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:18:43\tMarkDuplicates\tRead 6747764 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 19:18:46\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 875539352; totalMemory: 901251072; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:18:46\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 19:18:46\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:18:46\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:18:46\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 19:18:48\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 19:18:48\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 648630896; totalMemory: 899678208; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:18:48\tMarkDuplicates\tMarking 45024 records as duplicates.\n",
      "INFO\t2021-05-07 19:18:48\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 19:18:48\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 6424488, Read name D00645:415:CE7Y1ANXX:5:2109:4953:2523, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6424489, Read name D00645:415:CE7Y1ANXX:5:2315:19557:10207, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6424490, Read name D00645:415:CE7Y1ANXX:5:1215:11793:44066, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:19:34\tMarkDuplicates\tBefore output close freeMemory: 901839688; totalMemory: 915406848; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:19:34\tMarkDuplicates\tAfter output close freeMemory: 901839688; totalMemory: 915406848; maxMemory: 954728448\n",
      "[Fri May 07 19:19:34 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.59 minutes.\n",
      "Runtime.totalMemory()=915406848\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_2_30_GAACATCG_S45_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 58.06 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.32 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.68 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 76.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.44 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 60.78 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.65 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.66 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.41 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.21 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.05 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.03 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.83 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.23 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.05 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.50 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.94 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.00 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.23 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.00 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 62.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 12058624 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 12320768 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.96 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 12582912 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 12845056 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 30.24 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 13107200 sequences have been processed.\n",
      "0.03 sec\n",
      "[bwa_aln_core] 13233386 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_2_30_GAACATCG_S45_R1_001.fastq.gz\n",
      "[main] Real time: 418.077 sec; CPU: 3065.654 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.26 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.57 sec\n",
      "[bwa_aln_core] print alignments... 0.14 sec\n",
      "[bwa_aln_core] 13233386 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_2_30_GAACATCG_S45_R1_001.fastq.gz\n",
      "[main] Real time: 422.054 sec; CPU: 267.241 sec\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bam_sort_core] merging from 4 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 19:27:19\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_2_30_GAACATCG_S45_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_2_30_GAACATCG_S45_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_2_30_GAACATCG_S45_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "19:27:20.236 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 19:27:20 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_2_30_GAACATCG_S45_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_2_30_GAACATCG_S45_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_2_30_GAACATCG_S45_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 19:27:20 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 19:27:20\tMarkDuplicates\tStart of doWork freeMemory: 502528272; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:27:20\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 19:27:20\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 19:27:27\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:07s.  Time for last 1,000,000:    7s.  Last read position: 10:14,865,649\n",
      "INFO\t2021-05-07 19:27:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:27:32\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    5s.  Last read position: 12:12,779,276\n",
      "INFO\t2021-05-07 19:27:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:27:37\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    5s.  Last read position: 14:44,438,209\n",
      "INFO\t2021-05-07 19:27:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:27:49\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:29s.  Time for last 1,000,000:   12s.  Last read position: 17:32,332,795\n",
      "INFO\t2021-05-07 19:27:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:27:55\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:34s.  Time for last 1,000,000:    5s.  Last read position: 2:81,402,794\n",
      "INFO\t2021-05-07 19:27:55\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:28:03\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:43s.  Time for last 1,000,000:    8s.  Last read position: 21:29,517,929\n",
      "INFO\t2021-05-07 19:28:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:28:14\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:53s.  Time for last 1,000,000:   10s.  Last read position: 4:27,769,800\n",
      "INFO\t2021-05-07 19:28:14\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:28:19\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:58s.  Time for last 1,000,000:    4s.  Last read position: 5:131,744,711\n",
      "INFO\t2021-05-07 19:28:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:28:24\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:01:03s.  Time for last 1,000,000:    5s.  Last read position: 7:46,114,863\n",
      "INFO\t2021-05-07 19:28:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:28:29\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:01:09s.  Time for last 1,000,000:    5s.  Last read position: 8:102,310,467\n",
      "INFO\t2021-05-07 19:28:29\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 10635473, Read name D00645:415:CE7Y1ANXX:5:2306:16953:6460, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10985128, Read name D00645:415:CE7Y1ANXX:5:2314:14870:61677, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10989822, Read name D00645:415:CE7Y1ANXX:5:2301:5672:6635, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:28:35\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:01:15s.  Time for last 1,000,000:    6s.  Last read position: KI270733.1:152,089\n",
      "INFO\t2021-05-07 19:28:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:28:35\tMarkDuplicates\tRead 11063151 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 19:28:36\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 679870728; totalMemory: 706215936; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:28:36\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 19:28:36\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:28:36\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:28:36\tSortingCollection\tCreating merging iterator from 4 files\n",
      "INFO\t2021-05-07 19:28:38\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 19:28:38\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 454424632; totalMemory: 705691648; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:28:38\tMarkDuplicates\tMarking 141496 records as duplicates.\n",
      "INFO\t2021-05-07 19:28:38\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 19:28:38\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2021-05-07 19:29:40\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:01s.  Time for last 10,000,000:   61s.  Last read position: 8:102,310,467\n",
      "Ignoring SAM validation error: ERROR: Record 10635473, Read name D00645:415:CE7Y1ANXX:5:2306:16953:6460, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10985128, Read name D00645:415:CE7Y1ANXX:5:2314:14870:61677, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 10989822, Read name D00645:415:CE7Y1ANXX:5:2301:5672:6635, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:29:57\tMarkDuplicates\tBefore output close freeMemory: 708381408; totalMemory: 721944576; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:29:57\tMarkDuplicates\tAfter output close freeMemory: 708381408; totalMemory: 721944576; maxMemory: 954728448\n",
      "[Fri May 07 19:29:57 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.61 minutes.\n",
      "Runtime.totalMemory()=721944576\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_3_32_TCACAGCA_S44_R1_001.markdup.bam}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 74.41 sec\n",
      "[bwa_aln_core] write to the disk... 0.06 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 73.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 78.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 83.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 88.39 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 76.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 77.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.61 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 82.05 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.68 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 78.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.67 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 79.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.70 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 80.10 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 80.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.69 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 79.50 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 78.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 77.70 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 79.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.73 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 79.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 79.99 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 79.15 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 78.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.76 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 78.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 77.72 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.74 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 79.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.82 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 78.09 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 76.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 74.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 78.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 74.28 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 79.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.75 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.25 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 77.78 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.78 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 61.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.80 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.26 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 8072373 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_3_32_TCACAGCA_S44_R1_001.fastq.gz[bwa_aln_core] convert to sequence coordinate... \n",
      "[main] Real time: 261.337 sec; CPU: 2439.548 sec\n",
      "3.55 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.21 sec\n",
      "[bwa_aln_core] 8072373 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_3_32_TCACAGCA_S44_R1_001.fastq.gz\n",
      "[main] Real time: 265.735 sec; CPU: 153.355 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 19:34:47\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_3_32_TCACAGCA_S44_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_3_32_TCACAGCA_S44_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_3_32_TCACAGCA_S44_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "19:34:48.043 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 19:34:48 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_3_32_TCACAGCA_S44_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_3_32_TCACAGCA_S44_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_3_32_TCACAGCA_S44_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 19:34:48 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 19:34:48\tMarkDuplicates\tStart of doWork freeMemory: 502528272; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:34:48\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 19:34:48\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 19:34:54\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 12:90,630,800\n",
      "INFO\t2021-05-07 19:34:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:34:58\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    4s.  Last read position: 19:12,074,722\n",
      "INFO\t2021-05-07 19:34:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:35:04\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    5s.  Last read position: 3:107,998,901\n",
      "INFO\t2021-05-07 19:35:04\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:35:16\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 6:158,197,265\n",
      "INFO\t2021-05-07 19:35:16\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 4925017, Read name D00645:415:CE7Y1ANXX:5:1106:10635:80139, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4925018, Read name D00645:415:CE7Y1ANXX:5:2303:8181:1926, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4925019, Read name D00645:415:CE7Y1ANXX:5:1214:11687:52480, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:35:21\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    5s.  Last read position: X:38,401,996\n",
      "INFO\t2021-05-07 19:35:21\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 5279595, Read name D00645:415:CE7Y1ANXX:5:1211:2465:72657, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5279596, Read name D00645:415:CE7Y1ANXX:5:1216:5805:64668, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5279618, Read name D00645:415:CE7Y1ANXX:5:2110:16284:98480, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:35:23\tMarkDuplicates\tRead 5279846 records. 0 pairs never matched.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 19:35:26\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 699222608; totalMemory: 725614592; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:35:26\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 19:35:26\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:35:26\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:35:26\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 19:35:27\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 19:35:27\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 486497904; totalMemory: 737673216; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:35:27\tMarkDuplicates\tMarking 56531 records as duplicates.\n",
      "INFO\t2021-05-07 19:35:27\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 19:35:27\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 4925017, Read name D00645:415:CE7Y1ANXX:5:1106:10635:80139, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4925018, Read name D00645:415:CE7Y1ANXX:5:2303:8181:1926, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 4925019, Read name D00645:415:CE7Y1ANXX:5:1214:11687:52480, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5279595, Read name D00645:415:CE7Y1ANXX:5:1211:2465:72657, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5279596, Read name D00645:415:CE7Y1ANXX:5:1216:5805:64668, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5279618, Read name D00645:415:CE7Y1ANXX:5:2110:16284:98480, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:36:13\tMarkDuplicates\tBefore output close freeMemory: 740944288; totalMemory: 754450432; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:36:13\tMarkDuplicates\tAfter output close freeMemory: 740944288; totalMemory: 754450432; maxMemory: 954728448\n",
      "[Fri May 07 19:36:13 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.43 minutes.\n",
      "Runtime.totalMemory()=754450432\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_4_46_CTCCTAGA_S48_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 68.43 sec\n",
      "[bwa_aln_core] write to the disk... 0.05 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.81 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.04 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.09 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.38 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.02 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.06 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.07 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.16 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.95 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 68.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.68 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.12 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.24 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.83 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.54 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.36 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.23 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.99 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.38 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.14 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 71.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.50 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.08 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.46 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.27 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.32 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.17 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11750727 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_4_46_CTCCTAGA_S48_R1_001.fastq.gz\n",
      "[main] Real time: 377.932 sec; CPU: 3114.827 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.63 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.23 sec\n",
      "[bwa_aln_core] 11750727 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_4_46_CTCCTAGA_S48_R1_001.fastq.gz\n",
      "[main] Real time: 382.431 sec; CPU: 234.499 sec\n",
      "[bam_sort_core] merging from 3 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 19:43:13\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_4_46_CTCCTAGA_S48_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_4_46_CTCCTAGA_S48_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_4_46_CTCCTAGA_S48_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "19:43:13.981 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 19:43:13 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_4_46_CTCCTAGA_S48_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_4_46_CTCCTAGA_S48_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_4_46_CTCCTAGA_S48_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 19:43:13 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 19:43:14\tMarkDuplicates\tStart of doWork freeMemory: 502528272; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:43:14\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 19:43:14\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 19:43:19\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 10:70,933,790\n",
      "INFO\t2021-05-07 19:43:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:43:24\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: 13:20,351,871\n",
      "INFO\t2021-05-07 19:43:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:43:29\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    4s.  Last read position: 16:71,986,467\n",
      "INFO\t2021-05-07 19:43:29\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:43:40\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:26s.  Time for last 1,000,000:   11s.  Last read position: 2:75,183,307\n",
      "INFO\t2021-05-07 19:43:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:43:45\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    5s.  Last read position: 3:13,548,848\n",
      "INFO\t2021-05-07 19:43:45\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:43:54\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:40s.  Time for last 1,000,000:    8s.  Last read position: 4:158,983,207\n",
      "INFO\t2021-05-07 19:43:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:44:00\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:46s.  Time for last 1,000,000:    5s.  Last read position: 6:128,913,565\n",
      "INFO\t2021-05-07 19:44:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:44:05\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:51s.  Time for last 1,000,000:    4s.  Last read position: 8:139,073,649\n",
      "INFO\t2021-05-07 19:44:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 8392783, Read name D00645:415:CE7Y1ANXX:5:2108:15909:98578, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8858411, Read name D00645:415:CE7Y1ANXX:5:1108:19325:65310, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8884386, Read name D00645:415:CE7Y1ANXX:5:1110:9668:13675, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8934264, Read name D00645:415:CE7Y1ANXX:5:2112:7875:51954, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8934527, Read name D00645:415:CE7Y1ANXX:5:2202:7255:25684, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8934528, Read name D00645:415:CE7Y1ANXX:5:2102:5017:67485, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8936556, Read name D00645:415:CE7Y1ANXX:5:2208:18772:33234, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8936933, Read name D00645:415:CE7Y1ANXX:5:2114:12522:86032, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:44:10\tMarkDuplicates\tRead 8938409 records. 0 pairs never matched.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 19:44:10\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 762683344; totalMemory: 788529152; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:44:10\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 19:44:10\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:44:10\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:44:10\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 19:44:12\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 19:44:12\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 520014776; totalMemory: 771227648; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:44:12\tMarkDuplicates\tMarking 111556 records as duplicates.\n",
      "INFO\t2021-05-07 19:44:12\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 19:44:12\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 8392783, Read name D00645:415:CE7Y1ANXX:5:2108:15909:98578, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8858411, Read name D00645:415:CE7Y1ANXX:5:1108:19325:65310, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8884386, Read name D00645:415:CE7Y1ANXX:5:1110:9668:13675, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8934264, Read name D00645:415:CE7Y1ANXX:5:2112:7875:51954, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8934527, Read name D00645:415:CE7Y1ANXX:5:2202:7255:25684, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8934528, Read name D00645:415:CE7Y1ANXX:5:2102:5017:67485, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8936556, Read name D00645:415:CE7Y1ANXX:5:2208:18772:33234, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8936933, Read name D00645:415:CE7Y1ANXX:5:2114:12522:86032, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:45:13\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:00s.  Time for last 10,000,000:   60s.  Last read position: */*\n",
      "INFO\t2021-05-07 19:45:21\tMarkDuplicates\tBefore output close freeMemory: 789591472; totalMemory: 803209216; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:45:21\tMarkDuplicates\tAfter output close freeMemory: 789591472; totalMemory: 803209216; maxMemory: 954728448\n",
      "[Fri May 07 19:45:21 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.12 minutes.\n",
      "Runtime.totalMemory()=803209216\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_5_54_GTGAAGTG_S49_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 49.40 sec\n",
      "[bwa_aln_core] write to the disk... 0.07 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 48.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.12 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 49.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.10 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 63.27 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.19 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.91 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.32 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.33 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.78 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.78 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.10 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.37 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.83 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.89 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.68 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.29 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.19 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 22.24 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.02 sec\n",
      "[bwa_aln_core] 8758594 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n[bwa_aln_core] convert to sequence coordinate...  2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_5_54_GTGAAGTG_S49_R1_001.fastq.gz\n",
      "[main] Real time: 263.041 sec; CPU: 1751.636 sec\n",
      "3.16 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.56 sec\n",
      "[bwa_aln_core] print alignments... 0.18 sec\n",
      "[bwa_aln_core] 8758594 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_5_54_GTGAAGTG_S49_R1_001.fastq.gz\n",
      "[main] Real time: 266.962 sec; CPU: 176.628 sec\n",
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 19:50:17\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_5_54_GTGAAGTG_S49_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_5_54_GTGAAGTG_S49_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_5_54_GTGAAGTG_S49_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "19:50:17.494 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 19:50:17 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_5_54_GTGAAGTG_S49_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_5_54_GTGAAGTG_S49_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_5_54_GTGAAGTG_S49_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 19:50:17 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 19:50:17\tMarkDuplicates\tStart of doWork freeMemory: 502528272; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:50:17\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 19:50:17\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 19:50:22\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 11:70,666,580\n",
      "INFO\t2021-05-07 19:50:22\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:50:28\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    5s.  Last read position: 14:100,241,663\n",
      "INFO\t2021-05-07 19:50:28\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:50:34\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    5s.  Last read position: 19:48,624,969\n",
      "INFO\t2021-05-07 19:50:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:50:45\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 22:24,794,841\n",
      "INFO\t2021-05-07 19:50:45\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:50:51\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    5s.  Last read position: 5:58,184,046\n",
      "INFO\t2021-05-07 19:50:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:51:00\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    9s.  Last read position: 7:48,671,229\n",
      "INFO\t2021-05-07 19:51:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:51:11\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:54s.  Time for last 1,000,000:   11s.  Last read position: 9:134,037,291\n",
      "INFO\t2021-05-07 19:51:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 7317003, Read name D00645:415:CE7Y1ANXX:5:1206:2459:48633, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7353007, Read name D00645:415:CE7Y1ANXX:5:2206:14922:75388, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7353008, Read name D00645:415:CE7Y1ANXX:5:2208:9885:89499, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:51:13\tMarkDuplicates\tRead 7353373 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 19:51:13\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 886080448; totalMemory: 911736832; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:51:13\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 19:51:13\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:51:13\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 19:51:13\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 19:51:15\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 19:51:15\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 659064296; totalMemory: 910163968; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:51:15\tMarkDuplicates\tMarking 91513 records as duplicates.\n",
      "INFO\t2021-05-07 19:51:15\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 19:51:15\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 7317003, Read name D00645:415:CE7Y1ANXX:5:1206:2459:48633, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7353007, Read name D00645:415:CE7Y1ANXX:5:2206:14922:75388, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7353008, Read name D00645:415:CE7Y1ANXX:5:2208:9885:89499, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 19:52:10\tMarkDuplicates\tBefore output close freeMemory: 912861464; totalMemory: 926416896; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:52:10\tMarkDuplicates\tAfter output close freeMemory: 912861464; totalMemory: 926416896; maxMemory: 954728448\n",
      "[Fri May 07 19:52:10 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.88 minutes.\n",
      "Runtime.totalMemory()=926416896\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_6_65_GTTACGCA_S47_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 66.66 sec\n",
      "[bwa_aln_core] write to the disk... 0.09 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.85 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 71.55 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.02 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.87 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.52 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.41 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 72.06 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.55 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.21 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.23 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.41 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.45 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.38 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.39 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.68 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.22 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.69 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.24 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.22 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.68 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.32 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.88 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 71.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.91 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.59 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.10 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.75 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.21 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.85 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 69.03 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.85 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.14 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.99 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.07 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.00 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.19 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 72.41 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.95 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.60 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.88 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.81 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.21 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.66 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.67 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.16 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 46.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 11796480 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 11974381 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_6_65_GTTACGCA_S47_R1_001.fastq.gz\n",
      "[main] Real time: 389.060 sec; CPU: 3178.037 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.48 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.19 sec\n",
      "[bwa_aln_core] 11974381 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_6_65_GTTACGCA_S47_R1_001.fastq.gz\n",
      "[main] Real time: 393.337 sec; CPU: 242.162 sec\n",
      "[bam_sort_core] merging from 3 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 19:59:21\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_6_65_GTTACGCA_S47_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_6_65_GTTACGCA_S47_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_6_65_GTTACGCA_S47_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "19:59:21.791 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 19:59:21 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_6_65_GTTACGCA_S47_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_6_65_GTTACGCA_S47_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_6_65_GTTACGCA_S47_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 19:59:21 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 19:59:21\tMarkDuplicates\tStart of doWork freeMemory: 502528272; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 19:59:21\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 19:59:21\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 19:59:27\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 10:106,697,306\n",
      "INFO\t2021-05-07 19:59:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:59:32\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    5s.  Last read position: 13:31,035,882\n",
      "INFO\t2021-05-07 19:59:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:59:37\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    4s.  Last read position: 16:55,894,403\n",
      "INFO\t2021-05-07 19:59:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:59:49\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:27s.  Time for last 1,000,000:   11s.  Last read position: 2:83,153,610\n",
      "INFO\t2021-05-07 19:59:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 19:59:53\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    4s.  Last read position: 3:12,597,163\n",
      "INFO\t2021-05-07 19:59:53\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:00:02\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:40s.  Time for last 1,000,000:    8s.  Last read position: 4:183,027,049\n",
      "INFO\t2021-05-07 20:00:02\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:00:13\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:51s.  Time for last 1,000,000:   10s.  Last read position: 6:137,822,696\n",
      "INFO\t2021-05-07 20:00:13\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:00:18\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:56s.  Time for last 1,000,000:    4s.  Last read position: 8:116,955,236\n",
      "INFO\t2021-05-07 20:00:18\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 8522649, Read name D00645:415:CE7Y1ANXX:5:1103:6187:10874, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8530236, Read name D00645:415:CE7Y1ANXX:5:1102:2400:37422, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8876550, Read name D00645:415:CE7Y1ANXX:5:1206:4250:81861, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8936627, Read name D00645:415:CE7Y1ANXX:5:2110:13581:28609, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8938817, Read name D00645:415:CE7Y1ANXX:5:1215:5699:59012, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8939648, Read name D00645:415:CE7Y1ANXX:5:2112:8430:63669, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8939649, Read name D00645:415:CE7Y1ANXX:5:1308:17666:66708, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 20:00:22\tMarkDuplicates\tRead 8939987 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 20:00:23\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 883406608; totalMemory: 909639680; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:00:23\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 20:00:23\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 20:00:23\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 20:00:23\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 20:00:25\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 20:00:25\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 665215496; totalMemory: 916455424; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:00:25\tMarkDuplicates\tMarking 87416 records as duplicates.\n",
      "INFO\t2021-05-07 20:00:25\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 20:00:25\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 8522649, Read name D00645:415:CE7Y1ANXX:5:1103:6187:10874, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8530236, Read name D00645:415:CE7Y1ANXX:5:1102:2400:37422, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8876550, Read name D00645:415:CE7Y1ANXX:5:1206:4250:81861, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8936627, Read name D00645:415:CE7Y1ANXX:5:2110:13581:28609, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8938817, Read name D00645:415:CE7Y1ANXX:5:1215:5699:59012, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8939648, Read name D00645:415:CE7Y1ANXX:5:2112:8430:63669, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 8939649, Read name D00645:415:CE7Y1ANXX:5:1308:17666:66708, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 20:01:26\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:00s.  Time for last 10,000,000:   60s.  Last read position: */*\n",
      "INFO\t2021-05-07 20:01:34\tMarkDuplicates\tBefore output close freeMemory: 918691672; totalMemory: 932184064; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:01:34\tMarkDuplicates\tAfter output close freeMemory: 918691672; totalMemory: 932184064; maxMemory: 954728448\n",
      "[Fri May 07 20:01:34 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.22 minutes.\n",
      "Runtime.totalMemory()=932184064\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_7_66_ATGACGTC_S53_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 51.54 sec\n",
      "[bwa_aln_core] write to the disk... 0.07 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 50.77 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.12 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.63 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.00 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.79 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.39 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 51.39 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.37 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 54.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.26 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.16 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.15 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.37 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.62 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.39 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.09 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.40 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.17 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.04 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.20 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 55.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.48 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.17 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 53.06 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.28 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.25 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 39.94 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.04 sec\n",
      "[bwa_aln_core] 7010778 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_7_66_ATGACGTC_S53_R1_001.fastq.gz\n",
      "[main] Real time: 215.221 sec; CPU: 1433.926 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.22 sec\n",
      "[bwa_aln_core] 7010778 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_7_66_ATGACGTC_S53_R1_001.fastq.gz\n",
      "[main] Real time: 219.662 sec; CPU: 144.101 sec\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bam_sort_core] merging from 2 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 20:05:38\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_7_66_ATGACGTC_S53_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_7_66_ATGACGTC_S53_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_7_66_ATGACGTC_S53_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "20:05:38.645 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 20:05:38 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_7_66_ATGACGTC_S53_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_7_66_ATGACGTC_S53_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_7_66_ATGACGTC_S53_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 20:05:38 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n",
      "INFO\t2021-05-07 20:05:38\tMarkDuplicates\tStart of doWork freeMemory: 502528272; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:05:38\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 20:05:38\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 20:05:44\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 11:128,507,539\n",
      "INFO\t2021-05-07 20:05:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:05:49\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: 16:55,210,894\n",
      "INFO\t2021-05-07 20:05:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:05:54\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    4s.  Last read position: 2:225,090,045\n",
      "INFO\t2021-05-07 20:05:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:06:05\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:27s.  Time for last 1,000,000:   11s.  Last read position: 4:145,755,312\n",
      "INFO\t2021-05-07 20:06:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:06:10\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    4s.  Last read position: 7:110,623,716\n",
      "INFO\t2021-05-07 20:06:10\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 5644956, Read name D00645:415:CE7Y1ANXX:5:1302:9978:100554, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5644957, Read name D00645:415:CE7Y1ANXX:5:1303:18751:24731, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5644958, Read name D00645:415:CE7Y1ANXX:5:1105:3286:60397, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 20:06:18\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:39s.  Time for last 1,000,000:    8s.  Last read position: KI270709.1:15,151\n",
      "INFO\t2021-05-07 20:06:18\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 6003096, Read name D00645:415:CE7Y1ANXX:5:2301:14711:88834, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6003441, Read name D00645:415:CE7Y1ANXX:5:1304:9330:15063, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 20:06:18\tMarkDuplicates\tRead 6005749 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 20:06:19\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 752914928; totalMemory: 778567680; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:06:19\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 20:06:19\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 20:06:19\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 20:06:19\tSortingCollection\tCreating merging iterator from 2 files\n",
      "INFO\t2021-05-07 20:06:21\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 20:06:21\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 536481400; totalMemory: 787480576; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:06:21\tMarkDuplicates\tMarking 54707 records as duplicates.\n",
      "INFO\t2021-05-07 20:06:21\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 20:06:21\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 5644956, Read name D00645:415:CE7Y1ANXX:5:1302:9978:100554, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5644957, Read name D00645:415:CE7Y1ANXX:5:1303:18751:24731, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 5644958, Read name D00645:415:CE7Y1ANXX:5:1105:3286:60397, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6003096, Read name D00645:415:CE7Y1ANXX:5:2301:14711:88834, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 6003441, Read name D00645:415:CE7Y1ANXX:5:1304:9330:15063, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 20:07:03\tMarkDuplicates\tBefore output close freeMemory: 786518736; totalMemory: 800063488; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:07:03\tMarkDuplicates\tAfter output close freeMemory: 788091600; totalMemory: 801636352; maxMemory: 954728448\n",
      "[Fri May 07 20:07:03 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.42 minutes.\n",
      "Runtime.totalMemory()=801636352\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_8_71_CAGTCCAA_S52_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 66.04 sec\n",
      "[bwa_aln_core] write to the disk... 0.08 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 64.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.89 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.86 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 66.91 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.70 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.77 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.56 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.33 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.79 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.92 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.85 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.03 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.84 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 65.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.83 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.56 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.89 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.90 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.88 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.94 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.59 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.58 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.83 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.41 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 66.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.11 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.86 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.18 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.36 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.01 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.06 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.90 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 67.21 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.91 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.02 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 71.47 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.93 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.92 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.37 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.94 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.35 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.72 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 69.84 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.12 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.27 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 68.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.10 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 70.68 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 42.61 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 10380550 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2 /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_8_71_CAGTCCAA_S52_R1_001.fastq.gz\n",
      "[main] Real time: 333.946 sec; CPU: 2733.105 sec\n",
      "[bwa_aln_core] convert to sequence coordinate... 3.38 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.58 sec\n",
      "[bwa_aln_core] print alignments... 0.20 sec\n",
      "[bwa_aln_core] 10380550 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_8_71_CAGTCCAA_S52_R1_001.fastq.gz\n",
      "[main] Real time: 338.148 sec; CPU: 205.974 sec\n",
      "[bam_sort_core] merging from 3 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 20:13:16\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_8_71_CAGTCCAA_S52_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_8_71_CAGTCCAA_S52_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_8_71_CAGTCCAA_S52_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "20:13:16.618 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 20:13:16 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_8_71_CAGTCCAA_S52_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_8_71_CAGTCCAA_S52_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_8_71_CAGTCCAA_S52_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 20:13:16 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 20:13:16\tMarkDuplicates\tStart of doWork freeMemory: 502528272; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:13:16\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 20:13:16\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 20:13:24\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:07s.  Time for last 1,000,000:    7s.  Last read position: 11:13,109,374\n",
      "INFO\t2021-05-07 20:13:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:13:29\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    5s.  Last read position: 14:81,545,177\n",
      "INFO\t2021-05-07 20:13:29\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:13:34\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    4s.  Last read position: 19:41,252,976\n",
      "INFO\t2021-05-07 20:13:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:13:45\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:29s.  Time for last 1,000,000:   11s.  Last read position: 21:36,818,873\n",
      "INFO\t2021-05-07 20:13:45\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:13:50\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:34s.  Time for last 1,000,000:    4s.  Last read position: 4:189,218,951\n",
      "INFO\t2021-05-07 20:13:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:13:58\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    8s.  Last read position: 7:43,724,916\n",
      "INFO\t2021-05-07 20:13:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:14:05\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:48s.  Time for last 1,000,000:    6s.  Last read position: 9:27,943,049\n",
      "INFO\t2021-05-07 20:14:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 7757519, Read name D00645:415:CE7Y1ANXX:5:1215:8732:46850, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7785036, Read name D00645:415:CE7Y1ANXX:5:1204:8193:15943, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7803045, Read name D00645:415:CE7Y1ANXX:5:2114:12991:50245, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7804823, Read name D00645:415:CE7Y1ANXX:5:2204:7378:47163, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7805467, Read name D00645:415:CE7Y1ANXX:5:2308:19521:76309, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7805468, Read name D00645:415:CE7Y1ANXX:5:1304:3683:38707, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 20:14:09\tMarkDuplicates\tRead 7805752 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 20:14:10\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 740844432; totalMemory: 767557632; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:14:10\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 20:14:10\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 20:14:10\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 20:14:10\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 20:14:11\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 20:14:11\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 515771024; totalMemory: 767033344; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:14:11\tMarkDuplicates\tMarking 72231 records as duplicates.\n",
      "INFO\t2021-05-07 20:14:11\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 20:14:11\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 7757519, Read name D00645:415:CE7Y1ANXX:5:1215:8732:46850, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7785036, Read name D00645:415:CE7Y1ANXX:5:1204:8193:15943, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7803045, Read name D00645:415:CE7Y1ANXX:5:2114:12991:50245, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7804823, Read name D00645:415:CE7Y1ANXX:5:2204:7378:47163, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7805467, Read name D00645:415:CE7Y1ANXX:5:2308:19521:76309, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 7805468, Read name D00645:415:CE7Y1ANXX:5:1304:3683:38707, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 20:15:10\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:00:58s.  Time for last 10,000,000:   58s.  Last read position: */*\n",
      "INFO\t2021-05-07 20:15:12\tMarkDuplicates\tBefore output close freeMemory: 769713264; totalMemory: 783286272; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:15:12\tMarkDuplicates\tAfter output close freeMemory: 769713264; totalMemory: 783286272; maxMemory: 954728448\n",
      "[Fri May 07 20:15:12 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.93 minutes.\n",
      "Runtime.totalMemory()=783286272\n",
      "!!!!!!!!!!!!!! Expected final output {/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_9_92_CGACGTTA_S54_R1_001.markdup.bam}\n",
      "[bwa_aln_core] calculate SA coordinate... 56.21 sec\n",
      "[bwa_aln_core] write to the disk... 0.06 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.10 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 262144 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.69 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 524288 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.30 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.28 sec\n",
      "[bwa_aln_core] 786432 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.20 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.13 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 1048576 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.25 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.40 sec\n",
      "[bwa_aln_core] 1310720 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.76 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.00 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1572864 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.65 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 1835008 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bwa_aln_core] calculate SA coordinate... 57.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2097152 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.56 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.31 sec\n",
      "[bwa_aln_core] 2359296 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.51 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2621440 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.26 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.97 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 2883584 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.82 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.02 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 3145728 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.12 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3407872 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.93 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.96 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3670016 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.07 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 3932160 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.15 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.01 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4194304 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.85 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 3.95 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4456448 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.64 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.03 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4718592 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.88 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 4980736 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.05 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5242880 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.94 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5505024 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.13 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.04 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 5767168 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.87 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.11 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.62 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 6029312 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.08 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6291456 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.98 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.24 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 6553600 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 56.44 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.26 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 6815744 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.17 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.14 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7077888 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 72.08 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.24 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7340032 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.99 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.12 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.61 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 7602176 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.23 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.60 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 7864320 sequences have been processed.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.05 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.73 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.52 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 8126464 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.49 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.67 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.79 sec\n",
      "[bwa_aln_core] print alignments... 0.32 sec\n",
      "[bwa_aln_core] 8388608 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.35 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 8650752 sequences have been processed.\n",
      "0.09 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.36 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.19 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.64 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 8912896 sequences have been processed.\n",
      "0.06 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.18 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.06 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.74 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 9175040 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.30 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.57 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.78 sec\n",
      "[bwa_aln_core] print alignments... 0.35 sec\n",
      "[bwa_aln_core] 9437184 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 60.97 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.30 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.69 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 9699328 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.94 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.38 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.73 sec\n",
      "[bwa_aln_core] print alignments... 0.30 sec\n",
      "[bwa_aln_core] 9961472 sequences have been processed.\n",
      "0.05 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.39 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.40 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.69 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 10223616 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 58.53 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.14 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.67 sec\n",
      "[bwa_aln_core] print alignments... 0.34 sec\n",
      "[bwa_aln_core] 10485760 sequences have been processed.\n",
      "0.08 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 52.57 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.39 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.38 sec\n",
      "[bwa_aln_core] 10747904 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 57.34 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.57 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11010048 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 59.22 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.60 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11272192 sequences have been processed.\n",
      "0.07 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "[bwa_aln_core] calculate SA coordinate... 6.31 sec\n",
      "[bwa_aln_core] write to the disk... [bwa_aln_core] convert to sequence coordinate... 4.63 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.63 sec\n",
      "[bwa_aln_core] print alignments... 0.29 sec\n",
      "[bwa_aln_core] 11534336 sequences have been processed.\n",
      "0.01 sec\n",
      "[bwa_aln_core] 11558997 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa aln -t 32 -n 2[bwa_aln_core] convert to sequence coordinate...  /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_9_92_CGACGTTA_S54_R1_001.fastq.gz\n",
      "[main] Real time: 370.698 sec; CPU: 2597.612 sec\n",
      "2.98 sec\n",
      "[bwa_aln_core] refine gapped alignments... 0.55 sec\n",
      "[bwa_aln_core] print alignments... 0.04 sec\n",
      "[bwa_aln_core] 11558997 sequences have been processed.\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa samse -r @RG\\tID:inputfqgz\\tSM:inputfqgz /DATA/share/pipelines/lgleon/References/Ensembl/NGSfacility/Homo_sapiens.GRCh38.dna.primary_assembly.fa - /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_9_92_CGACGTTA_S54_R1_001.fastq.gz\n",
      "[main] Real time: 374.279 sec; CPU: 243.975 sec\n",
      "[bam_sort_core] merging from 3 files and 1 in-memory blocks...\n",
      "INFO\t2021-05-07 20:22:03\tMarkDuplicates\t\n",
      "\n",
      "********** NOTE: Picard's command line syntax is changing.\n",
      "**********\n",
      "********** For more information, please see:\n",
      "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
      "**********\n",
      "********** The command line looks like this in the new syntax:\n",
      "**********\n",
      "**********    MarkDuplicates -I /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_9_92_CGACGTTA_S54_R1_001.sorted.bam -O /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_9_92_CGACGTTA_S54_R1_001.markdup.bam -M /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_9_92_CGACGTTA_S54_R1_001.markdup_metrics.txt -ASSUME_SORTED true -CREATE_INDEX false -VALIDATION_STRINGENCY LENIENT\n",
      "**********\n",
      "\n",
      "\n",
      "20:22:04.138 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/miniconda3/share/picard-2.18.29-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
      "[Fri May 07 20:22:04 CEST 2021] MarkDuplicates INPUT=[/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_9_92_CGACGTTA_S54_R1_001.sorted.bam] OUTPUT=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_9_92_CGACGTTA_S54_R1_001.markdup.bam METRICS_FILE=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq/5867_9_92_CGACGTTA_S54_R1_001.markdup_metrics.txt ASSUME_SORTED=true VALIDATION_STRINGENCY=LENIENT CREATE_INDEX=false    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
      "[Fri May 07 20:22:04 CEST 2021] Executing as l.gonzalez@virchow on Linux 5.4.0-72-generic amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.29-SNAPSHOT\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO\t2021-05-07 20:22:04\tMarkDuplicates\tStart of doWork freeMemory: 502528272; totalMemory: 514850816; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:22:04\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2021-05-07 20:22:04\tMarkDuplicates\tWill retain up to 3459161 data points before spilling to disk.\n",
      "INFO\t2021-05-07 20:22:10\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    6s.  Last read position: 10:68,107,769\n",
      "INFO\t2021-05-07 20:22:10\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:22:15\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    4s.  Last read position: 12:95,357,273\n",
      "INFO\t2021-05-07 20:22:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:22:20\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    5s.  Last read position: 15:77,385,573\n",
      "INFO\t2021-05-07 20:22:20\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:22:32\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:   11s.  Last read position: 19:36,995,364\n",
      "INFO\t2021-05-07 20:22:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:22:37\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    5s.  Last read position: 20:35,614,714\n",
      "INFO\t2021-05-07 20:22:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:22:46\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    8s.  Last read position: 4:529,266\n",
      "INFO\t2021-05-07 20:22:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:22:57\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:53s.  Time for last 1,000,000:   11s.  Last read position: 5:132,813,454\n",
      "INFO\t2021-05-07 20:22:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:23:02\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:58s.  Time for last 1,000,000:    4s.  Last read position: 7:90,664,431\n",
      "INFO\t2021-05-07 20:23:02\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2021-05-07 20:23:08\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:01:03s.  Time for last 1,000,000:    5s.  Last read position: 9:76,472,075\n",
      "INFO\t2021-05-07 20:23:08\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "Ignoring SAM validation error: ERROR: Record 9272252, Read name D00645:415:CE7Y1ANXX:5:1310:6730:9741, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272253, Read name D00645:415:CE7Y1ANXX:5:2215:16010:27859, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272254, Read name D00645:415:CE7Y1ANXX:5:1110:8789:94034, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272255, Read name D00645:415:CE7Y1ANXX:5:2216:7134:12767, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272256, Read name D00645:415:CE7Y1ANXX:5:2205:3416:90978, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272257, Read name D00645:415:CE7Y1ANXX:5:1316:16675:99645, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272258, Read name D00645:415:CE7Y1ANXX:5:1111:10144:23855, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9293704, Read name D00645:415:CE7Y1ANXX:5:1111:6079:13706, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9293705, Read name D00645:415:CE7Y1ANXX:5:2111:8726:41942, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9293706, Read name D00645:415:CE7Y1ANXX:5:1302:1505:85996, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9685410, Read name D00645:415:CE7Y1ANXX:5:2114:6617:95317, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9747488, Read name D00645:415:CE7Y1ANXX:5:2312:16035:9626, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9747765, Read name D00645:415:CE7Y1ANXX:5:2108:8032:97934, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 20:23:11\tMarkDuplicates\tRead 9751361 records. 0 pairs never matched.\n",
      "INFO\t2021-05-07 20:23:12\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 870106184; totalMemory: 896008192; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:23:12\tMarkDuplicates\tWill retain up to 29835264 duplicate indices before spilling to disk.\n",
      "INFO\t2021-05-07 20:23:12\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2021-05-07 20:23:12\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2021-05-07 20:23:12\tSortingCollection\tCreating merging iterator from 3 files\n",
      "INFO\t2021-05-07 20:23:15\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2021-05-07 20:23:15\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 651085336; totalMemory: 902299648; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:23:15\tMarkDuplicates\tMarking 90041 records as duplicates.\n",
      "INFO\t2021-05-07 20:23:15\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2021-05-07 20:23:15\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "Ignoring SAM validation error: ERROR: Record 9272252, Read name D00645:415:CE7Y1ANXX:5:1310:6730:9741, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272253, Read name D00645:415:CE7Y1ANXX:5:2215:16010:27859, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272254, Read name D00645:415:CE7Y1ANXX:5:1110:8789:94034, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272255, Read name D00645:415:CE7Y1ANXX:5:2216:7134:12767, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272256, Read name D00645:415:CE7Y1ANXX:5:2205:3416:90978, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272257, Read name D00645:415:CE7Y1ANXX:5:1316:16675:99645, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9272258, Read name D00645:415:CE7Y1ANXX:5:1111:10144:23855, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9293704, Read name D00645:415:CE7Y1ANXX:5:1111:6079:13706, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9293705, Read name D00645:415:CE7Y1ANXX:5:2111:8726:41942, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9293706, Read name D00645:415:CE7Y1ANXX:5:1302:1505:85996, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9685410, Read name D00645:415:CE7Y1ANXX:5:2114:6617:95317, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9747488, Read name D00645:415:CE7Y1ANXX:5:2312:16035:9626, MAPQ should be 0 for unmapped read.\n",
      "Ignoring SAM validation error: ERROR: Record 9747765, Read name D00645:415:CE7Y1ANXX:5:2108:8032:97934, MAPQ should be 0 for unmapped read.\n",
      "INFO\t2021-05-07 20:24:16\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:01s.  Time for last 10,000,000:   61s.  Last read position: */*\n",
      "INFO\t2021-05-07 20:24:23\tMarkDuplicates\tBefore output close freeMemory: 904413064; totalMemory: 918028288; maxMemory: 954728448\n",
      "INFO\t2021-05-07 20:24:23\tMarkDuplicates\tAfter output close freeMemory: 904413064; totalMemory: 918028288; maxMemory: 954728448\n",
      "[Fri May 07 20:24:23 CEST 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 2.33 minutes.\n",
      "Runtime.totalMemory()=918028288\n",
      "mv: cannot stat '*.sorted.bam': No such file or directory\n",
      "mv: cannot stat '*markdup.bam': No such file or directory\n"
     ]
    }
   ],
   "source": [
    "## Alignment with bwa aln, code is at /home/l.gonzalez/Notebooks/bash_R_scripts/alternatve_bwaAln_hg38_lgl.sh\n",
    "## Go to the folder where you want to save the results\n",
    "## Check in the code, reference is hard copied\n",
    "## path to the fastq files must be change for each study or anaylsis\n",
    "## this code I meant to be run from the same folder where teh fastq files are store\n",
    "### Remember the project folder IS HARDCODE, need to be changed\n",
    "# once you chack the folder, just the cell \n",
    "\n",
    "!bash /DATA/home/l.gonzalez/Notebooks/bash_R_scripts/alternatve_bwaAln_hg38_lgl.sh\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Check the results, \n",
    "put them in the appropiate folders. Folder should be created in the process, but better check\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Indexing the bam files to create the bai, otherwise we can not continue"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#script\n",
    "#!/bin/bash\n",
    "#prj=/DATA/share/pipelines/lgleon/organoids_rawdata/DNA/markdup\n",
    "prj=/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/fastq\n",
    "for i in ${prj}/*markdup.bam\n",
    "\n",
    "do\n",
    "\n",
    "echo \"indexing: \"$i\n",
    "samtools index $i $i\".bai\"\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5564_10_Tissue_pt_264_2_S40_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5564_2_Tissue_pt_83_2_S32_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5564_5_Tissue_pt_120_2_S35_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5564_6_Tissue_pt_137_2_S36_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5564_9_Tissue_pt_216_2_S39_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5710_1_74_CTGATCGT_S2_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5710_19_88_CTTAGGAC_S19_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5710_5_152_CTTGTCGA_S9_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_10_110_TAACCGGT_S51_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_11_111_CTACTTGG_S41_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_1_1_ATGCACGA_S42_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_12_132_CAACGGAT_S43_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_13_142_CATGGCTA_S46_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_14_162_AGCGTGTT_S55_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_15_181_TTGGACGT_S50_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_16_186_TTGCAGAC_S56_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_17_203_TACGCCTT_S57_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_18_221_AACCGTTC_S58_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_19_229_CCTCAGTT_S59_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_20_272_TGGCTATC_S60_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_2_30_GAACATCG_S45_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_3_32_TCACAGCA_S44_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_4_46_CTCCTAGA_S48_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_5_54_GTGAAGTG_S49_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_6_65_GTTACGCA_S47_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_7_66_ATGACGTC_S53_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_8_71_CAGTCCAA_S52_R1_001.markdup.bam\n",
      "indexing: /DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/markdup/5867_9_92_CGACGTTA_S54_R1_001.markdup.bam\n"
     ]
    }
   ],
   "source": [
    "# for Indexing the bam I have a tiny script that work quite nicely\n",
    "### Remember the project folder IS HARDCODE, need to be changed\n",
    "\n",
    "##WORKS\n",
    "\n",
    "!bash /DATA/home/l.gonzalez/Notebooks/bash_R_scripts/indexing.sh"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### After the analysis we will check the QC of the steps that we ran\n",
    "To do that, please go to the cell below and change the path to the folders where the fastqc files are, also the bam/bai files and multiqc will create a html with all the files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# check and change the path to the folder then click run:\n",
    "!multiqc /DATA/share/pipelines/lgleon/test/testCode_steps_Nousefuldata/testdata/fastq/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run QDNAseq for the calling, \n",
    "This new script, modified form Erik run from bam to called \n",
    "The CLI can be run directly but using R\n",
    "\n",
    "The code have some hardcode parts, path to project directory(output) and path to bam files (input)\n",
    "If you wanna run it with hg38, not need to change that part\n",
    "For the bin size it should be change also inside the code, for the calling and devawing \n",
    "\n",
    "To make those cahnges, there are three options:\n",
    "- Open the file here using the notebooks \n",
    "- Open in Rstudio  \n",
    "- Open in the terminal\n",
    "\n",
    "Since it is a Rmarkdown file there are also two ways to run it:\n",
    "- Inside the Rstudio, step/chunk by step/chunk\n",
    "- From terminal (easier)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "R version 3.6.3 (2020-02-29) -- \"Holding the Windsock\"\n",
      "Copyright (C) 2020 The R Foundation for Statistical Computing\n",
      "Platform: x86_64-pc-linux-gnu (64-bit)\n",
      "\n",
      "R is free software and comes with ABSOLUTELY NO WARRANTY.\n",
      "You are welcome to redistribute it under certain conditions.\n",
      "Type 'license()' or 'licence()' for distribution details.\n",
      "\n",
      "  Natural language support but running in an English locale\n",
      "\n",
      "R is a collaborative project with many contributors.\n",
      "Type 'contributors()' for more information and\n",
      "'citation()' on how to cite R or R packages in publications.\n",
      "\n",
      "Type 'demo()' for some demos, 'help()' for on-line help, or\n",
      "'help.start()' for an HTML browser interface to help.\n",
      "Type 'q()' to quit R.\n",
      "\n",
      "> rmarkdown::render('/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/qdnaseq_output/QDNASeqFlow_cergentis-hg38_30Kb.Rmd')\n",
      "\n",
      "\n",
      "processing file: QDNASeqFlow_cergentis-hg38_30Kb.Rmd\n",
      "  |..                                                                    |   3%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.....                                                                 |   7%\n",
      "label: install (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |.......                                                               |  10%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.........                                                             |  13%\n",
      "label: functions (with options) \n",
      "List of 2\n",
      " $ message: logi FALSE\n",
      " $ warning: logi FALSE\n",
      "\n",
      "  |............                                                          |  17%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |..............                                                        |  20%\n",
      "label: setup (with options) \n",
      "List of 1\n",
      " $ include: logi FALSE\n",
      "\n",
      "  |................                                                      |  23%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |...................                                                   |  27%\n",
      "label: unnamed-chunk-1\n",
      "  |.....................                                                 |  30%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.......................                                               |  33%\n",
      "label: unnamed-chunk-2\n",
      "  |..........................                                            |  37%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |............................                                          |  40%\n",
      "label: unnamed-chunk-3\n",
      "  |..............................                                        |  43%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.................................                                     |  47%\n",
      "label: unnamed-chunk-4\n",
      "  |...................................                                   |  50%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.....................................                                 |  53%\n",
      "label: unnamed-chunk-5 (with options) \n",
      "List of 1\n",
      " $ message: logi FALSE\n",
      "\n",
      "Performing segmentation:\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "NA\n",
      "EM algorithm started ... \n",
      "\n",
      "411147541777398219.6318.87463545108359326398.6826.87463545135449157398.61033.4\n",
      "Calling iteration1:\n",
      "\n",
      "optim results\n",
      "time: 47\n",
      "minimum: 1153105.52987857\n",
      "281152671.82896009-0.850918366533648-0.3687318489574320.00618926673547950.3791897146827070.6482290988144241.085608863438260.1562493322648640.1304019687890120.08938660985108530.1382345701000480.1615395679486620.167634795276553\n",
      "EM algorithm done ...\n",
      "\n",
      "Computing posterior probabilities for all segments ...\n",
      "\n",
      "Total time:4minutes\n",
      "\n",
      "Adjusting segmented data for cellularity ... \n",
      "\n",
      "Cellularity sample1: 1\n",
      "\n",
      "Cellularity sample2: 1\n",
      "\n",
      "Cellularity sample3: 1\n",
      "\n",
      "Cellularity sample4: 1\n",
      "\n",
      "Cellularity sample5: 1\n",
      "\n",
      "Cellularity sample6: 1\n",
      "\n",
      "Cellularity sample7: 1\n",
      "\n",
      "Cellularity sample8: 1\n",
      "\n",
      "Cellularity sample9: 1\n",
      "\n",
      "Cellularity sample10: 1\n",
      "\n",
      "Cellularity sample11: 1\n",
      "\n",
      "Cellularity sample12: 1\n",
      "\n",
      "Cellularity sample13: 1\n",
      "\n",
      "Cellularity sample14: 1\n",
      "\n",
      "Cellularity sample15: 1\n",
      "\n",
      "Cellularity sample16: 1\n",
      "\n",
      "Cellularity sample17: 1\n",
      "\n",
      "Cellularity sample18: 1\n",
      "\n",
      "Cellularity sample19: 1\n",
      "\n",
      "Cellularity sample20: 1\n",
      "\n",
      "Cellularity sample21: 1\n",
      "\n",
      "Cellularity sample22: 1\n",
      "\n",
      "Cellularity sample23: 1\n",
      "\n",
      "Cellularity sample24: 1\n",
      "\n",
      "Cellularity sample25: 1\n",
      "\n",
      "Cellularity sample26: 1\n",
      "\n",
      "Cellularity sample27: 1\n",
      "\n",
      "Cellularity sample28: 1\n",
      "\n",
      "Adjusting normalized data for cellularity ... \n",
      "\n",
      "Cellularity sample1: 1\n",
      "\n",
      "Cellularity sample2: 1\n",
      "\n",
      "Cellularity sample3: 1\n",
      "\n",
      "Cellularity sample4: 1\n",
      "\n",
      "Cellularity sample5: 1\n",
      "\n",
      "Cellularity sample6: 1\n",
      "\n",
      "Cellularity sample7: 1\n",
      "\n",
      "Cellularity sample8: 1\n",
      "\n",
      "Cellularity sample9: 1\n",
      "\n",
      "Cellularity sample10: 1\n",
      "\n",
      "Cellularity sample11: 1\n",
      "\n",
      "Cellularity sample12: 1\n",
      "\n",
      "Cellularity sample13: 1\n",
      "\n",
      "Cellularity sample14: 1\n",
      "\n",
      "Cellularity sample15: 1\n",
      "\n",
      "Cellularity sample16: 1\n",
      "\n",
      "Cellularity sample17: 1\n",
      "\n",
      "Cellularity sample18: 1\n",
      "\n",
      "Cellularity sample19: 1\n",
      "\n",
      "Cellularity sample20: 1\n",
      "\n",
      "Cellularity sample21: 1\n",
      "\n",
      "Cellularity sample22: 1\n",
      "\n",
      "Cellularity sample23: 1\n",
      "\n",
      "Cellularity sample24: 1\n",
      "\n",
      "Cellularity sample25: 1\n",
      "\n",
      "Cellularity sample26: 1\n",
      "\n",
      "Cellularity sample27: 1\n",
      "\n",
      "Cellularity sample28: 1\n",
      "\n",
      "1\n",
      "451032046093969240.9351.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451033846508167240.9354.97463545108359326398.6826.87463545135449157398.61033.4\n",
      "451033846508167240.9354.97463545108359326398.6826.87463545135449157398.61033.4\n",
      "451033846508167240.9354.97463545108359326398.6826.87463545135449157398.61033.4\n",
      "451033846508167240.9354.97463545108359326398.6826.87463545135449157398.61033.4\n",
      "451033846508167240.9354.97463545108359326398.6826.87463545135449157398.61033.4\n",
      "451033846508167240.9354.97463545108359326398.6826.87463545135449157398.61033.4\n",
      "451033846508167240.9354.97463545108359326398.6826.87463545135449157398.61033.4\n",
      "451033646508164240.9354.97463545108359326398.6826.87463545135449157398.61033.4\n",
      "451036347668037240.9363.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451042747668175240.9363.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451043647668179240.9363.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451044547668183240.9363.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451045447668187240.9363.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451046347668191240.9363.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451046947668194240.9363.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451049047668232240.9363.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451091347668566241363.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "2\n",
      "4510931518933352413967463545108359326398.6826.87463545135449157398.61033.4\n",
      "4510933518933382413967463545108359326398.6826.87463545135449157398.61033.4\n",
      "4510933518933382413967463545108359326398.6826.87463545135449157398.61033.4\n",
      "4510933518933382413967463545108359326398.6826.87463545135449157398.61033.4\n",
      "4510933518933382413967463545108359326398.6826.87463545135449157398.61033.4\n",
      "4510933518933382413967463545108359326398.6826.87463545135449157398.61033.4\n",
      "4510933518933382413967463545108359326398.6826.87463545135449157398.61033.4\n",
      "4510933518933382413967463545108359326398.6826.87463545135449157398.61033.4\n",
      "4510931518933352413967463545108359326398.6826.87463545135449157398.61033.4\n",
      "451093452473068241400.47463545108359326398.6826.87463545135449157398.61033.4\n",
      "451093452473068241400.47463545108359326398.6826.87463545135449157398.61033.4\n",
      "451094352473072241400.47463545108359326398.6826.87463545135449157398.61033.4\n",
      "451095252473076241400.47463545108359326398.6826.87463545135449157398.61033.4\n",
      "451096152473080241400.47463545108359326398.6826.87463545135449157398.61033.4\n",
      "451097052473084241400.47463545108359326398.6826.87463545135449157398.61033.4\n",
      "451097652473087241400.47463545108359326398.6826.87463545135449157398.61033.4\n",
      "451099752473125241400.47463545108359326398.6826.87463545135449157398.61033.4\n",
      "451514855382846241.2422.67463545108359326398.6826.87463545135449157398.61033.4\n",
      "3\n",
      "451515458365031241.2445.37463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515658365034241.2445.37463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515658365034241.2445.37463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515658365034241.2445.37463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515658365034241.2445.37463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515658365034241.2445.37463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515658365034241.2445.37463545108359326398.6826.87463545135449157398.61033.4\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "451515658365034241.2445.37463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515458365031241.2445.37463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515758944934241.2449.87463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515258282221241.2444.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451516158282225241.2444.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451517058282229241.2444.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451517958282233241.2444.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451518858282237241.2444.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451519458282240241.2444.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451521558282278241.2444.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451514860021865241.24587463545108359326398.6826.87463545135449157398.61033.4\n",
      "4\n",
      "451515463004050241.2480.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515663004053241.2480.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515663004053241.2480.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515663004053241.2480.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515663004053241.2480.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515663004053241.2480.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515663004053241.2480.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515663004053241.2480.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515463004050241.2480.77463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515763587724241.2485.27463545108359326398.6826.87463545135449157398.61033.4\n",
      "451515262925011241.2480.17463545108359326398.6826.87463545135449157398.61033.4\n",
      "451516162925015241.2480.17463545108359326398.6826.87463545135449157398.61033.4\n",
      "451517062925019241.2480.17463545108359326398.6826.87463545135449157398.61033.4\n",
      "451517962925023241.2480.17463545108359326398.6826.87463545135449157398.61033.4\n",
      "451518862925027241.2480.17463545108359326398.6826.87463545135449157398.61033.4\n",
      "451519462925030241.2480.17463545108359326398.6826.87463545135449157398.61033.4\n",
      "451521562925068241.2480.17463545108359326398.6826.87463545135449157398.61033.4\n",
      "451514864664677241.2493.47463545108359326398.6826.87463545135449157398.61033.4\n",
      "FINISHED!\n",
      "\n",
      "Total time:1minutes\n",
      "\n",
      "  |........................................                              |  57%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |..........................................                            |  60%\n",
      "label: unnamed-chunk-6\n",
      "  |............................................                          |  63%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |...............................................                       |  67%\n",
      "label: unnamed-chunk-7\n",
      "  |.................................................                     |  70%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |...................................................                   |  73%\n",
      "label: unnamed-chunk-8\n",
      "  |......................................................                |  77%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |........................................................              |  80%\n",
      "label: unnamed-chunk-9\n",
      "  |..........................................................            |  83%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.............................................................         |  87%\n",
      "label: segment (with options) \n",
      "List of 2\n",
      " $ warning: logi FALSE\n",
      " $ message: logi FALSE\n",
      "\n",
      "  |...............................................................       |  90%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |.................................................................     |  93%\n",
      "label: TBL (with options) \n",
      "List of 2\n",
      " $ warning: logi FALSE\n",
      " $ message: logi FALSE\n",
      "\n",
      "Loading required package: CGHcall\n",
      "Loading required package: DNAcopy\n",
      "Loading required package: snowfall\n",
      "Loading required package: snow\n",
      "\n",
      "Attaching package: 'snow'\n",
      "\n",
      "The following objects are masked from 'package:BiocGenerics':\n",
      "\n",
      "    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,\n",
      "    clusterExport, clusterMap, clusterSplit, parApply, parCapply,\n",
      "    parLapply, parRapply, parSapply\n",
      "\n",
      "The following objects are masked from 'package:parallel':\n",
      "\n",
      "    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,\n",
      "    clusterExport, clusterMap, clusterSplit, makeCluster, parApply,\n",
      "    parCapply, parLapply, parRapply, parSapply, splitIndices,\n",
      "    stopCluster\n",
      "\n",
      "\n",
      "Attaching package: 'CGHcall'\n",
      "\n",
      "The following object is masked from 'package:BiocGenerics':\n",
      "\n",
      "    normalize\n",
      "\n",
      "Loading required package: GenomicRanges\n",
      "Loading required package: stats4\n",
      "Loading required package: S4Vectors\n",
      "\n",
      "Attaching package: 'S4Vectors'\n",
      "\n",
      "The following object is masked from 'package:future':\n",
      "\n",
      "    values\n",
      "\n",
      "The following object is masked from 'package:base':\n",
      "\n",
      "    expand.grid\n",
      "\n",
      "Loading required package: IRanges\n",
      "\n",
      "Attaching package: 'IRanges'\n",
      "\n",
      "The following object is masked _by_ '.GlobalEnv':\n",
      "\n",
      "    shift\n",
      "\n",
      "Loading required package: GenomeInfoDb\n",
      "\n",
      "Attaching package: 'GeneBreak'\n",
      "\n",
      "The following object is masked _by_ '.GlobalEnv':\n",
      "\n",
      "    sampleNames\n",
      "\n",
      "The following object is masked from 'package:CGHcall':\n",
      "\n",
      "    segmentData\n",
      "\n",
      "The following object is masked from 'package:Biobase':\n",
      "\n",
      "    sampleNames\n",
      "\n",
      "\u001b[0;1;31mFailed to query server: Connection timed out\u001b[0m\n",
      "── Attaching packages ─────────────────────────────────────── tidyverse 1.3.0 ──\n",
      "✔ tibble  3.1.1     ✔ purrr   0.3.4\n",
      "✔ tidyr   1.1.3     ✔ dplyr   1.0.6\n",
      "✔ readr   1.4.0     ✔ forcats 0.5.1\n",
      "── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──\n",
      "✖ dplyr::collapse()   masks IRanges::collapse()\n",
      "✖ dplyr::combine()    masks Biobase::combine(), BiocGenerics::combine()\n",
      "✖ tidyr::complete()   masks RCurl::complete()\n",
      "✖ dplyr::desc()       masks IRanges::desc()\n",
      "✖ tidyr::expand()     masks S4Vectors::expand()\n",
      "✖ dplyr::filter()     masks stats::filter()\n",
      "✖ dplyr::first()      masks S4Vectors::first()\n",
      "✖ dplyr::lag()        masks stats::lag()\n",
      "✖ ggplot2::Position() masks BiocGenerics::Position(), base::Position()\n",
      "✖ purrr::reduce()     masks GenomicRanges::reduce(), IRanges::reduce()\n",
      "✖ dplyr::rename()     masks S4Vectors::rename()\n",
      "✖ dplyr::select()     masks MASS::select()\n",
      "✖ dplyr::slice()      masks IRanges::slice()\n",
      "  |....................................................................  |  97%\n",
      "  ordinary text without R code\n",
      "\n",
      "  |......................................................................| 100%\n",
      "label: unnamed-chunk-10\n",
      "\n",
      "output file: QDNASeqFlow_cergentis-hg38_30Kb.knit.md\n",
      "\n",
      "/usr/bin/pandoc +RTS -K512m -RTS QDNASeqFlow_cergentis-hg38_30Kb.utf8.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output QDNASeqFlow_cergentis-hg38_30Kb.html --lua-filter /DATA/home/l.gonzalez/R/x86_64-pc-linux-gnu-library/3.6/rmarkdown/rmarkdown/lua/pagebreak.lua --lua-filter /DATA/home/l.gonzalez/R/x86_64-pc-linux-gnu-library/3.6/rmarkdown/rmarkdown/lua/latex-div.lua --self-contained --variable bs3=TRUE --standalone --section-divs --template /DATA/home/l.gonzalez/R/x86_64-pc-linux-gnu-library/3.6/rmarkdown/rmd/h/default.html --no-highlight --variable highlightjs=1 --variable theme=bootstrap --include-in-header /tmp/RtmpLVK8do/rmarkdown-str15a04f3cb2e888.html --mathjax --variable 'mathjax-url:https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML' \n",
      "\n",
      "Output created: QDNASeqFlow_cergentis-hg38_30Kb.html\n",
      "Warning messages:\n",
      "1: In system(\"timedatectl\", intern = TRUE) :\n",
      "  running command 'timedatectl' had status 1\n",
      "2: The `.dots` argument of `group_by()` is deprecated as of dplyr 1.0.0.\n",
      "This warning is displayed once every 8 hours.\n",
      "Call `lifecycle::last_warnings()` to see where this warning was generated. \n",
      "> \n",
      "> \n"
     ]
    }
   ],
   "source": [
    "# To run as CLI\n",
    "# Here is the example of the NGSprotocol data as an example, DO NOT RUN IT\n",
    "!R -e \"rmarkdown::render('/DATA/share/pipelines/lgleon/cergentis_C5/Raw_data/qdnaseq_output/QDNASeqFlow_cergentis-hg38_30Kb.Rmd')\"\n",
    "#!/opt/R/4.0.4/bin/Rscript -e \"rmarkdown::render('/DATA/share/pipelines/lgleon/NGS_protocol/results_QDNAseq/normalS/QDNASeqFlow_hg38_30Kb.Rmd')\"\n",
    "\n",
    "#The General Code, which is not, it is at \n",
    "# !R -e \"/home/l.gonzalez/Notebooks/bash_R_scripts/QDNASeqFlow_hg38_NGSprotocol_30Kb.Rmd\"\n",
    "\n",
    "#The best way to work is make a copy of the general one, in the results folder, and run it form there\n",
    "#as a knit it will create a html file at the same time\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] TRUE\r\n"
     ]
    }
   ],
   "source": [
    "#! selectR-config --select 3.6\n",
    "#! selectR-config --select 4.0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "R version 3.6.3 (2020-02-29) -- \"Holding the Windsock\"\n",
      "Copyright (C) 2020 The R Foundation for Statistical Computing\n",
      "Platform: x86_64-pc-linux-gnu (64-bit)\n",
      "\n",
      "R is free software and comes with ABSOLUTELY NO WARRANTY.\n",
      "You are welcome to redistribute it under certain conditions.\n",
      "Type 'license()' or 'licence()' for distribution details.\n",
      "\n",
      "  Natural language support but running in an English locale\n",
      "\n",
      "R is a collaborative project with many contributors.\n",
      "Type 'contributors()' for more information and\n",
      "'citation()' on how to cite R or R packages in publications.\n",
      "\n",
      "Type 'demo()' for some demos, 'help()' for on-line help, or\n",
      "'help.start()' for an HTML browser interface to help.\n",
      "Type 'q()' to quit R.\n",
      "\n",
      "> "
     ]
    }
   ],
   "source": [
    "#!R\n",
    "#!/opt/R/4.0.4/bin/Rscript"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "alignmentSieve\t\t  google-oauthlib-tool\t    pyrsa-keygen\r\n",
      "bamCompare\t\t  import_pb_to_tensorboard  pyrsa-priv2pub\r\n",
      "bamCoverage\t\t  markdown_py\t\t    pyrsa-sign\r\n",
      "bamPEFragmentSize\t  multiBamSummary\t    pyrsa-verify\r\n",
      "bigwigCompare\t\t  multiBigwigSummary\t    saved_model_cli\r\n",
      "computeGCBias\t\t  pbr\t\t\t    sphinx-apidoc\r\n",
      "computeMatrix\t\t  plotCorrelation\t    sphinx-autogen\r\n",
      "computeMatrixOperations   plotCoverage\t\t    sphinx-build\r\n",
      "correctGCBias\t\t  plotEnrichment\t    sphinx-quickstart\r\n",
      "deeptools\t\t  plotFingerprint\t    tensorboard\r\n",
      "estimateReadFiltering\t  plotHeatmap\t\t    tflite_convert\r\n",
      "estimateScaleFactor\t  plotPCA\t\t    tf_upgrade_v2\r\n",
      "estimator_ckpt_converter  plotProfile\t\t    toco\r\n",
      "f2py\t\t\t  pybabel\t\t    toco_from_protos\r\n",
      "f2py3\t\t\t  pyrsa-decrypt\t\t    tqdm\r\n",
      "f2py3.8\t\t\t  pyrsa-encrypt\t\t    wheel\r\n"
     ]
    }
   ],
   "source": [
    "#!ls /usr/local/bin/\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
